---
title: Where do the UniProtKB protein sequences come from?
categories: UniProtKB,Sequence,faq
---

More than 95% of the protein sequences provided by UniProtKB come from the translations of coding sequences (CDS) submitted to the EMBL-Bank/GenBank/DDBJ nucleotide sequence resources (International Nucleotide Sequence Database Collaboration ( [INSDC](http://www.insdc.org/) )). These CDS are either generated by gene prediction programs or are experimentally proven. A protein identifier ("protein\_id") is assigned to the translated CDS and can be found in the original EMBL-Bank/GenBank/DDBJ record and in the relevant UniProtKB entry.

The translated CDS sequences are automatically transferred to the TrEMBL section of UniProtKB. The TrEMBL records can be selected for further manual annotation and then integrated into the UniProtKB/Swiss-Prot section. The "protein\_id" are listed in the cross-reference part of the 'Sequence' section, of the UniProtKB entries (see for example [P13744](https://www.uniprot.org/uniprotkb/P13744#sequences) 'Translation').

In addition to translated CDS, UniProtKB protein sequences may come from:

-   the [PDB](http://www.wwpdb.org/) database.
-   sequences experimentally obtained by direct protein sequencing, by Edman degradation or MS/MS experiments and [submitted to UniProtKB/Swiss-Prot](http://www.uniprot.org/help/submissions) . Only about 5% of the UniProtKB/Swiss-Prot entries contain sequence data obtained by direct protein sequencing ( [list of entries with the keyword `'Direct protein sequencing'`](https://www.uniprot.org/uniprotkb/?query=keyword:KW-0903) ).
-   sequences scanned from the literature (i.g. [PRF](http://www.genome.ad.jp/dbget-bin/www%5Fbfind?prf) or other journal scan project).
-   sequences derived from gene prediction, not submitted to EMBL-Bank/GenBank/DDBJ ( [Ensembl](http://www.ensembl.org/), [Ensembl Genomes](http://ensemblgenomes.org/), [WormBase ParaSite](http://parasite.wormbase.org/) or [VectorBase](https://www.vectorbase.org/) ) (1), [RefSeq](https://www.ncbi.nlm.nih.gov/RefSeq/), [CCDS](https://www.ncbi.nlm.nih.gov/projects/CCDS/CcdsBrowse.cgi), etc).
-   sequences derived from in-house gene prediction, in very specific cases.

The FAQ [Does UniProtKB contain all protein sequences?](http://www.uniprot.org/help/uniprotkb%5Fcoverage) gives information on our UniProtKB protein sequence exclusion policies, e.g.Â for [redundant proteomes](http://www.uniprot.org/help/proteome%5Fredundancy) .

\(1\) Complementary pipelines for import of protein sequences have been developed in collaboration with Ensembl for vertebrate species, Ensembl Genomes for non-vertebrate species, WormBase ParaSite for parasitic nematodes and VectorBase for pathogen vector genomes. In addition, a new pipeline imports selected non-redundant genomes annotated by NCBI RefSeq. These sources provide proteome sequences for a number of key genomes of special interest where the INSDC submission is lacking gene model annotation.

To date, these pipeline have been used to populate UniProtKB with additional predicted sequences for the human and mouse proteomes as well as a number of other important vertebrate and non-vertebrate species. See: [What are proteomes?](http://www.uniprot.org/help/proteome)

See also:

-   [Why is UniProtKB composed of 2 sections, UniProtKB/Swiss-Prot and UniProtKB/TrEMBL?](http://www.uniprot.org/help/uniprotkb%5Fsections)
-   [Does UniProtKB contain all protein sequences?](http://www.uniprot.org/help/uniprotkb%5Fcoverage)
-   [What are UniProtKB's criteria for defining a CDS as a protein?](http://www.uniprot.org/help/cds%5Fprotein%5Fdefinition)
-   [Why do we keep dubious sequences in UniProtKB? How to discard them from a protein set?](http://www.uniprot.org/help/dubious%5Fsequences)
-   [How do I get the nucleotide sequence that corresponds to the canonical UniProtKB sequence?](http://www.uniprot.org/help/canonical%5Fnucleotide)
-   [Sequences](https://www.uniprot.org/help/sequences)

Related terms: imported, source, origin
