---
title: Transit peptide
type: help
categories: PTM_processing,manual
---

This subsection of the 'PTM / Processing' section describes the extent of a transit peptide.

Transit peptides are responsible for the transport of a protein encoded by a nuclear gene to a particular organelle. To date, we report transit peptides for the following organelles:

-   Mitochondrion
-   Apicoplast
-   Chromoplast
-   Chloroplast
-   Cyanelle
-   Thylakoid
-   Amyloplast
-   Peroxisome
-   Glyoxysome
-   Hydrogenosome

Cyanelles are the plastids of glaucocystophyte algae.

Peroxisomes, hydrogenosomes, glyoxysomes and glycosomes constitute the class of microbodies, which are defined as single membrane-bounded small organelles. Proteins can be targeted to microbodies via an N-terminal transit peptide (which is rare), or via a C-terminal motif (which is more common).

Chromoplasts are plastids containing pigments other than chlorophyll. Found in flowers, petals and fruits.

Transit peptides are generally cleaved from the mature protein and so will logically be found in proteins tagged with the 'Sequence processing' information: 'The displayed sequence is further processed into a mature form.'.

The particular organelle to which a protein is targeted is indicated in the ['Subcellular location'](https://www.uniprot.org/help/subcellular_location) subsection and by the presence of a specific keyword.

# 1. Annotation of experimentally proven transit peptides:

We annotate experimentally proven transit peptides when the cleavage site has been determined by direct protein sequencing.  
Example: [P15690](https://www.uniprot.org/uniprotkb/P15690#ptm_processing)

Large-scale proteomics data are used to annotate transit peptides through the combination of experimental and computational methods, tagged with evidence from ['Combined sources'](https://www.uniprot.org/help/evidences#ECO:00007744).  
Example: [P28331](https://www.uniprot.org/uniprotkb/P28331#ptm_processing)

This information can then be propagated ['By similarity'](https://www.uniprot.org/help/evidences#ECO:0000250) to closely related homologs provided that the transit peptide sequence is conserved.  
Example: [Q0MQG1](https://www.uniprot.org/uniprotkb/Q0MQG1#ptm_processing)

When a protein contains a transit peptide (according to experimental data or its similarity with a family of proteins), but the precise cleavage position has not been experimentally determined, we use a question mark instead of a precise position.  
Example: [P82673](https://www.uniprot.org/uniprotkb/P82673#ptm_processing)

# 2. Annotation of predicted transit peptides:

We also annotate transit peptides which are predicted by the application of the predictive tools [Mitofates](http://mitf.cbrc.jp/MitoFates/cgi-bin/top.cgi), [Predotar](https://urgi.versailles.inra.fr/Tools/Predotar) and [TargetP](https://services.healthtech.dtu.dk/services/TargetP-2.0/), but only when such predictions are consistent with the known or presumed subcellular location of the protein concerned. The predicted positions of the transit peptide are annotated with evidence ['Sequence analysis'](https://www.uniprot.org/help/evidences#ECO:0000255).  
Example: [P91929](https://www.uniprot.org/uniprotkb/P91929#ptm_processing)

# See also

-   [Evidence](https://www.uniprot.org/help/evidences)
-   [Peptide (describing small active peptides)](https://www.uniprot.org/help/peptide)
-   [Sequence annotation (features)](https://www.uniprot.org/help/sequence_annotation)

Related keywords:  
[Transit peptide](https://www.uniprot.org/keywords/809),  
[Chloroplast](https://www.uniprot.org/keywords/150),  
[Cyanelle](https://www.uniprot.org/keywords/194),  
[Glyoxysome](https://www.uniprot.org/keywords/330),  
[Hydrogenosome](https://www.uniprot.org/keywords/377),  
[Mitochondrion](https://www.uniprot.org/keywords/496),  
[Peroxisome](https://www.uniprot.org/keywords/576),  
[Thylakoid](https://www.uniprot.org/keywords/793).
