---
title: UniProt release 2013_03
categories: Prokaryotes
---

## Headline

### Latest from the prokaryotic world: bacterial Cas9, a new tool for genome engineering

The

CRISPR system (Clustered Regularly Interspaced Short Palindromic Repeat) is a bacterial and archaeal,

RNA -based adaptive immune system, which degrades invading genetic material. Very briefly, invading viruses or plasmids are recognized by their complementarity to

CRISPR RNA (crRNA) and degraded by dedicated nucleases.

There are 3 major

CRISPR systems, with a growing number of recognized subtypes depending on the Cas proteins (CRISPR-associated proteins) used to affect the various steps of crRNA generation and [invading nucleic acid destruction](http://www.ncbi.nlm.nih.gov/pubmed/21552286). In type I and

III CRISPR systems, different specialized Cas endonucleases generate crRNAs, which then assemble with other Cas proteins to create large crRNA-protein complexes that recognize and degrade invading nucleic acids complementary to the crRNA. Type

II CRISPR systems are a little different. In these systems, correct processing of pre-crRNA requires a

trans-encoded small

RNA (tracrRNA), endogenous RNase

III and the Cas9 protein. The tracrRNA serves as a guide for RNase

III -aided processing of pre-crRNA. Subsequently the Cas9/crRNA/tracrRNA complex endonucleolytically cleaves linear or circular [dsDNA target complementary to the crRNA](http://www.ncbi.nlm.nih.gov/pubmed/22745249,22949671). Degradation requires the Cas9 protein and both

RNA species. Thus, in type

II CRISPR systems, crRNA-guided degradation of

DNA relies upon a single protein. This discovery has implications beyond the world of bacteria. Expressing Cas9 with specifically chosen crRNA should allow site-specific genome modifications, knocking-out genes on demand not only in bacteria where it is already relatively simple to do so, but also in higher organisms, such as vertebrates.

And indeed it works! In 2 [back-to-back Science articles](http://www.ncbi.nlm.nih.gov/pubmed/23287718,23287722) published online in January of this year, Streptococcus pyogenes strain

SF370 Cas9 endonuclease was codon-optimized and targeted to the nucleus in human or mouse cells. In [one article](http://www.ncbi.nlm.nih.gov/pubmed/23287718), RNase

III was engineered in a similar fashion while the tracrRNA and pre-crRNA were expressed either separately or as a hybrid molecule, while in [the other](http://www.ncbi.nlm.nih.gov/pubmed/23287722), only a hybrid crRNA-tracrRNA was expressed. In both papers, various gene targets were cloned into the crRNA locus, leading to site-specific target cleavage which was subsequently repaired by either nonhomologous end-joining or homologous recombination. While the efficiency of the process varies, introducing multiple targets within a single gene or targeting multiple genes at a time is feasible, allowing for comparatively easy manipulation of a genome of interest. Additionally, [no toxicity](http://www.ncbi.nlm.nih.gov/pubmed/23287722) has been observed upon expression in human cells.

A similar approach has been successfully used not only in [other bacteria](http://www.ncbi.nlm.nih.gov/pubmed/23360965), but also in [zebrafish](http://www.ncbi.nlm.nih.gov/pubmed/23360964), as well as in different [human cell lines](http://www.ncbi.nlm.nih.gov/pubmed/23360966,23386978) .

The work described above has been carried out using

Cas9 from Streptococcus pyogenes strain

SF370, and the corresponding UniProtKB/Swiss-Prot entry has been updated, as have been experimentally characterized orthologous proteins in other bacteria (

Streptococcus thermophilus strain

DGCC7710,

Streptococcus thermophilus strain

ATCC BAA -491 /

LMD -9 and

Listeria innocua serovar 6a strain

CLIP 11262 ). Additionally, a new

HAMAP rule has been made for the Cas9 family ( [MF\_01480](http://hamap.expasy.org/unirule/MF_01480) ).

## UniProtKB news

### Cross-references to ChiTaRS

Cross-references have been added to ChiTaRS, a database of human, mouse and fruit fly chimeric transcripts and

RNA -sequencing data.

ChiTaRS is available at <http://chitars.bioinfo.cnio.es/>

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 46%" /><col style="width: 53%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>ChiTaRS</td></tr><tr class="even"><td>Resource identifier</td><td>gene name</td></tr><tr class="odd"><td>Optional information 1</td><td>organism name</td></tr><tr class="even"><td>Example</td><td><p>P16320 :</p><pre><code>DR   ChiTaRS; ATP6AP1; drosophila.</code></pre></td></tr></tbody></table>

[Show all the entries having a cross-reference to ChiTaRS](http://www.uniprot.org/uniprot/?query=database%3Achitars&sort=score) .

Cross-references to

SABIO -RK

Cross-references have been added to

SABIO -RK, a database of biochemical reaction kinetics.

SABIO -RK is available at <http://sabiork.h-its.org/>

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 50%" /><col style="width: 50%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>SABIO -RK</td></tr><tr class="even"><td>Resource identifier</td><td>UniProtKB accession number</td></tr><tr class="odd"><td>Example</td><td><p>P10172 :</p><pre><code>DR   SABIO-RK; P10172; -.</code></pre></td></tr></tbody></table>

Show all the entries having a cross-reference to

SABIO -RK .

### Removal of the cross-reference to 8 2D gel databases

Cross-references to 2DBase-Ecoli, Aarhus/Ghent-2DPAGE,

ANU -2DPAGE, Cornea-2DPAGE,

PHCI -2DPAGE,

PMMA -2DPAGE, Siena-2DPAGE, and Rat-heart-2DPAGE have been removed.

Removal of the cross-reference to

AGD

Cross-references to

AGD have been removed.

### Gene3D

The Gene3D database no longer provides names for their signatures. The entry name that has been displayed in the cross-references was therefore replaced by a dash (‘-’).

Examples:

Q12933 :

Previous format:

    DR   Gene3D; 2.60.210.10; TRAF-type; 1.
    DR   Gene3D; 3.30.40.10; Znf_RING/FYVE/PHD; 1.

New format:

    DR   Gene3D; 2.60.210.10; -; 1.
    DR   Gene3D; 3.30.40.10; -; 1.

### Changes to [keywords](http://www.uniprot.org/docs/keywlist)

New keyword:

-   [Corneal dystrophy](http://www.uniprot.org/keywords/KW-1212)
-   [Schizophrenia](http://www.uniprot.org/keywords/KW-1211)

Modified keywords:

-   Ionic channel -&gt; [Ion channel](http://www.uniprot.org/keywords/KW-0407)
-   Ionic channel inhibitor -&gt; [Ion channel impairing toxin](http://www.uniprot.org/keywords/KW-0872)
-   Viral ionic channel -&gt; [Viral ion channel](http://www.uniprot.org/keywords/KW-1182)
