---
title: UniProt release 2019_08
categories: Prokaryotes
---

## Headline

### Magnetic personalities

[Magnetotactic bacteria sense and align to the Earth's magnetic field](https://www.ncbi.nlm.nih.gov/pubmed/27620945,28754204), swimming north in the Northern Hemisphere and south in the Southern in the presence of oxygen. This amazing ability to sense the Earth's magnetic field is provided by small organelles, called magnetosomes, formed by iron nanocrystals of either [magnetite](https://en.wikipedia.org/wiki/Magnetite) (Fe <sub>3</sub> O <sub>4</sub> ) or [greigite](https://en.wikipedia.org/wiki/Greigite) (Fe <sub>3</sub> S <sub>4</sub> ) surrounded by a phospholipid bilayer. Generally, magnetosomes form chains that align along the long axis of the cell using a dedicated actin-like cytoskeletal structure.

![image](https://github.com/ebi-uniprot/uniprot-manual/raw/main/images/2019-09-18-release-1.jpeg)  
*Source: Frank Mickoleit [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0)*

Magnetosome formation is a complex process, which includes invagination of the cell inner membrane to form vesicles, iron ion uptake, crystal biomineralization and magnetosome chain assembly. It involves a large number of proteins, encoded by genes clustered in an approximately 100 kb magnetosome island. Among all proteins involved in magnetosome formation, one of the single most important is [MamB](http://www.uniprot.org/uniprot/?query=name%3A%22Magnetosome+protein+MamB%22+AND+reviewed:yes), a probable iron transporter with a [role in both vesicle formation and biomineralization](https://www.ncbi.nlm.nih.gov/pubmed/29243866,20212111). MamB is [stabilized by heterodimerization with MamM](https://www.ncbi.nlm.nih.gov/pubmed/22007638). Studies in genetically tractable [*Magnetospirillum magneticum* (strain AMB-1)](https://www.uniprot.org/taxonomy/342108) and [*Magnetospirillum gryphiswaldense* (strain MSR)](http://www.uniprot.org/taxonomy/431944) have pinpointed the function of many more proteins. For instance, MamA [forms a scaffold](https://www.ncbi.nlm.nih.gov/pubmed/21784982) to which other proteins attach on the organelle's exterior. MamI aids in [magnetite nucleation](https://www.ncbi.nlm.nih.gov/pubmed/20212111,27286560), while MamH is another [probable iron transporter](https://www.ncbi.nlm.nih.gov/pubmed/23889511). MamN may [control the pH of the magnetosome lumen](https://www.ncbi.nlm.nih.gov/pubmed/20212111,27286560). 4 redox-active multi-heme proteins are probably involved in correct iron oxidization ( [MamP](https://www.ncbi.nlm.nih.gov/pubmed/25775527), [MamT](https://www.ncbi.nlm.nih.gov/pubmed/23176475), [MamX](https://www.ncbi.nlm.nih.gov/pubmed/24020498) and MamE), the latter is also a protease [necessary for magnetosome protein maturation](https://www.ncbi.nlm.nih.gov/pubmed/27302060). There are proteins that positively regulate crystal size (including [MamC, MamD MamG, MamF](https://www.ncbi.nlm.nih.gov/pubmed/17965152,22716969) and those that negatively regulate crystal size ( [Mms36 and Mms48](https://www.ncbi.nlm.nih.gov/pubmed/22043287,24816605) ). Finally MamK is an actin-like protein [involved in organelle positioning](https://www.ncbi.nlm.nih.gov/pubmed/20487281,24957623), along with [MamJ](https://www.ncbi.nlm.nih.gov/pubmed/16299495,27733152) .

The interest in magnetosomes goes far beyond the understanding of these fascinating bacteria. Magnetosomes may be instrumental for the improvement of magnetic nanoparticle biotechnologies. Purified bacterial magnetosomes represent magnetic nanoparticles with exceptionally [well-defined characteristics](https://www.ncbi.nlm.nih.gov/pubmed/27620945), owing to the precise control that is exerted during all stages of biogenesis, and several unprecedented properties, such as high crystallinity, strong magnetization, and a uniform distribution of shape and size that cannot be replicated by synthesis using abiotic processes. In the biomedical field, promising results suggest that magnetosomes could be used in medical imaging, [targeted drug delivery](https://www.ncbi.nlm.nih.gov/pubmed/23043508) and [tumor hyperthermia](https://www.ncbi.nlm.nih.gov/pubmed/22698862,30168058). In the context of wastewater treatment, it has been shown that heavy metal ions can be [adsorbed onto magnetosome-producing microorganisms and then removed by magnetic separation](https://www.ncbi.nlm.nih.gov/pubmed/22763846). In addition, it may also help us learn more about the origin of life and the evolution of membrane-bound eukaryotic organelles.

As of this release nearly 70 magnetosome proteins have been annotated and can be retrieved using the term [magnetosome](http://www.uniprot.org/uniprot/?query=magnetosome+AND+reviewed:yes) .

## UniProtKB news

### Cross-references to DrugCentral

Cross-references have been added to DrugCentral, an online drug information resource providing information on active ingredients chemical entities, pharmaceutical products, drug mode of action, indications, pharmacologic action.

DrugCentral is available at <http://drugcentral.org> .

The format of the explicit links is:

|                       |                            |
|:----------------------|:---------------------------|
| Resource abbreviation | DrugCentral                |
| Resource identifier   | UniProtKB accession number |

Example: [P35372](http://www.uniprot.org/uniprot/P35372)

[Show all entries having a cross-reference to DrugCentral.](http://www.uniprot.org/uniprot/?query=database:(type:drugcentral)&sort=score)

#### Text format

Example: [P35372](http://www.uniprot.org/uniprot/P35372.txt)

    DR   DrugCentral; P35372; -.

#### XML format

Example: [P35372](http://www.uniprot.org/uniprot/P35372.xml)

    <dbReference type="DrugCentral" id="P35372"/>

#### RDF format

Example: [P35372](http://www.uniprot.org/uniprot/P35372.ttl)

    uniprot:P35372
      rdfs:seeAlso <http://purl.uniprot.org/drugcentral/P35372> .
    <http://purl.uniprot.org/drugcentral/P35372>
      rdf:type up:Resource ;
      up:database <http://purl.uniprot.org/database/DrugCentral> .

### Cross-references to Pharos

Cross-references have been added to Pharos, a user interface to the knowledge-base for the Druggable Genome (DG), whose goal is to illuminate the uncharacterized and/or poorly annotated portion of the DG, focusing on three of the most commonly drug-targeted protein families: G-protein-coupled receptors (GPCRs), ion channels (ICs) and kinases.

Pharos is available at <https://pharos.nih.gov> .

The format of the explicit links is:

|                       |                            |
|:----------------------|:---------------------------|
| Resource abbreviation | Pharos                     |
| Resource identifier   | UniProtKB accession number |

Example: [Q7Z3E2](http://www.uniprot.org/uniprot/Q7Z3E2)

[Show all entries having a cross-reference to Pharos.](http://www.uniprot.org/uniprot/?query=database:(type:pharos)&sort=score)

#### Text format

Example: [Q7Z3E2](http://www.uniprot.org/uniprot/Q7Z3E2.txt)

    DR   Pharos; Q7Z3E2; -.

#### XML format

Example: [Q7Z3E2](http://www.uniprot.org/uniprot/Q7Z3E2.xml)

    <dbReference type="Pharos" id="Q7Z3E2"/>

#### RDF format

Example: [Q7Z3E2](http://www.uniprot.org/uniprot/Q7Z3E2.ttl)

    uniprot:Q7Z3E2
      rdfs:seeAlso <http://purl.uniprot.org/pharos/Q7Z3E2> .
    <http://purl.uniprot.org/pharos/Q7Z3E2>
      rdf:type up:Resource ;
      up:database <http://purl.uniprot.org/database/Pharos> .

### Cross-references to MassIVE

Cross-references have been added to MassIVE, a community resource developed by the NIH-funded Center for Computational Mass Spectrometry to promote the global, free exchange of mass spectrometry data and provide a reusable aggregation of community-scale detection of peptides and proteins observations.

MassIVE is available at <https://massive.ucsd.edu/> .

The format of the explicit links is:

|                       |                            |
|:----------------------|:---------------------------|
| Resource abbreviation | MassIVE                    |
| Resource identifier   | UniProtKB accession number |

Example: [Q8IY92](http://www.uniprot.org/uniprot/Q8IY92)

[Show all entries having a cross-reference to MassIVE.](http://www.uniprot.org/uniprot/?query=database:(type:massive)&sort=score)

#### Text format

Example: [Q8IY92](http://www.uniprot.org/uniprot/Q8IY92.txt)

    DR   MassIVE; Q8IY92; -.

#### XML format

Example: [Q8IY92](http://www.uniprot.org/uniprot/Q8IY92.xml)

    <dbReference type="MassIVE" id="Q8IY92"/>

#### RDF format

Example: [Q8IY92](http://www.uniprot.org/uniprot/Q8IY92.ttl)

    uniprot:Q8IY92
      rdfs:seeAlso <http://purl.uniprot.org/massive/Q8IY92> .
    <http://purl.uniprot.org/massive/Q8IY92>
      rdf:type up:Resource ;
      up:database <http://purl.uniprot.org/database/MassIVE> .

### Changes to the [controlled vocabulary of human diseases](http://www.uniprot.org/docs/humdisease)

New diseases:

-   [Abnormal hair, joint laxity, and developmental delay](http://www.uniprot.org/diseases/DI-05602)
-   [Bleeding disorder, platelet-type 22](http://www.uniprot.org/diseases/DI-05589)
-   [Brain abnormalities, neurodegeneration, and dysosteosclerosis](http://www.uniprot.org/diseases/DI-05595)
-   [Calvarial doughnut lesions with bone fragility](http://www.uniprot.org/diseases/DI-05600)
-   [Calvarial doughnut lesions with bone fragility and spondylometaphyseal dysplasia](http://www.uniprot.org/diseases/DI-05601)
-   [Cerebellar, ocular, craniofacial, and genital syndrome](http://www.uniprot.org/diseases/DI-05597)
-   [Contractures, pterygia, and variable skeletal fusions syndrome 1B](http://www.uniprot.org/diseases/DI-05594)
-   [Deafness, autosomal recessive, 114](http://www.uniprot.org/diseases/DI-05583)
-   [Deafness, autosomal recessive, 115](http://www.uniprot.org/diseases/DI-05584)
-   [Deafness, autosomal recessive, 99](http://www.uniprot.org/diseases/DI-05585)
-   [Developmental delay with or without dysmorphic facies and autism](http://www.uniprot.org/diseases/DI-05586)
-   [Encephalopathy, acute, infection-induced, 9](http://www.uniprot.org/diseases/DI-05581)
-   [Epileptic encephalopathy, early infantile, 76](http://www.uniprot.org/diseases/DI-05592)
-   [Generalized epilepsy with febrile seizures plus 10](http://www.uniprot.org/diseases/DI-05599)
-   [Immunodeficiency 62](http://www.uniprot.org/diseases/DI-05587)
-   [Intellectual developmental disorder with severe speech and ambulation defects](http://www.uniprot.org/diseases/DI-05593)
-   [Kahn-Kahn-Katsanis syndrome](http://www.uniprot.org/diseases/DI-05588)
-   [Keipert syndrome](http://www.uniprot.org/diseases/DI-05580)
-   [Long QT syndrome 8](http://www.uniprot.org/diseases/DI-05582)
-   [Neurodevelopmental disorder with seizures and speech and walking impairment](http://www.uniprot.org/diseases/DI-05598)
-   [Paganini-Miozzo syndrome](http://www.uniprot.org/diseases/DI-05579)
-   [Paragangliomas 6](http://www.uniprot.org/diseases/DI-05590)
-   [Paragangliomas 7](http://www.uniprot.org/diseases/DI-05591)
-   [Uridine-cytidineuria](http://www.uniprot.org/diseases/DI-05596)

### Changes in [subcellular location controlled vocabulary](http://www.uniprot.org/docs/subcell)

New subcellular locations:

-   [Photoreceptor inner segment membrane](http://www.uniprot.org/locations/SL-0514)
-   [Photoreceptor outer segment membrane](http://www.uniprot.org/locations/SL-0513)

Modified subcellular location:

-   Exosome -&gt; [Extracellular exosome](http://www.uniprot.org/locations/SL-0466)

## UniRef news

### Change of UniRef clustering method from CD-HIT to MMseqs2

We have switched the clustering program for UniRef90 and UniRef50 from CD-HIT to MMseqs2 ( [Steinegger M. and Soeding J., Nat. Commun. 9 (2018)](https://doi.org/10.1038/s41467-018-04964-5) ).

The clustering algorithm remains "Greedy Incremental Clustering" with the same parameters (thanks to the MMseqs2 authors for making this available). UniRef100 was not affected.

## UniProt XML news

### Removal of whitespace characters in the XML amino acid sequence representations

The `<sequence>` elements of the UniProtKB, UniParc and UniRef XML representations formatted the amino acid sequence for historic reasons with spaces and newlines. These whitespace characters had to be removed before parsing with native XML tools. To avoid this complication we have removed all whitespace characters in the `<sequence>` elements, so that they contain only [IUPAC](https://iupac.org/) amino acid codes.
