---
title: UniProt release 2020_02
categories: eukaryota,Prokaryotes
---

<h2 id="Headline">
 Headline
</h2>
<h3 id="Genome_integrity_maintenance_by_HMCES">
 Genome integrity maintenance by HMCES
</h3>
<p>
 Apurinic or apyrimidinic sites, also known as abasic or AP sites, are one of the most common DNA lesions. They occur at a frequency of
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/2444877">
  about 15,000 per day in human cells
 </a>
 . In double-stranded DNA, the majority of AP sites are removed by base excision repair. After removal of the lesion, the undamaged strand is used as a template for repair synthesis. AP sites also form in single-stranded DNA (ssDNA), but until recently there was no known mechanism involved in their repair in this context. A major breakthrough in the field was reported last year in
 <em>
  Cell
 </em>
 .
</p>
<p>
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/30554877">
  Mohni et al.
 </a>
 were interested in
 <a href="http://www.uniprot.org/uniprot/Q96FZ2">
  HMCES
 </a>
 . HMCES full name is 'stem cell-specific 5-hydroxymethylcytosine-binding protein'. It was originally thought to be a regulator of 5-hydroxymethylcytosine. However, it had also been
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/26365379">
  identified in the replisome
 </a>
 , a large protein machine that carries out DNA replication.
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/23945014">
  HMCES is conserved in almost all organisms
 </a>
 , even in those that do not utilize methylcytosine for epigenetic control. Taken together, these observations suggested that HMCES could bear another crucial function, possibly in replication. Surprisingly HMCES knockout in cells did not affect DNA replication, nor cell division, but rather
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/30554877">
  exacerbated cell sensitivity toward several DNA-damaging agents
 </a>
 . Knockout cells accumulated DNA damage and exhibited increased genetic instability. Different DNA-damaging agents were tested and the only common kind of lesion they induced was the formation of AP sites.
</p>
<p>
 HMCES appears to act as the initiating step of a replication-coupled repair mechanism for abasic sites in ssDNA. In eukaryotic cells, HMCES interacts with proliferating cell nuclear antigen (PCNA), an essential factor for replication, and travels with replication forks. When it senses AP sites in ssDNA, it covalently crosslinks to ssDNA AP sites generating a DNA-protein intermediate. The
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/31235913,31235915,31504793">
  nature of this crosslink has been identified by crystallographic studies
 </a>
 as a stable
 <a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:145793">
  thiazolidine DNA-protein linkage
 </a>
 formed between the N-terminal cysteine and the aldehyde form of the AP deoxyribose. The crosslink is so stable that its resolution requires HMCES degradation via the proteasome. This sequence of events may appear counterintuitive. It is almost as if HMCES takes a bad situation and makes it worse. However, this crosslink effectively shields the lesion from endonucleases and error-prone trans-lesion bypass (TLS) polymerases, such as REV1 and REV3L, and prevents mutagenesis they might engender. The DNA repair mechanism acting downstream of HMCES is not known.
</p>
<p>
 As of this release, human
 <a href="http://www.uniprot.org/uniprot/Q96FZ2">
  HMCES
 </a>
 , as well as
 <a href="http://www.uniprot.org/uniprot/P76318">
  YedK
 </a>
 , an
 <em>
  Escherichia coli
 </em>
 homolog have been updated and are available in UniProtKB/Swiss-Prot. The exact structure of the chemical crosslink was submitted to
 <a href="https://www.ebi.ac.uk/chebi">
  ChEBI
 </a>
 where more details are provided.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Change_of_annotation_topic__Interaction_">
 Change of annotation topic 'Interaction'
</h3>
<p>
 The annotation topic 'Interaction' provides information about
 <a href="http://www.uniprot.org/help/binary%5Finteractions">
  binary protein-protein interactions
 </a>
 . This data is curated in the
 <a href="https://www.ebi.ac.uk/intact">
  IntAct
 </a>
 database and a quality-filtered subset is imported into UniProtKB at each release.
</p>
<p>
 In the context of improving the
 <a href="http://www.uniprot.org/news/2019/12/18/release#functional%5Fannotation%5Fof%5Fdifferent%5Fgene%5Fproducts%5Fin%5Funiprotkb%5Fswiss%5Fprot">
  functional annotation of different gene products in UniProtKB/Swiss-Prot
 </a>
 , we have started to import more detailed data from IntAct. Our previous representation of a binary protein-protein interaction provided details only for the protein that was described in another entry. This left ambiguity in UniProtKB/Swiss-Prot entries that describe more than one protein (
 <a href="http://www.uniprot.org/help/alternative%5Fproducts">
  isoforms
 </a>
 or/and products of proteolytic cleavage). To address this we now describe both interacting proteins by unique UniProtKB identifiers.
</p>
<p>
 This change affects the three main UniProtKB distribution formats (text, XML, RDF). The details are described for each format in a separate section below. The following placeholders are used in the format descriptions:
</p>
<ul>
 <li>
  <code>
   &lt;Interactant&gt;
  </code>
  represents a UniProtKB protein.
  <ul>
   <li>
    <code>
     &lt;Accession&gt;
    </code>
    is a UniProtKB
    <a href="http://www.uniprot.org/help/accession%5Fnumbers">
     accession number
    </a>
    .
   </li>
   <li>
    <code>
     &lt;IsoId&gt;
    </code>
    is a UniProtKB
    <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fid">
     isoform ID
    </a>
    .
   </li>
   <li>
    <code>
     &lt;ProductId&gt;
    </code>
    is a UniProtKB
    <a href="http://www.uniprot.org/help/sequence%5Fannotation#annotation%5Fid">
     product ID
    </a>
    .
   </li>
   <li>
    <code>
     &lt;Gene&gt;
    </code>
    is either the gene name, ordered locus name or ORF name of the gene that encodes the UniProtKB protein (see
    <a href="http://www.uniprot.org/help/gene%5Fname">
     Gene names
    </a>
    ).
   </li>
  </ul>
 </li>
</ul>
<ul>
 <li>
  <code>
   &lt;Experiments&gt;
  </code>
  is the number of experiments in IntAct that support an interaction.
 </li>
</ul>
<ul>
 <li>
  <code>
   &lt;IntActId&gt;
  </code>
  is an IntAct protein ID.
 </li>
</ul>
<p>
 Note: The format descriptions make use of
 <a href="https://en.wikibooks.org/wiki/Regular%5FExpressions/POSIX-Extended%5FRegular%5FExpressions">
  POSIX ERE
 </a>
 syntax.
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 <a href="http://www.uniprot.org/news/2005/02/01/release">
  Previous format
 </a>
 :
</p>
<pre>CC   -!- INTERACTION:
CC       <code style="color: grey;">&lt;Interactant&gt;</code>( \(xeno\))?; NbExp=<code style="color: grey;">&lt;Experiments&gt;</code>; IntAct=<code style="color: grey;">&lt;IntActId&gt;</code>, <code style="color: grey;">&lt;IntActId&gt;</code>;
CC       <code style="color: grey;">&lt;Interactant&gt;</code>( \(xeno\))?; NbExp=<code style="color: grey;">&lt;Experiments&gt;</code>; IntAct=<code style="color: grey;">&lt;IntActId&gt;</code>, <code style="color: grey;">&lt;IntActId&gt;</code>;
CC       ...
</pre>
<p>
 The
 <code>
  &lt;Interactant&gt;
 </code>
 was described in the following way:
</p>
<pre>Self|(<code style="color: grey;">&lt;Accession&gt;</code>|<code style="color: grey;">&lt;IsoId&gt;</code>):(<code style="color: grey;">&lt;Gene&gt;</code>|-)
</pre>
<p>
 Where
 <code>
  Self
 </code>
 represents a self-interaction and a dash is shown for proteins with an undefined
 <code>
  &lt;Gene&gt;
 </code>
 .
 <code>
  xeno
 </code>
 is an optional flag that indicates that the interacting proteins are derived from different species. This may be due to the experimental set-up or may reflect a pathogen-host interaction.
</p>
<p>
 New format:
</p>
<pre>CC   -!- INTERACTION:
CC       <code style="color: grey;">&lt;Interactant&gt;</code>; <code style="color: grey;">&lt;Interactant&gt;</code>;( Xeno;)? NbExp=<code style="color: grey;">&lt;Experiments&gt;</code>; IntAct=<code style="color: grey;">&lt;IntActId&gt;</code>, <code style="color: grey;">&lt;IntActId&gt;</code>;
CC       <code style="color: grey;">&lt;Interactant&gt;</code>; <code style="color: grey;">&lt;Interactant&gt;</code>;( Xeno;)? NbExp=<code style="color: grey;">&lt;Experiments&gt;</code>; IntAct=<code style="color: grey;">&lt;IntActId&gt;</code>, <code style="color: grey;">&lt;IntActId&gt;</code>;
CC       ...
</pre>
<p>
 Where
</p>
<ul>
 <li>
  the first
  <code>
   &lt;Interactant&gt;
  </code>
  is represented by:
  <pre>(<code style="color: grey;">&lt;Accession&gt;</code>|<code style="color: grey;">&lt;IsoId&gt;</code>|<code style="color: grey;">&lt;ProductId&gt;</code>)
</pre>
 </li>
 <li>
  the second
  <code>
   &lt;Interactant&gt;
  </code>
  is represented by:
  <pre>(<code style="color: grey;">&lt;Accession&gt;</code>|<code style="color: grey;">&lt;IsoId&gt;</code>|<code style="color: grey;">&lt;ProductId&gt;</code> [<code style="color: grey;">&lt;Accession&gt;</code>])(: <code style="color: grey;">&lt;Gene&gt;</code>)?
</pre>
 </li>
</ul>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P11309.txt">
  P11309
 </a>
</p>
<p>
 <em>
  Binary interactions with different isoforms that are described in P11309.
 </em>
</p>
<p>
 Previous format:
</p>
<pre>CC   -!- INTERACTION:
CC       Q9BZS1-1:FOXP3; NbExp=3; IntAct=EBI-1018629, EBI-9695448;
CC       Q9UNQ0:ABCG2; NbExp=5; IntAct=EBI-1018633, EBI-1569435;
</pre>
<p>
 New format:
</p>
<pre>CC   -!- INTERACTION:
CC       P11309-1; Q9BZS1-1: FOXP3; NbExp=3; IntAct=EBI-1018629, EBI-9695448;
CC       P11309-2; Q9UNQ0: ABCG2; NbExp=5; IntAct=EBI-1018633, EBI-1569435;
</pre>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P27958.txt">
  P27958
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/Q9NPY3.txt">
  Q9NPY3
 </a>
</p>
<p>
 <em>
  Binary interaction with a product of proteolytic cleavage. Interactions involving products of proteolytic cleavage were previously not imported from IntAct, therefore only the new data/format is shown.
 </em>
</p>
<p>
 New data and format of P27958:
</p>
<pre>CC   -!- INTERACTION:
CC       PRO_0000037566; Q9NPY3: CD93; Xeno; NbExp=2; IntAct=EBI-6377335, EBI-1755002;
</pre>
<p>
 New data and format of Q9NPY3:
</p>
<pre>CC   -!- INTERACTION:
CC       Q9NPY3; PRO_0000037566 [P27958]; Xeno; NbExp=2; IntAct=EBI-1755002, EBI-6377335;
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 The
 <a href="http://www.uniprot.org/docs/uniprot.xsd">
  UniProtKB XSD
 </a>
 represents a binary interaction with:
</p>
<ul>
 <li>
  two
  <code>
   interactant
  </code>
  elements of
  <code>
   interactantType
  </code>
 </li>
</ul>
<ul>
 <li>
  a boolean
  <code>
   organismsDiffer
  </code>
  element that indicates that the interacting proteins are derived from different species. This may be due to the experimental set-up or may reflect a pathogen-host interaction.
 </li>
</ul>
<ul>
 <li>
  an
  <code>
   experiments
  </code>
  element that gives the number of experiments in IntAct that support an interaction.
 </li>
</ul>
<p>
 The
 <code>
  interactantType
 </code>
 uses an
 <code>
  interactantGroup
 </code>
 to represent a sequence of:
</p>
<ul>
 <li>
  an
  <code>
   id
  </code>
  element
 </li>
</ul>
<ul>
 <li>
  an optional
  <code>
   label
  </code>
  element
 </li>
</ul>
<p>
 We have added an optional
 <code>
  dbReference
 </code>
 element to the
 <code>
  interactantGroup
 </code>
 to allow us to represent the UniProtKB
 <code>
  &lt;Accession&gt;
 </code>
 for a
 <code>
  &lt;ProductId&gt;
 </code>
 :
</p>
<pre>&lt;xs:group name="interactantGroup"&gt;
        &lt;xs:sequence&gt;
            &lt;xs:element name="id" type="xs:string"/&gt;
            &lt;xs:element name="label" type="xs:string" minOccurs="0"/&gt;
            <code style="color: red;">&lt;xs:element name="dbReference" type="dbReferenceType" minOccurs="0"/&gt;</code>
        &lt;/xs:sequence&gt;
    &lt;/xs:group&gt;

</pre>
<p>
 Previous format:
</p>
<pre>&lt;comment type="interaction"&gt;
  &lt;interactant intactId="<code style="color: grey;">&lt;IntActId&gt;</code>"/&gt;
  &lt;interactant intactId="<code style="color: grey;">&lt;IntActId&gt;</code>"&gt;
    &lt;id&gt;<code style="color: grey;">&lt;Accession&gt;</code>|<code style="color: grey;">&lt;IsoId&gt;</code>&lt;/id&gt;
    &lt;label&gt;<code style="color: grey;">&lt;Gene&gt;</code>&lt;/label&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;<code style="color: grey;">true</code>|<code style="color: grey;">false</code>&lt;/organismsDiffer&gt;
  &lt;experiments&gt;<code style="color: grey;">&lt;Experiments&gt;</code>&lt;/experiments&gt;
&lt;/comment&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;comment type="interaction"&gt;
  &lt;interactant intactId="<code style="color: grey;">&lt;IntActId&gt;</code>"&gt;
    &lt;id&gt;<code style="color: grey;">&lt;Accession&gt;</code>|<code style="color: grey;">&lt;IsoId&gt;</code>|<code style="color: grey;">&lt;ProductId&gt;</code>&lt;/id&gt;
  &lt;/interactant&gt;
  &lt;interactant intactId="<code style="color: grey;">&lt;IntActId&gt;</code>"&gt;
    &lt;id&gt;<code style="color: grey;">&lt;Accession&gt;</code>|<code style="color: grey;">&lt;IsoId&gt;</code>|<code style="color: grey;">&lt;ProductId&gt;</code>&lt;/id&gt;
    &lt;label&gt;<code style="color: grey;">&lt;Gene&gt;</code>&lt;/label&gt;
    <code style="color: red;">&lt;!-- If &lt;id&gt; is a <code style="color: grey;">&lt;ProductId&gt;</code>: --&gt;</code>
    <code style="color: red;">&lt;dbReference type="UniProtKB" id="</code><code style="color: grey;">&lt;Accession&gt;</code><code style="color: red;">"/&gt;</code>
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;<code style="color: grey;">true</code>|<code style="color: grey;">false</code>&lt;/organismsDiffer&gt;
  &lt;experiments&gt;<code style="color: grey;">&lt;Experiments&gt;</code>&lt;/experiments&gt;
&lt;/comment&gt;
</pre>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P11309.xml">
  P11309
 </a>
</p>
<p>
 <em>
  Binary interactions with different isoforms that are described in P11309.
 </em>
</p>
<p>
 Previous format:
</p>
<pre>&lt;comment type="interaction"&gt;
  &lt;interactant intactId="EBI-1018629"/&gt;
  &lt;interactant intactId="EBI-9695448"&gt;
    &lt;id&gt;Q9BZS1-1&lt;/id&gt;
    &lt;label&gt;FOXP3&lt;/label&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;false&lt;/organismsDiffer&gt;
  &lt;experiments&gt;3&lt;/experiments&gt;
&lt;/comment&gt;
&lt;comment type="interaction"&gt;
  &lt;interactant intactId="EBI-1018633"/&gt;
  &lt;interactant intactId="EBI-1569435"&gt;
    &lt;id&gt;Q9UNQ0&lt;/id&gt;
    &lt;label&gt;ABCG2&lt;/label&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;false&lt;/organismsDiffer&gt;
  &lt;experiments&gt;5&lt;/experiments&gt;
&lt;/comment&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;comment type="interaction"&gt;
  &lt;interactant intactId="EBI-1018629"&gt;
    &lt;id&gt;P11309-1&lt;/id&gt;
  &lt;/interactant&gt;
  &lt;interactant intactId="EBI-9695448"&gt;
    &lt;id&gt;Q9BZS1-1&lt;/id&gt;
    &lt;label&gt;FOXP3&lt;/label&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;false&lt;/organismsDiffer&gt;
  &lt;experiments&gt;3&lt;/experiments&gt;
&lt;/comment&gt;
&lt;comment type="interaction"&gt;
  &lt;interactant intactId="EBI-1018633"&gt;
    &lt;id&gt;P11309-2&lt;/id&gt;
  &lt;/interactant&gt;
  &lt;interactant intactId="EBI-1569435"&gt;
    &lt;id&gt;Q9UNQ0&lt;/id&gt;
    &lt;label&gt;ABCG2&lt;/label&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;false&lt;/organismsDiffer&gt;
  &lt;experiments&gt;5&lt;/experiments&gt;
&lt;/comment&gt;
</pre>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P27958.xml">
  P27958
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/Q9NPY3.xml">
  Q9NPY3
 </a>
</p>
<p>
 <em>
  Binary interaction with a product of proteolytic cleavage. Interactions involving products of proteolytic cleavage had previously not been imported from IntAct, therefore only the new data/format is shown.
 </em>
</p>
<p>
 New data and format of P27958:
</p>
<pre>&lt;comment type="interaction"&gt;
  &lt;interactant intactId="EBI-6377335"&gt;
    &lt;id&gt;PRO_0000037566&lt;/id&gt;
  &lt;/interactant&gt;
  &lt;interactant intactId="EBI-1755002"&gt;
    &lt;id&gt;Q9NPY3&lt;/id&gt;
    &lt;label&gt;CD93&lt;/label&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;true&lt;/organismsDiffer&gt;
  &lt;experiments&gt;2&lt;/experiments&gt;
&lt;/comment&gt;
</pre>
<p>
 New data and format of Q9NPY3:
</p>
<pre>&lt;comment type="interaction"&gt;
  &lt;interactant intactId="EBI-1755002"&gt;
    &lt;id&gt;Q9NPY3&lt;/id&gt;
  &lt;/interactant&gt;
  &lt;interactant intactId="EBI-6377335"&gt;
    &lt;id&gt;PRO_0000037566&lt;/id&gt;
    &lt;dbReference type="UniProtKB" id="P27958"/&gt;
  &lt;/interactant&gt;
  &lt;organismsDiffer&gt;true&lt;/organismsDiffer&gt;
  &lt;experiments&gt;2&lt;/experiments&gt;
&lt;/comment&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 The
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/rdf/core.owl">
  UniProt RDF schema ontology
 </a>
 represents a binary interaction with an
 <a href="http://purl.uniprot.org/core/interaction">
  <code>
   interaction
  </code>
 </a>
 property whose
 <code>
  rdfs:range
 </code>
 is the
 <a href="http://purl.uniprot.org/core/Interaction">
  <code>
   Interaction
  </code>
 </a>
 class. This class is the domain of the following properties that describe the interaction:
</p>
<ul>
 <li>
  <a href="http://purl.uniprot.org/core/participant">
   <code>
    participant
   </code>
  </a>
  refers to an object of type
  <a href="http://purl.uniprot.org/core/Participant">
   <code>
    Participant
   </code>
  </a>
  .
 </li>
</ul>
<ul>
 <li>
  <a href="http://purl.uniprot.org/core/xeno">
   <code>
    xeno
   </code>
  </a>
  is a boolean that indicates that the interacting proteins are derived from different species. This may be due to the experimental set-up or may reflect a pathogen-host interaction.
 </li>
</ul>
<ul>
 <li>
  <a href="http://purl.uniprot.org/core/experiments">
   <code>
    experiments
   </code>
  </a>
  gives the number of experiments in IntAct that support an interaction.
 </li>
</ul>
<p>
 A
 <a href="http://purl.uniprot.org/core/Participant">
  <code>
   Participant
  </code>
 </a>
 is identified by its unique IntAct identifier. It also refers to the corresponding UniProtKB protein which is represented as described in the news article about the
 <a href="http://www.uniprot.org/news/2019/12/18/release#rdf%5Fproduct">
  functional annotation of different gene products in UniProtKB/Swiss-Prot
 </a>
 . An optional
 <code>
  rdfs:label
 </code>
 property may provide the gene name, ordered locus name or ORF name of the gene that encodes the UniProtKB protein.
</p>
<p>
 The RDF schema ontology required no changes to represent the more detailed data that we now import from IntAct. Due to the symmetry of binary interactions, the
 <a href="https://sparql.uniprot.org/">
  UniProt SPARQL server
 </a>
 already provided access to the full details about both interacting proteins. We have however taken this opportunity to normalize the URI of a binary interaction so that the two UniProtKB entries that describe the interacting proteins refer to the interaction with the same URI:
</p>
<p>
 Previous format:
</p>
<pre>&lt;<code style="color: grey;">&lt;Accession&gt;</code>#interaction-<code style="color: grey;">&lt;IntActId&gt;</code>-<code style="color: grey;">&lt;IntActId&gt;</code>&gt; .
</pre>
<p>
 New format:
</p>
<pre>&lt;http://purl.uniprot.org/intact/<code style="color: grey;">&lt;IntActId&gt;</code>-<code style="color: grey;">&lt;IntActId&gt;</code>&gt; .
</pre>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P11309.ttl">
  P11309
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/Q8N9N5.ttl">
  Q8N9N5
 </a>
</p>
<p>
 Previous format:
</p>
<p>
 P11309:
</p>
<pre>&lt;P11309#interaction-696621-744695&gt;
</pre>
<p>
 Q8N9N5:
</p>
<pre>&lt;Q8N9N5#interaction-744695-696621&gt;
</pre>
<p>
 New format:
</p>
<p>
 P11309 and Q8N9N5:
</p>
<pre>&lt;http://purl.uniprot.org/intact/EBI-696621-EBI-744695&gt;
</pre>
<h3 id="Cross_references_to_Antibodypedia">
 Cross-references to Antibodypedia
</h3>
<p>
 Cross-references have been added to Antibodypedia, a portal providing access to publicly available research antibodies towards human protein targets from many different providers.
</p>
<p>
 Antibodypedia is available at
 <a href="https://www.antibodypedia.com/">
  https://www.antibodypedia.com/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   Antibodypedia
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Resource identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Number of antibodies
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P04626">
  P04626
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:(type:antibodypedia)&amp;sort=score">
  Show all entries having a cross-reference to Antibodypedia.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P04626.txt">
  P04626
 </a>
</p>
<pre>DR   Antibodypedia; 740; 5394 antibodies.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P04626.xml">
  P04626
 </a>
</p>
<pre>&lt;dbReference type="Antibodypedia" id="740"&gt;
   &lt;property type="antibodies" value="5394 antibodies"/&gt;
&lt;/dbReference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P04626.ttl">
  P04626
 </a>
</p>
<pre>uniprot:P04626
  rdfs:seeAlso &lt;http://purl.uniprot.org/antibodypedia/740&gt; .

&lt;http://purl.uniprot.org/antibodypedia/740&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/Antibodypedia&gt; ;
  rdfs:comment "5394 antibodies" .
</pre>
<h3 id="Cross_references_to_MetOSite">
 Cross-references to MetOSite
</h3>
<p>
 Cross-references have been added to MetOSite, a database of methionine sulfoxide sites. Each collected site has been classified according to the effect of its sulfoxidation on the biological properties of the modified protein. Thus, MetOSite documents cases where the sulfoxidation of methionine leads to gain or loss of activity, increased or decreased protein-protein interaction susceptibility, and to changes in protein stability or in subcellular location.
</p>
<p>
 MetOSite is available at
 <a href="https://metosite.uma.es/">
  https://metosite.uma.es/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   MetOSite
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10987">
  P10987
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:(type:metosite)&amp;sort=score">
  Show all entries having a cross-reference to MetOSite.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10987.txt">
  P10987
 </a>
</p>
<pre>DR   MetOSite; P10987; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10987.xml">
  P10987
 </a>
</p>
<pre>&lt;dbReference type="MetOSite" id="P10987"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10987.ttl">
  P10987
 </a>
</p>
<pre>uniprot:P10987
  rdfs:seeAlso &lt;http://purl.uniprot.org/metosite/P10987&gt; .
&lt;http://purl.uniprot.org/metosite/P10987&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/MetOSite&gt; .
</pre>
<h3 id="Cross_references_to_PHI_base">
 Cross-references to PHI-base
</h3>
<p>
 Cross-references have been added to PHI-base, a database providing expertly curated molecular and biological information on genes proven to affect the outcome of pathogen-host interactions.
</p>
<p>
 PHI-base is available at
 <a href="http://www.phi-base.org/">
  http://www.phi-base.org/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   PHI-base
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Resource identifier
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q00310">
  Q00310
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:(type:phi-base)&amp;sort=score">
  Show all entries having a cross-reference to PHI-base.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q00310.txt">
  Q00310
 </a>
</p>
<pre>DR   PHI-base; PHI:104; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q00310.xml">
  Q00310
 </a>
</p>
<pre>&lt;dbReference type="PHI-base" id="PHI:104"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q00310.ttl">
  Q00310
 </a>
</p>
<pre>uniprot:Q00310
  rdfs:seeAlso &lt;http://purl.uniprot.org/phi-base/PHI:104&gt; .
&lt;http://purl.uniprot.org/phi-base/PHI:104&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/PHI-base&gt; .
</pre>
<h3 id="Change_to_the_cross_references_to_Human_Protein_Atlas__HPA_">
 Change to the cross-references to Human Protein Atlas (HPA)
</h3>
<p>
 We have changed the way we present the
 <a href="https://proteinatlas.org">
  Human Protein Atlas
 </a>
 database cross-references. Links between UniProtKB entries and HPA used to be established by HPA antibody identifier, but are now based on Ensembl Gene identifiers.
</p>
<p>
 We have also introduced an additional field in these cross-references to indicate the level of RNA tissue specificity. The RNA specificity category is based on mRNA expression levels in the analyzed samples. The categories include: 'Tissue enriched', 'Group enriched', 'Tissue enhanced', 'Low tissue specificity' and 'Not detected'. For more details on these categories, see the
 <a href="https://www.proteinatlas.org/about/assays+annotation#classification%5Frna">
  Classification of transcriptomics data
 </a>
 by Human Protein Atlas.
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9NSG2.txt">
  Q9NSG2
 </a>
</p>
<p>
 Previous format:
</p>
<pre>DR   HPA; HPA023778; -.
DR   HPA; HPA024451; -.
</pre>
<p>
 New format:
</p>
<pre>DR   HPA; ENSG00000000460; Tissue enhanced (lymphoid).
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9NSG2.xml">
  Q9NSG2
 </a>
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="HPA" id="HPA023778"/&gt;
&lt;dbReference type="HPA" id="HPA024451"/&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="HPA" id="ENSG00000000460"&gt;
  &lt;property type="expression patterns" value="Tissue enhanced (lymphoid)"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change does not affect the XSD, but may nevertheless require code changes.
</p>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9NSG2.ttl">
  Q9NSG2
 </a>
</p>
<p>
 Previous format:
</p>
<pre>uniprot:Q9NSG2
  rdfs:seeAlso &lt;http://purl.uniprot.org/hpa/HPA023778&gt; ,
               &lt;http://purl.uniprot.org/hpa/HPA024451&gt; .
&lt;http://purl.uniprot.org/hpa/HPA023778&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/HPA&gt; .
&lt;http://purl.uniprot.org/hpa/HPA024451&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/HPA&gt; .
</pre>
<p>
 New format:
</p>
<pre>uniprot:Q9NSG2
  rdfs:seeAlso &lt;http://www.proteinatlas.org/ENSG00000000460&gt; .
&lt;http://www.proteinatlas.org/ENSG00000000460&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/HPA&gt; ;
  rdfs:comment "Tissue enhanced (lymphoid)" .
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05714">
   Ciliary dyskinesia, primary, 42
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05715">
   Ciliary dyskinesia, primary, 43
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05696">
   Epileptic encephalopathy, early infantile, 81
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05722">
   Epileptic encephalopathy, early infantile, 82
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05704">
   Hemolytic anemia due to glutathione reductase deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05706">
   Hydrocephalus, congenital communicating, 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05705">
   Intellectual developmental disorder, autosomal recessive 72
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05699">
   Intellectual developmental disorder with impaired language and dysmorphic facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05712">
   Intellectual developmental disorder with short stature and behavioral abnormalities
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05707">
   Intellectual developmental disorder with speech delay, autism and dysmorphic facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05713">
   Leukodystrophy, hypomyelinating, 19, transient infantile
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05721">
   Megabladder, congenital
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05709">
   Megacystis-microcolon-intestinal hypoperistalsis syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05725">
   Microphthalmia, syndromic, 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05711">
   Mitochondrial complex V deficiency, nuclear type 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05700">
   Myopathy, congenital, with structured cores and Z-line abnormalities
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05701">
   Myopathy, distal, 6, adult onset, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05718">
   Neurodevelopmental disorder with absent language and variable seizures
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05720">
   Neurodevelopmental disorder with behavioral abnormalities, absent speech, and hypotonia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05703">
   Neurodevelopmental disorder with dysmorphic facies and distal skeletal anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05723">
   Neurodevelopmental disorder with macrocephaly and with or without seizures
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05719">
   Neurodevelopmental disorder with non-specific brain abnormalities and with or without seizures
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05697">
   Neurodevelopmental disorder with spastic quadriplegia, optic atrophy, seizures, and structural brain anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05726">
   Neuronal intranuclear inclusion disease
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05698">
   Neurooculocardiogenitourinary syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05724">
   Premature ovarian failure 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05708">
   Pulmonary fibrosis, and/or bone marrow failure, telomere-related, 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05716">
   Retinitis pigmentosa 87 with choroidal involvement
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05717">
   Short stature and microcephaly with genital anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05695">
   Sitosterolemia 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05710">
   Spondyloepimetaphyseal dysplasia, X-linked, with hypomyelinating leukodystrophy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05702">
   Zimmermann-Laband syndrome 3
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  3-methylglutaconic aciduria with cataracts, neurologic involvement and neutropenia -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04365">
   3-methylglutaconic aciduria 7
  </a>
 </li>
 <li>
  Cowchock syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03693">
   Charcot-Marie-Tooth disease, X-linked recessive, 4, with or without cerebellar ataxia
  </a>
 </li>
 <li>
  Developmental delay, intellectual disability, obesity, and dysmorphic features -&gt;
  <a href="http://www.uniprot.org/diseases/DI-05259">
   Chung-Jansen syndrome
  </a>
 </li>
 <li>
  Developmental delay with short stature, dysmorphic features, and sparse hair -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04703">
   Developmental delay with short stature, dysmorphic facial features, and sparse hair
  </a>
 </li>
 <li>
  Ischiocoxopodopatellar syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02312">
   Ischiocoxopodopatellar syndrome with or without pulmonary arterial hypertension
  </a>
 </li>
 <li>
  Mental retardation, X-linked, syndromic, 36 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-05649">
   Basilicata-Akhtar syndrome
  </a>
 </li>
 <li>
  Metabolic encephalomyopathic crises, recurrent, with rhabdomyolysis, cardiac arrhythmias, and neurodegeneration -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04674">
   Metabolic crises, recurrent, with rhabdomyolysis, cardiac arrhythmias, and neurodegeneration
  </a>
 </li>
 <li>
  Neuropathy, hereditary motor and sensory, 6A -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00292">
   Neuropathy, hereditary motor and sensory, 6A, with optic atrophy
  </a>
 </li>
 <li>
  Seizures, scoliosis, and macrocephaly syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04595">
   Seizures, scoliosis, and macrocephaly/microcephaly syndrome
  </a>
 </li>
 <li>
  Spondyloepimetaphyseal dysplasia MATN3-related -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02330">
   Spondyloepimetaphyseal dysplasia, MATN3-related
  </a>
 </li>
 <li>
  Spondyloepimetaphyseal dysplasia Missouri type -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02332">
   Spondyloepimetaphyseal dysplasia, Missouri type
  </a>
 </li>
 <li>
  Spondyloepimetaphyseal dysplasia short limb-hand type -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02538">
   Spondyloepimetaphyseal dysplasia, short limb-hand type
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  Thiazolidine linkage to a ring-opened DNA abasic site
 </li>
 <li>
  Deoxyhypusine
 </li>
</ul>
<h2 id="RDF_news">
 RDF news
</h2>
<h3 id="Change_of_URIs_for_the_Human_Protein_Atlas__HPA__database">
 Change of URIs for the Human Protein Atlas (HPA) database
</h3>
<p>
 For historic reasons, UniProt had to generate URIs to cross-reference databases that did not have an RDF representation. Our policy is to replace these by the URIs generated by the cross-referenced database once it starts to distribute an RDF representation of its data.
</p>
<p>
 The URIs for the
 <a href="https://www.proteinatlas.org/">
  Human Protein Atlas
 </a>
 database have therefore been updated from:
</p>
<pre>http://purl.uniprot.org/hpa/&lt;ID&gt;
</pre>
<p>
 to:
</p>
<pre>http://www.proteinatlas.org/&lt;ID&gt;
</pre>
<p>
 If required for backward compatibility, you will be able to use the following query to add the old URIs:
</p>
<pre>PREFIX owl:&lt;http://www.w3.org/2002/07/owl#&gt;
PREFIX up:&lt;http://purl.uniprot.org/core/&gt;
INSERT
{
   ?protein rdfs:seeAlso ?old .
   ?old owl:sameAs ?new .
   ?old up:database &lt;http://purl.uniprot.org/database/HPA&gt; .
}
WHERE
{
   ?protein rdfs:seeAlso ?new .
   ?new up:database &lt;http://purl.uniprot.org/database/HPA&gt; .
   BIND(iri(concat('http://purl.uniprot.org/hpa/', substr(str(?new),29))) AS ?old)
}
</pre>
<p>
 The dereferencing of existing http://purl.uniprot.org/hpa/&lt;ID&gt; URIs will be maintained.
</p>
<h3 id="Standardized_MD5_checksums_in_UniProt_RDF">
 Standardized MD5 checksums in UniProt RDF
</h3>
<p>
 The UniProt databases UniProtKB, UniRef and UniParc have historically provided a CRC-64 checksum for the amino acid sequences. In the UniParc RDF representation we had already introduced an MD5 checksum, and we have now replaced it with a SPARQL 1.1 compliant MD5 representation (lowercase string) and use this across all databases. This allows to use the
 <a href="https://www.w3.org/TR/sparql11-query/#func-md5">
  MD5 function defined in SPARQL 1.1
 </a>
 to check that the sequence string is not corrupted, without the need to use the lowercase (LCASE) function and a cast to string, as it was formerly the case:
</p>
<pre>PREFIX rdf:&lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt;
PREFIX up:&lt;http://purl.uniprot.org/core/&gt;
SELECT ?computedMD5 ((?uniprotMD5 = ?computedMD5) AS ?md5SumsMatch)
WHERE
{
  ?protein a up:Protein ;
    up:sequence ?sequence .
  ?sequence rdf:value ?value ;
    up:md5Checksum ?uniprotMD5 .
  BIND(MD5(?value) AS ?computedMD5)
}
</pre>