---
title: TrEMBL release 9.0
type: releaseNotes
date: 1999-01-01
categories: tr-major
---

                                  TrEMBL Release Notes
                                  Release 9, January 1999

        EMBL Outstation
        European Bioinformatics Institute (EBI)
        Wellcome Trust Genome Campus
        Hinxton
        Cambridge CB10 1SD
        United Kingdom

        Telephone: (+44 1223) 494 444
        Fax: (+44 1223) 494 468
        Electronic mail address: DATALIB@EBI.AC.UK
        WWW server: http://www.ebi.ac.uk/

        Amos Bairoch
        Swiss Institute of Bioinformatics (SIB)
        Centre Medical Universitaire
        1, rue Michel Servet
        1211 Geneva 4
        Switzerland

        Telephone: (+41 22) 784 40 82
        Fax: (+41 22) 702 55 02
        Electronic mail address: BAIROCH@CMU.UNIGE.CH
        WWW server: http://www.expasy.ch/


        Acknowledgements

        TrEMBL has been prepared by:

        o  Rolf Apweiler, Kirsty Bates, Sergio Contrino, Wolfgang Fleischmann,
           Gill Fraser, Henning Hermjakob, Vivien Junker, Youla Karavidopoulou,
           Fiona Lang, Michele Magrane, Maria Jesus Martin, Steffen Moeller,
           Nicoletta Mitaritonna, Claire O'Donovan and Eleanor Whitfield
           at the EMBL Outstation - European Bioinformatics Institute (EBI) in
           Hinxton, UK;
        o  Amos Bairoch and Alain Gateau at the Swiss Institute of Bioinformatics
           in Geneva, Switzerland.

        Notes

        This manual and the database it accompanies may be copied and
        redistributed freely, without advance permission, provided
        that this statement is reproduced with each copy.

        Citation

        If you  want to  cite  TrEMBL  in  a  publication  please  use  the
        following reference:


                  Bairoch A, and Apweiler R.
                  The SWISS-PROT protein sequence data bank and its supplement
                  TrEMBL in 1999.
                  Nucleic Acids Res. 27:49-54(1999).


                             1. Introduction


    TrEMBL is a computer-annotated protein sequence database supplementing the
    SWISS-PROT Protein Sequence Data Bank. TrEMBL contains the translations of
    all coding sequences (CDS) present in the EMBL Nucleotide Sequence Database
    not yet integrated in SWISS-PROT. TrEMBL can be considered as a preliminary
    section of SWISS-PROT. For all TrEMBL entries which should finally be
    upgraded to the standard SWISS-PROT quality, SWISS-PROT accession numbers
    have been assigned.

                            2. Why a supplement to SWISS-PROT?

    The ongoing gene sequencing and mapping projects have dramatically
    increased the number of protein sequences to be incorporated into SWISS-PROT.
    We do not want to dilute the quality standards of SWISS-PROT by incorporating
    sequences without proper sequence analysis and annotation, but we do want to
    make the sequences available as fast as possible. TrEMBL achieves this second
    goal, and is a major step in the process of speeding up subsequent
    upgrading of annotation to the standard SWISS-PROT quality.
    To address the problem of redundancy, the translations of all coding
    sequences (CDS) in the EMBL Nucleotide Sequence Database already included
    in SWISS-PROT have been removed from TrEMBL.

    We name this supplement TrEMBL (Translation from EMBL), since the tools
    used to create the translations of the CDS are based on the program
    'trembl' written by Thure Etzold at the EMBL.

                                 3. The Release

    The goal of this TrEMBL release is to achieve synchronization with SWISS-PROT
    release 37.0. Therefore, all sequence entries present in SWISS-PROT release 37.0
    have been removed from TrEMBL release 9, further upgrading of existing TrEMBL
    entries was achieved and only a very few new entries were incorporated.

    TrEMBL release 9 contains 221422 sequence entries, comprising 59'461'791 amino
    acids.

    TrEMBL is split in two main sections: SP-TrEMBL and REM-TrEMBL:

    SP-TrEMBL (SWISS-PROT TrEMBL) contains the entries (179'066) which should be
    eventually incorporated into SWISS-PROT. SWISS-PROT accession numbers have
    been assigned for all SP-TrEMBL entries.

    SP-TrEMBL is organized in subsections:

    arc.dat (Archea):               7315 entries
    fun.dat (Fungi):                5862 entries
    hum.dat (Human):                7594 entries
    inv.dat (Invertebrates):       22665 entries
    mam.dat (Other Mammals):        2792 entries
    mhc.dat (MHC proteins):         3981 entries
    org.dat (Organelles):          13996 entries
    phg.dat (Bacteriophages):       1736 entries
    pln.dat (Plants):              14626 entries
    pro.dat (Prokaryotes):         39243 entries
    rod.dat (Rodents):              6863 entries
    unc.dat (Unclassified):           44 entries
    vrl.dat (Viruses):             48436 entries
    vrt.dat (Other Vertebrates):    3913 entries

    407 new entries have been integrated in SP-TrEMBL. The sequences of
    979 SP-TrEMBL entries have been updated and the annotation has been updated in
    22'224 entries.

    In the document deleteac.txt, you will find a list of all accession numbers
    which were present in the TrEMBL data bank, but which have now been deleted from
    the database.

    REM-TrEMBL (REMaining TrEMBL) contains the entries (42'356) that we do
    not want to include in SWISS-PROT. REM-TrEMBL entries have no accession
    numbers. This section is organized in five subsections:

       1) Immunoglobulins and T-cell receptors (Immuno.dat)
          Most REM-TrEMBL entries are  immunoglobulins and  T-cell receptors. We
          stopped entering immunoglobulins and T-cell receptors into SWISS-PROT,
          because we only want to keep  the  germ line gene derived translations
          of these proteins in SWISS-PROT and not all known somatic recombinated
          variations of  these proteins.  We would like to  create a specialized
          database  dealing  with  these  sequences  as a further  supplement to
          SWISS-PROT  and  keep  only a  representative  cross-section of  these
          proteins in SWISS-PROT.

       2) Synthetic sequences (Synth.dat)
          Another category of data, which will not be included in SWISS-PROT are
          synthetic sequences.  Again, we do not want to  leave these entries in
          TrEMBL. Ideally one should build a specialized database for artificial
          sequences as a further supplement to SWISS-PROT.

       3) Patent application sequences (Patent.dat)
          A third  subsection consists of  coding sequences captured from patent
          applications.  A thorough  survey of  these  entries  have shown  that
          apart from a rather small minority  (which in  most cases have already
          been integrated in SWISS-PROT), most of these sequences contain either
          erroneous data or concern artificially generated sequences outside the
          scope of SWISS-PROT.

       4) Small fragments (Smalls.dat)
          Another  subsection  consists of fragments  with less than eight amino
          acids.

       5) CDS not coding for real proteins (Pseudo.dat)
          The last subsection consists of  CDS translations where we have strong
          evidence to believe that these CDS are not coding for real proteins.


                    4. Format Differences Between SWISS-PROT and TrEMBL

    The format and conventions used by TrEMBL follow as closely as possible
    that of SWISS-PROT. Hence, it is not necessary to produce an additional
    user manual and extensive release notes for TrEMBL. The information given
    in the SWISS-PROT release notes and user manual are in general valid for
    TrEMBL. The differences are mentioned below.

    The general structure of an entry is identical in SWISS-PROT and TrEMBL.
    The data class used in TrEMBL (in the ID line) is always 'PRELIMINARY',
    whereas in SWISS-PROT it is always 'STANDARD'.

    Differences in line types present in SWISS-PROT and TrEMBL:

    The ID line (IDentification):

    The entry name used in SP-TrEMBL is the same as the Accession Number of the
    entry. The entry name used in REM-TrEMBL is the PID tagged to the
    corresponding CDS in the EMBL Nucleotide Sequence Database. 'PID' stands for
    the "Protein IDentification" number. It is a number that you will find in
    EMBL nucleotide sequence entries in a qualifier called "/db_xref" which is
    tagged to every CDS in the nucleotide database. Example:

       FT   CDS            54..1382
       FT                  /note="ribulose-1,5-bisphosphate carboxylase/
       FT                  oxygenase activase precursor"
       FT                  /db_xref="PID:g1006835"

    The DT line (DaTe)

    The format of the DT lines that serve to indicate when an entry was
    created and updated are identical to that defined in SWISS-PROT; but the
    DT lines in TrEMBL are referring to the TEMBL release. The difference is
    shown in the example below.

        DT lines in a SWISS-PROT entry:

        DT   01-JAN-1988 (REL. 06, CREATED)
        DT   01-JUL-1989 (REL. 11, LAST SEQUENCE UPDATE)
        DT   01-AUG-1992 (REL. 23, LAST ANNOTATION UPDATE)

        DT lines in a TrEMBL entry:

        DT   01-NOV-1996 (TREMBLREL. 01, CREATED)
        DT   01-NOV-1996 (TREMBLREL. 01, LAST SEQUENCE UPDATE)
        DT   01-FEB-1997 (TREMBLREL. 02, LAST ANNOTATION UPDATE)

                    5. Weekly updates of TrEMBL and non-redundant data sets

    Weekly cumulative updates of TrEMBL are available by anonymous FTP and
    from the EBI SRS server.
    We also produce every week a complete non-redundant protein sequence
    collection by providing three compressed files (these are in the directory
    /pub/databases/sp_tr_nrdb on the EBI FTP server and in databases/sp_tr_nrdb
    on the ExPASy server): sprot.dat.Z, trembl.dat.Z and trembl_new.dat.Z.
    This set of non-redundant files is especially important for two types of
    users:
    (i) Managers of similarity search services. They can now provide what is
    currently the most comprehensive and non-redundant data set of protein
    sequences.
    (ii) Anybody wanting to update their full copy of SWISS-PROT + TrEMBL to
    their own schedule without having to wait for full releases of SWISS-PROT
    or of TrEMBL.

                    6. Access/Data Distribution

    FTP server:     ftp.ebi.ac.uk/pub/databases/trembl
    SRS server:     http://srs.ebi.ac.uk/

    TrEMBL is also available on the SWISS-PROT CD-ROM.
    SWISS-PROT + TrEMBL is searchable on the following servers at the EBI:
    FASTA3  (http://www2.ebi.ac.uk/fasta3/)
    BLAST2  (http://www2.ebi.ac.uk/blast2/)
    Bic_sw  (http://www2.ebi.ac.uk/bic_sw/)
    Scanps  (http://www2.ebi.ac.uk/scanps/)
    SSearch (http://www2.ebi.ac.uk/ssearch3/)



                    7. Planned changes

    7.1  Extension of the accession number system

    As explained in detail in the SWISS-PROT release 37.0 release notes section 2.5,
    we will modify the accession number format in the TrEMBL release 10.

    7.2  Conversion of TrEMBL to mixed-case characters

    In synchronization with SWISS-PROT, we will gradually convert TrEMBL
    entries from all 'UPPER CASE' to 'MiXeD CaSe'. This conversion is described
    in detail in SWISS-PROT release 37.0 release notes section 3.1.
