---
title: TrEMBL release 13.0
type: releaseNotes
date: 2000-05-02
categories: tr-major
---

                                  TrEMBL Release Notes
                                  Release 13, May 2000

        EMBL Outstation
        European Bioinformatics Institute (EBI)
        Wellcome Trust Genome Campus
        Hinxton
        Cambridge CB10 1SD
        United Kingdom

        Telephone: (+44 1223) 494 444
        Fax: (+44 1223) 494 468
        Electronic mail address: DATALIB@EBI.AC.UK
        WWW server: http://www.ebi.ac.uk/

        Amos Bairoch
        Swiss Institute of Bioinformatics (SIB)
        Centre Medical Universitaire
        1, rue Michel Servet
        1211 Geneva 4
        Switzerland

        Telephone: (+41 22) 702 58 60
        Fax: (+41 22) 702 55 02
        Electronic mail address: BAIROCH@CMU.UNIGE.CH
        WWW server: http://www.expasy.ch/


        Acknowledgements

        TrEMBL has been prepared by:

        o  Rolf Apweiler, Kirsty Bates, Margaret Biswas, Sergio Contrino,
           Kirill Degtyarenko, Wolfgang Fleischmann, Gill Fraser, Cathy Gedman,
           Henning Hermjakob, Vivien Junker, Youla Karavidopoulou, Paul Kersey,
           Fiona Lang, Minna Lehvaslaiho, Michele Magrane, Maria Jesus Martin,
           Steffen Moeller, Nicoletta Mitaritonna, Virginie Mittard, Nicola Mulder,
           Claire O'Donovan, John F. O'Rourke, Isabelle Phan, Sandrine Pilbout,
           Lucia Rodriguez-Monge, Eleanor Whitfield and  Allyson Williams
           at the EMBL Outstation - European Bioinformatics Institute (EBI)
           in Hinxton, UK;
        o  Amos Bairoch and Alain Gateau at the Swiss Institute of Bioinformatics
           in Geneva, Switzerland.

        Copyright Notice
        TrEMBL copyright (c) 2000 EMBL-EBI
        This manual and the database it accompanies may be copied and
        redistributed freely, without advance permission, provided
        that this copyright statement is reproduced with each copy.

        Citation

        If you  want to  cite  TrEMBL  in  a  publication  please  use  the
        following reference:


                  Bairoch A, and Apweiler R.
                  The SWISS-PROT protein sequence data bank and its supplement
                  TrEMBL in 2000.
                  Nucl. Acids Res. 28:45-48(2000).


                             1. Introduction


    TrEMBL is a computer-annotated protein sequence database supplementing the
    SWISS-PROT Protein Sequence Data Bank. TrEMBL contains the translations of
    all coding sequences (CDS) present in the EMBL Nucleotide Sequence Database
    not yet integrated in SWISS-PROT. TrEMBL can be considered as a preliminary
    section of SWISS-PROT. For all TrEMBL entries which should finally be
    upgraded to the standard SWISS-PROT quality, SWISS-PROT accession numbers
    have been assigned.

                            2. Why a supplement to SWISS-PROT?

    The ongoing gene sequencing and mapping projects have dramatically
    increased the number of protein sequences to be incorporated into SWISS-PROT.
    We do not want to dilute the quality standards of SWISS-PROT by incorporating
    sequences without proper sequence analysis and annotation, but we do want to
    make the sequences available as fast as possible. TrEMBL achieves this second
    goal, and is a major step in the process of speeding up subsequent
    upgrading of annotation to the standard SWISS-PROT quality.
    To address the problem of redundancy, the translations of all coding
    sequences (CDS) in the EMBL Nucleotide Sequence Database already included
    in SWISS-PROT have been removed from TrEMBL.


                                 3. The Release

    This TrEMBL release was created from the EMBL Nucleotide Sequence Database
    release 62 and contains 353'156 sequence entries, comprising 100'750'187 amino
    acids. To minimize redundancy, the translations of all coding sequences (CDS)
    in the EMBL Nucleotide Sequence Database already included in SWISS-PROT release
    38 and updates up to 26.4.2000 have been removed from TrEMBl release 13.

    TrEMBL is split in two main sections: SP-TrEMBL and REM-TrEMBL:
    SP-TrEMBL (SWISS-PROT TrEMBL) contains the entries (300'192) which should be
    eventually incorporated into SWISS-PROT. SWISS-PROT accession numbers have
    been assigned for all SP-TrEMBL entries.

    SP-TrEMBL is organized in subsections:

    arc.dat (Archaea):             11814 entries
    fun.dat (Fungi):                8916 entries
    hum.dat (Human):               13979 entries
    inv.dat (Invertebrates):       45133 entries
    mam.dat (Other Mammals):        5037 entries
    mhc.dat (MHC proteins):         5265 entries
    org.dat (Organelles):          25755 entries
    phg.dat (Bacteriophages):       2667 entries
    pln.dat (Plants):              28814 entries
    pro.dat (Prokaryotes):         62455 entries
    rod.dat (Rodents):             10377 entries
    unc.dat (Unclassified):           31 entries
    vrl.dat (Viruses):             72264 entries
    vrt.dat (Other Vertebrates):    6788 entries

    65'315 new entries have been integrated in SP-TrEMBL. The sequences of
    1'204 SP-TrEMBL entries have been updated and the annotation has been
    updated in 161'387 entries.

    In the document deleteac.txt, you will find a list of all accession numbers
    which were previously present in TrEMBL, but which have now been deleted from
    the database.

    REM-TrEMBL (REMaining TrEMBL) contains the entries (53'861) that we do
    not want to include in SWISS-PROT. REM-TrEMBL entries have no accession
    numbers. This section is organized in six subsections:

       1) Immunoglobulins and T-cell receptors (Immuno.dat)
          Most REM-TrEMBL entries are  immunoglobulins and  T-cell receptors. We
          stopped entering immunoglobulins and T-cell receptors into SWISS-PROT,
          because we only want to keep  the  germ line gene derived translations
          of these proteins in SWISS-PROT and not all known somatic recombinated
          variations of  these proteins.  We would like to  create a specialized
          database  dealing  with  these  sequences  as a further  supplement to
          SWISS-PROT  and  keep  only a  representative  cross-section of  these
          proteins in SWISS-PROT.

       2) Synthetic sequences (Synth.dat)
          Another category of data, which will not be included in SWISS-PROT are
          synthetic sequences.  Again, we do not want to  leave these entries in
          TrEMBL. Ideally one should build a specialized database for artificial
          sequences as a further supplement to SWISS-PROT.

       3) Patent application sequences (Patent.dat)
          A third  subsection consists of  coding sequences captured from patent
          applications.  A thorough  survey of  these  entries  have shown  that
          apart from a rather small minority  (which in  most cases have already
          been integrated in SWISS-PROT), most of these sequences contain either
          erroneous data or concern artificially generated sequences outside the
          scope of SWISS-PROT.

       4) Small fragments (Smalls.dat)
          Another  subsection  consists of fragments  with less than eight amino
          acids.

       5) CDS not coding for real proteins (Pseudo.dat)
          This subsection consists of CDS translations where we have strong
          evidence to believe that these CDS are not coding for real proteins.

       6) Truncated proteins (Truncated.dat)
          The last subsection consists of truncated proteins which result from
          events like mutations introducing a stop codon leading to the truncation
          of the protein product.

                    4. Format Differences Between SWISS-PROT and TrEMBL

    The format and conventions used by TrEMBL follow as closely as possible
    that of SWISS-PROT. Hence, it is not necessary to produce an additional
    user manual and extensive release notes for TrEMBL. The information given
    in the SWISS-PROT release notes and user manual are in general valid for
    TrEMBL. The differences are mentioned below.

    The general structure of an entry is identical in SWISS-PROT and TrEMBL.
    The data class used in TrEMBL (in the ID line) is always 'PRELIMINARY',
    whereas in SWISS-PROT it is always 'STANDARD'.

    Differences in line types present in SWISS-PROT and TrEMBL:

    The ID line (IDentification):

    The entry name used in SP-TrEMBL is the same as the Accession Number of the
    entry. The entry name used in REM-TrEMBL is the stable part of the protein_id
    tagged to the corresponding CDS in the EMBL Nucleotide Sequence Database.
    'protein_id' stands for the "Protein Identification" number. It is a number
    that you will find in the feature table of the EMBL nucleotide sequence
    entries in a qualifier called "/protein_id" which is tagged to every CDS.

    Example:

    FT   CDS             339..1514
    FT                   /codon_start=1
    FT                   /db_xref="PID:g1256015"
    FT                   /product="dystrobrevin-epsilon"
    FT                   /protein_id="AAC50431.1"

    The protein_id is defined as follows in the The DDBJ/EMBL/GenBank Feature Table
    Definition documentation
    Qualifier          /protein_id
    Definition         Protein Identifier, issued by International collaborators.
                       This qualifier consists of a stable ID portion (3+5 format
                       with 3 position letters and 5 numbers) plus a version
                       number after the decimal point.

    Value format       <identifier>
    Example            /protein_id="AAA12345.1"
    Comment            When the protein sequence encoded by the CDS changes, only
                       the version number of the /protein_id value is incremented.
                       The stable part of the /protein_id remains unchanged and
                       as a result will permanently be associated with a given
                       protein. This qualifier is valid only on CDS features
                       which translate into a valid protein.


    The DT line (DaTe)

    The format of the DT lines that serve to indicate when an entry was
    created and updated are identical to that defined in SWISS-PROT; but the
    DT lines in TrEMBL refer to the TrEMBL release. The difference is
    shown in the example below.

        DT lines in a SWISS-PROT entry:

        DT   01-JAN-1988 (Rel. 06, Created)
        DT   01-JUL-1989 (Rel. 11, Last sequence update)
        DT   01-AUG-1992 (Rel. 23, Last annotation update)

        DT lines in a TrEMBL entry:

        DT   01-NOV-1996 (TrEMBLrel. 01, Created)
        DT   01-NOV-1996 (TrEMBLrel. 01, Last sequence update)
        DT   01-FEB-1997 (TrEMBLrel. 02, Last annotation update)

                    5. Weekly updates of TrEMBL and non-redundant data sets

    Weekly cumulative updates of TrEMBL are available by anonymous FTP and
    from the EBI SRS server.

    We also produce every week a complete non-redundant protein sequence
    collection by providing three compressed files (these are in the directory
    /pub/databases/sp_tr_nrdb on the EBI FTP server and in databases/sp_tr_nrdb
    on the ExPASy server): sprot.dat.Z, trembl.dat.Z and trembl_new.dat.Z.
    This set of non-redundant files is especially important for two types of
    users:
    (i) Managers of similarity search services. They can now provide what is
    currently the most comprehensive and non-redundant data set of protein
    sequences.
    (ii) Anybody wanting to update their full copy of SWISS-PROT + TrEMBL to
    their own schedule without having to wait for full releases of SWISS-PROT
    or of TrEMBL.

                    6. Access/Data Distribution

    FTP server:     ftp.ebi.ac.uk/pub/databases/trembl
    SRS server:     http://srs.ebi.ac.uk/

    TrEMBL is also available on the SWISS-PROT CD-ROM.
    SWISS-PROT + TrEMBL is searchable on the following servers at the EBI:

    FASTA3  (http://www.ebi.ac.uk/fasta3/)
    BLAST2  (http://www.ebi.ac.uk/blast2/)
    Bic_sw  (http://www.ebi.ac.uk/bic_sw/)
    Scanps  (http://www.ebi.ac.uk/scanps/)
    MPSrch  (http://www.ebi.ac.uk/MPsrch/)

      7. Description of changes made to TrEMBL since release 12.

    7.1 Conversion of TrEMBL to mixed cases.

    In this release we have converted the RA lines to mixed cases.

    7.2 Change in the syntax of the SQ line.

    The last item in the SQ line, the 32-bit CRC value, has been replaced by a
    64-bit CRC. The format of the SQ line is:

    SQ   SEQUENCE  XXXX AA; XXXXXX MW;  XXXXXXXXXXXXXXXX CRC64;

    Example:

    SQ   SEQUENCE   308 AA;  35309 MW;  B26AE1BBDA64E683 CRC64;

    7.3 Multiple AC lines

    In the ongoing process of updating and annotating TrEMBL entries, we find
    entries which can be merged. We keep all the original accession numbers
    in the resultinh merged entry. The repetition of such a process sometimes
    produces an accession number list which can no longer fit in a single AC
    line. Therefore there will now be some entries with two, three or more AC
    lines (as shown below).

    AC   Q00626; Q57339; O08022; O08102; O09490; O09483; O09393; O09396;
    AC   O09397; O09398; O09399; O09400; O09401; O09402; O09403; O09404;
    AC   O09405; O09406; O09407; O09408; O09481; O09482;


    7.4  Changes concerning cross-references (DR line)

    We have added cross-references from TrEMBL to
    the INTERPRO (Integrated Resource of Protein Domains and Funcional Sites)
    database available at ftp.ebi.ac.uk/pub/databases/interpro/


    A list of all crossreferences in TrEMBL to other databases is provided below:
    AARHUS/GHENT-2DPAGE;
    EMBL
    FLYBASE
    GCRDB
    HSSP
    MENDEL
    MGD
    MIM
    PDB
    INTERPRO
    PFAM
    PIR
    PRINTS
    PROSITE
    SGD
    SUBTILIST
    TIGR
    TRANSFAC
    WORMPEP
    ZFIN
      
