---
title: UniProt release 1.7
type: releaseNotes
date: 2004-04-13
categories: release,Taxonomy
---

# Headlines

## Extinct organisms and Swiss-Prot...

Did you know that Swiss-Prot contains proteins originating from extinct organisms? Since the beginning of the 90s, various groups have sequenced gene fragments from a variety of extinct organisms. Most of the time, the resulting sequences are too small or too fragmentary to be translated into protein sequences. But this is not always the case, and we harbor a few complete or partial sequences originating from species that existed on earth in various periods of time.

For example we have a RuBisCO large subunit from a fossil leaf of a Miocene (17-20 Myr old) Magnolia, [P30828](http://www.uniprot.org/uniprotkb/P30828) .

Much more recent is a complete cytochrome b sequence from a Siberian mammoth, [P92658](http://www.uniprot.org/uniprotkb/P92658) .

But what is more interesting for those interested in the longevity of proteins, is the complete sequence of an osteocalcin from a steppe bison. This sequence was ontained by mass spectrometry directly from permafrost fossilized bones, about 55-56 Kyr old, [P83489](http://www.uniprot.org/uniprotkb/P83489) .

# UniProtKB news

## Cross-references to Oxford GlycoProteomics 2-DE database (OGP)

We have added cross-references to the Oxford GlycoProteomics 2-DE database (OGP), available at <http://proteomewww.bioch.ox.ac.uk/2d/2d.html> .

The identifiers of the appropriate DR line are:

<table><colgroup><col style="width: 24%" /><col style="width: 76%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>OGP</td></tr><tr class="even"><td>Resource identifier</td><td>OGP's unique identifier for a protein, which is identical to the Swiss-Prot primary AC number of that protein.</td></tr><tr class="odd"><td>Example</td><td><pre><code>P31946:
DR   OGP; P31946; -.</code></pre></td></tr></tbody></table>

## Cross-references to 2-DE database of rat heart

We have added cross-references to the 2-DE database of rat heart, at German Heart Institute Berlin, available at <http://www.mpiib-berlin.mpg.de/2D-PAGE/RAT-HEART/2d/> .

The identifiers of the appropriate DR line are:

<table><colgroup><col style="width: 22%" /><col style="width: 77%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>Rat-heart-2DPAGE</td></tr><tr class="even"><td>Resource identifier</td><td>Rat-heart-2DPAGE's unique identifier for a protein, which is identical to the Swiss-Prot primary AC number of that protein.</td></tr><tr class="odd"><td>Example</td><td><pre><code>P03996:
DR   Rat-heart-2DPAGE; P03996; -.</code></pre></td></tr></tbody></table>

## Changes concerning keywords

New keyword:

- [Leber congenital amaurosis](http://www.uniprot.org/keywords/KW-0901)
- [Two-component regulatory system](http://www.uniprot.org/keywords/KW-0902)

## Changes concerning the [controlled vocabulary for PTMs](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/ptmlist)

We are continuously overhauling the annotation of post-translational modifications (PTMs). Phosphorylation sites are described in the description field of the feature key MOD_RES, all entries with such a site contain the keyword 'Phosphorylation'. The initially defined controlled vocabulary for phosphorylation sites is listed below:

    Phosphocysteine
    4-aspartylphosphate
    Phosphohistidine
    Tele-phosphohistidine
    Pros-phosphohistidine
    Phosphoserine
    Phosphothreonine
    Phosphotyrosine
