---
title: UniProt release 14.6
type: releaseNotes
date: 2008-12-16
categories: release,Cross-references,Chordata
---

# Headlines

## GeneCards: yet another means to get human gene chromosomal location

UniProtKB aims to be a central hub for biological information on proteins. While the protein sequence is described in depth at the residue level in the 'Sequence annotation (Features)' section of UniProtKB/Swiss-Prot entries, the general context in which the protein exists and functions (mostly provided in the 'General annotation (Comments)' section) is kept at a general interest level. Users interested in more detailed information are invited to deepen their knowledge by looking into the original publications (in the 'References' section) and making use of the [numerous cross-references](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/dbxref), mostly found in the 'Cross- references' section that is becoming larger and larger with each release.

In the current release, we have added cross-references to [GeneCards](http://www.genecards.org/). This database focuses on human genes. The information provided by GeneCards is automatically extracted from more than 50 [databases](http://www.genecards.org/sources.shtml), some of which are manually annotated, such as OMIM and UniProtKB/Swiss-Prot. While much of the information provided by GeneCards overlaps with that found in UniProtKB/Swiss-Prot, it also contains additional data which complement our annotations.

GeneCards indicates very precisely the chromosomal location of each gene, not only at the chromosome (sub)bands, but also at the level of base pairs, clearly indicating from which end of the chromosome the position is calculated (see for instance [ATP10A](http://www.genecards.org/cgi-bin/carddisp.pl?gc_id=GC15M023473)). This type of information is not currently provided directly in UniProtKB/Swiss-Prot entries, but can be accessed through links to other databases, such as Ensembl and now GeneCards. Note, however, that we provide a complete list of all human proteins, chromosome by chromosome, on the ['human-centric' page](http://www.expasy.org/sprot/hpi/) on the ExPASy server. For each chromosome, the list can be downloaded from the UniProt ftp site (see for instance all proteins encoded on [chromosome 1](ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humchr01.txt)).

# UniProtKB News

## Cross-references to GeneCards

Cross-references have been added to [GeneCards](http://www.genecards.org/). GeneCards is a searchable, integrated database of human genes that provides concise genomic, proteomic, transcriptomic, genetic and functional information on all known and predicted human genes.

GeneCards is available at <http://www.genecards.org/>.

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 48%" /><col style="width: 51%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>GeneCards</td></tr><tr class="even"><td>Resource identifier</td><td>GeneCards unique identifier.</td></tr><tr class="odd"><td>Examples</td><td><pre><code>Q6PCB8:
DR   GeneCards; GC05M049731; -.

P69905:
DR GeneCards; GC16P000162; -.
DR GeneCards; GC16P000166; -.</code></pre></td></tr></tbody></table>

## Cross-references to PRIDE

Cross-references have been added to [PRIDE PRoteomics IDEntifications database](http://www.ebi.ac.uk/pride//). The PRIDE PRoteomics IDEntifications database is a centralized, standards compliant, public data repository for proteomics data.

PRIDE is available at <http://www.ebi.ac.uk/pride/>.

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 50%" /><col style="width: 50%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>PRIDE</td></tr><tr class="even"><td>Resource identifier</td><td>UniProtKB accession number.</td></tr><tr class="odd"><td>Examples</td><td><pre><code>Q9Y5P4:
DR   PRIDE; Q9Y5P4; -.

P25296:
DR PRIDE; P25296; -.</code></pre></td></tr></tbody></table>

## Changes concerning cross-references to IntAct

We have changed the format of the cross-reference lines to IntAct to add the number of interactions.

<table><colgroup><col style="width: 50%" /><col style="width: 50%" /></colgroup><tbody><tr class="odd"><td>Optional information 1</td><td>Number of interactions.</td></tr><tr class="even"><td>Example</td><td><pre><code>O01802:
DR   IntAct; O01802; 12.</code></pre></td></tr></tbody></table>
