---
title: UniProt release 2011_10
---

# Headline

## The sound of silence

Cytosine methylation is the major and best characterized epigenetic modification of metazoan

DNA. It is implicated in long-term gene silencing, X chromosome inactivation, genomic imprinting, etc. 5-methylcytosine (5mC) is recognized by methyl-binding proteins ( [MBDs](http://www.uniprot.org/uniprot/?query=name:%22Methyl-CpG-binding+domain+protein%22+reviewed:yes) ), that in turn recruit repressive histone modifiers, such as

H3K9 methyltransferases, to establish a heterochromatin state.

Cytosine base methylation is catalyzed by the [C5-methyltransferase](http://www.uniprot.org/uniprot/?query=family:%22C5-methyltransferase+family%22+reviewed:yes) enzyme family.

DNMT3A and

DNMT3B methylate

DNA de novo.

DNMT1 maintains the methylation status across cell divisions. In the absence of

DNMT1 activity,

DNA methylation is progressively lost since methylation is not replicated onto the newly synthesized strand, leading to passive

DNA demethylation.

However, passive

DNA demethylation cannot account for rapid demethylation that occurs in the paternal genome in the zygote within the first 4 hours following fertilization or that observed in primordial germ cells, both of which are independent of

DNA replication. While [demethylases](http://www.uniprot.org/uniprot/?query=family:%22DNA+glycosylase+family+DEMETER+subfamily%22) have been identified in *Arabidopsis thaliana*, the mechanism of active demethylation in mammals remained elusive ( [reviews](http://www.ncbi.nlm.nih.gov/pubmed/21811096,20683471) ). [2011](http://www.ncbi.nlm.nih.gov/pubmed/21778364,21817016,21490601,21460036,21750410,21451524,21552279,21460836,21817016,21295276,21321204,21722948,21278727,21496894) has unveiled the central role played by

TET family members. These enzymes have already been shown to [to catalyze the conversion of 5mC into 5-hydroxymethylcytosine](http://www.ncbi.nlm.nih.gov/pubmed/19372391 "5hmC"). 5hmC can be further processed, either by G/T mismatch-specific thymine

DNA glycosylase (TDG) or by deamination enzymes, such as

APOBEC1 and

AICDA /AID, and eventually removed and replaced by unmodified cytosine by base excision repair mechanism.

Interestingly,

TET1 has a [role in transcriptional repression](http://www.ncbi.nlm.nih.gov/pubmed/21490601), independently of its enzymatic activity. It binds a significant proportion of Polycomb group target genes and associates and colocalizes with the

SIN3A co-repressor complex.

These new exciting data pave the way for understanding transcriptional fine-tuning during embryonic development, as well as in adult organisms and will keep us busy updating UniProtKB for quite a while.

# UniProtKB news

## Changes to [keywords](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/keywlist)

Deleted keyword:

-   [Virus reference strain](http://www.uniprot.org/keywords/KW-1019)
