---
title: UniProt release 2012_06
type: releaseNotes
date: 2012-06-13
categories: Fungi,Proteomes
---

# Headline

## Fungal prion proteins – disease or evolutionary motor?

The word “prion”, coined in 1982 by Stanley B. Prusiner, is derived from the words “protein” and “infection”. It is used to describe the infectious, non-chromosomal genetic elements that are at the heart of the mammalian transmissible spongiform encephalopathies (TSEs, including scrapie of sheep, “Mad cow disease”, and Creutzfeldt-Jakob disease of humans). It is believed that these diseases are caused by the self-propagating conformational change of a protein,

PRNP, or its assembly into an amyloid form.

A prion is an infectious agent made of a protein in a misfolded form. This altered inactive form converts its normal active counterpart into the same inactive form. Three distinct genetic traits have been defined that must be satisfied by a prion: 1. “Curing” of a prion is reversible. In the appropriate conditions, for instance in the absence of a specific molecular chaperone, the protein can reacquire its active conformation. The prion form can nonetheless arise again de novo because the protein is still present in the cell. 2. Overproduction of the protein should increase its frequency of conversion to the prion (infectious) form, whatever the mechanism. 3. Prions being inactive forms of physiological occurring proteins, the protein-encoding gene should be necessary for propagation of the prion, and inactivating mutations of this gene could produce a similar phenotype to that observed in the presence of the prion. Based on these criteria, prion proteins were also identified in fungi, primarily in the yeast _Saccharomyces cerevisiae_, for example the well-studied [\[PSI+\]](http://www.uniprot.org/uniprotkb/P05453), [\[URE3\]](http://www.uniprot.org/uniprotkb/P23202), and [\[PIN\]](http://www.uniprot.org/uniprotkb/P25367) prions. These classical, amyloid-forming prion proteins provide an excellent model for the understanding of the disease-forming mammalian prions.

In recent years, several additional fungal prion proteins have been identified. Their study provided 2 fundamental insights into prion biology. First, a protein does not need to form amyloid aggregates to be infectious. Other mechanisms like covalent autoactivation of an enzyme ( [\[beta\]](http://www.uniprot.org/uniprotkb/P09232) ) or even the interaction between two proteins ( [\[GAR+\]](http://www.uniprot.org/uniprotkb?query=accession:Q02794+OR+accession:P05030) ) can turn proteins into prions. But even more interesting is the fact that some of the fungal prions are not associated with any disease state, but may even have a beneficial role for the host. The _Podospora anserina_ [\[Het-s\]](http://www.uniprot.org/uniprotkb/Q03689) prion confers heterokaryon incompatibility, a process that ensures that during spontaneous, vegetative cell fusion only compatible cells from the same colony survive (non-self-recognition). In _S. cerevisiae_, the prevalence of transcriptional regulators ( [Cyc8](http://www.uniprot.org/uniprotkb/P14922), [Mot3](http://www.uniprot.org/uniprotkb/P54785), [Sfp1](http://www.uniprot.org/uniprotkb/P32432), [Swi1](http://www.uniprot.org/uniprotkb/P09547) and [Ure2](http://www.uniprot.org/uniprotkb/P23202) ) among the yeast prions led to the speculation that prion properties of transcription factors may generate an optimized phenotypic heterogeneity that buffers yeast populations against diverse environmental insults. Even more recent results on the adaptation of cells to anti-fungal drugs by the prion form of the mitochondrial tRNA dimethylallyltransferase ( [\[MOD+\]](http://www.uniprot.org/uniprotkb/P07884) ) shows that this may also be true for enzymes and supports the hypothesis that fungal prions may be beneficial for the host and contribute to cellular adaptation in living organisms.

As of this release, all prion-forming fungal proteins known to date have been reviewed and updated, with a special emphasis put on the prion-forming mechanism and on the consequences and phenotypes of the intracellular prion form. To make a clear distinction between the prion form characteristics and the physiological properties of the soluble cellular protein, the annotation dealing with prion have been integrated in a separate subsection (‘Miscellaneous’) in ‘General annotation (Comments)’.

[Fungal prion-forming proteins](http://www.uniprot.org/uniprotkb?query=taxonomy:Fungi+AND+keyword:KW-0640+AND+reviewed:true) can be retrieved using the keyword [‘Prion’](http://www.uniprot.org/keywords/KW-0640) .

# UniProtKB news

## Complete proteomes for Ensembl Genomes species

[Ensembl Genomes](http://www.ensemblgenomes.org/) species were made available for the first time in [UniProt release 2012_04](http://www.uniprot.org/release-notes/2012-04-18-release) .

For UniProt release 2012_06, 5 new Ensembl Genome species have been made available, these are:

[_Amphimedon queenslandica_](http://metazoa.ensembl.org/Amphimedon_queenslandica/Info/Index)  
[_Gibberella zeae_](http://fungi.ensembl.org/Gibberella_zeae/Info/Index)  
[_Brachypodium distachyon_](http://www.gramene.org/Brachypodium_distachyon/Info/Index)  
[_Glycine max_](http://www.gramene.org/Glycine_max/Info/Index)  
[_Oryza glaberrima_](http://www.gramene.org/Oryza_glaberrima/Info/Index)

All predicted protein sequences from an Ensembl Genome are mapped to their UniProtKB counterparts under stringent conditions: 100% identity over 100% of the length of the two sequences is required. Any sequence found to be absent from UniProtKB is imported into the unreviewed component of UniProtKB, UniProtKB/TrEMBL. All UniProtKB entries that map to an Ensembl Genome are used to build the proteome; they are tagged with the keyword [Complete proteome](http://www.uniprot.org/keywords/KW-0181) and an Ensembl Genome cross-reference is added.

We very much welcome the feedback of the community on our efforts. In future UniProt releases, we expect to make proteomes for the remaining Ensembl Genome species currently absent from UniProtKB.

## Genome submission for _Bos taurus_ updated to be in line with Ensembl

The underlying genome submission for _Bos taurus_ has been updated to be in sync with the third party assembly of the genome used by Ensembl for their annotations. For details of the Ensembl assembly for _Bos taurus_, see the [Ensembl website](http://www.ensembl.org/Bos_taurus/Info/Index) .

## Changes to cross-reference to PhosSite

The resource identifiers of the cross-references to the Phosphorylation Site Database for Archaea and Bacteria ( [PhosSite](http://www.phosphorylation.biochem.vt.edu/) ) have changed from a UniProtKB primary accession number to a Phosphorylation Site Database unique identifier for a phosphoprotein.

Example:  
Previous format:

    DR   PhosSite; P08839; -.

New format:

    DR   PhosSite; P0810428; -.

[Show all the entries having a cross-reference to PhosSite.](http://www.uniprot.org/uniprotkb?query=database%3Aphossite&sort=score)

## UniProt Gene Ontology Annotation

UniProt is a central member of the [Gene Ontology Consortium](http://www.geneontology.org/), an initiative founded in 1998 to develop and use a set of ontologies to represent three aspects of biology carried out by gene products from any organism. Terms within the Gene Ontology (GO) describe those molecular functions and biological processes that gene products carry out and the subcellular locations in which they are located.

UniProt curators contribute manual GO annotations to proteins from a wide range of species. In addition, to ensure that UniProt provides a comprehensive GO annotation resource and to avoid duplication of effort, GO annotations are also integrated from more than 30 external model organism and multi-species databases including dictyBase, EcoCyc, FlyBase, Gramene, Human Protein Atlas, IntAct, LifeDB,

MGI, PomBase, Reactome,

RGD, TAIR,

SGD, WormBase and

ZFIN .

High-quality automatic GO annotations are also supplied to the UniProt GO annotation set by Ensembl, EnsemblGenomes, InterPro and UniProt prediction pipelines. Such automatic pipelines differently exploit gene orthology data, protein sequence signatures and existing cross-references or keywords from external controlled vocabularies, to infer that a protein has a particular function or subcellular location. The inclusion of such high-quality, automatic annotation predictions ensures the UniProt GO annotation dataset supplies functional information to a wide range of proteins, including those from poorly characterised, non-model organism species. In the May 2012 UniProt release, a total of 125 million GO annotations are supplied for 14.8 million proteins from more than 338,000 taxonomic groups.

GO annotations are present in the ‘Ontologies’ section UniProtKB entries (see for example

P09960 ) and are available to download from the

GOA ftp site. GO annotations can additionally be viewed via the [QuickGO browser](http://www.ebi.ac.uk/QuickGO/). We are pleased to announce an addition to the UniProt GO automatic annotation pipelines: UniPathway2GO.

**New UniPathway2GO pipeline**

In collaboration with the

SIB Swiss Institute of Bioinformatics,

INRIA (Rhone-Alpes) and Laboratoire d’Ecologie Alpine (Grenoble), UniProt is pleased to announce the inclusion of an additional 113,285 GO annotations that describe the pathway(s) in which 105,041 UniProtKB entries are involved.

[UniPathway](http://www.grenoble.prabi.fr/obiwarehouse/unipathway) is a manually curated resource of enzyme-catalyzed and spontaneous chemical reactions that provides a hierarchical representation of metabolic pathways.

Currently 425 UniPathway pathway terms have been manually mapped to GO terms and 48% of these annotations apply a GO term that either uniquely describes a protein’s involvement in a certain process, or supplies a more granular term than is supplied by other automatic annotation methods.
