---
title: UniProt release 2014_11
categories: human
---

<h2 id="Headline">
 Headline
</h2>
<h3 id="Higher_and_higher">
 Higher and higher
</h3>
<p>
 It is in human nature to push back the frontiers of what is possible. Modern humans left Africa and conquered the world. During their exploration, they met other humans who had already colonized the most improbable places tens of thousands of years earlier, maybe themselves being driven by the same urge to discover new horizons. Among the most challenging dwelling places is the Tibetan plateau, with an average elevation exceeding 4,500 meters. At this altitude, the oxygen concentration is only 60% of that available at sea level. Nevertheless, the Tibetan plateau is thought to have been inhabited for some 25,000 years.
</p>
<p>
 To maintain oxygen homeostasis at high altitude (over 2,500 meters), the body responds in various ways, including increasing ventilation over the short term and increasing red blood cell production over the long term (see
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/25319824">
  review
 </a>
 ). Hypoxia-inducible factor (HIF) plays a key role in the regulation of gene transcription in this process. HIF is a dimer composed of a common subunit beta, called
 <a href="http://www.uniprot.org/uniprot/P27540">
  ARNT
 </a>
, and 1 of 3 alpha subunits, called
 <a href="http://www.uniprot.org/uniprot/?query=%28gene:HIF1A+OR+gene:HIF3A+OR+gene:EPAS1%29+AND+organism:9606">
  HIF1A, EPAS1, or HIF3A
 </a>
. Under normoxic conditions, HIFs-alpha are hydroxylated by prolyl hydroxylases
 <a href="http://www.uniprot.org/uniprot/?query=accession:q9gzt9+OR+accession:q96ks0+OR+accession:q9h6z9">
  EGLN1 (also known as PHD2), EGLN2 or EGLN3
 </a>
. Hydroxylation allows interaction with an E3-ubiquitin ligase, named
 <a href="http://www.uniprot.org/uniprot/P40337">
  VHL
 </a>
, followed by proteasomal degradation. Under hypoxic conditions, hydroxylation is arrested and HIFs-alpha are stabilized. They dimerize with ARNT and initiate the hypoxia response transcriptional program, which includes the stimulation of erythropoiesis. Strikingly, Tibetans exhibit a blunted erythropoietic response and their hemoglobin concentration is maintained at values expected at sea-level.
</p>
<p>
 In 2010,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/20466884,20534544,20838600">
  3 independent publications
 </a>
 identified genes or loci showing evidence of hypoxia adaptation in Tibetans. All 3 studies pointed to 2 genes, among many others, being significantly associated with the decreased hemoglobin phenotype. They are
 <a href="http://www.uniprot.org/uniprot/Q99814">
  EPAS1
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/Q9GZT9">
  EGLN1
 </a>
. Interestingly, Tibetans may have
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/25043035">
  inherited EPAS1 SNPs from Denisova man
 </a>
, an archaic
 <em>
  Homo
 </em>
 species identified in the Altai mountains of Siberia. The Tibetan-specific EGLN1 variant is more recent, currently estimated to have appeared some 8,000 years ago. It contains 2 single amino acid polymorphisms: p.Asp4Cys and p.Cys127Ser. Some characterization of this double variant came in September this year.
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/25129147">
  Lorenzo et al.
 </a>
 showed that it exhibited a lower K(m) value for oxygen, suggesting that it promotes increased HIF-alpha hydroxylation and degradation under hypoxic conditions. It could hence abrogate hypoxia-induced and HIF-mediated augmentation of erythropoiesis.
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24711448">
  Song et al.
 </a>
 reported that the double variant specifically interferes with binding to
 <a href="http://www.uniprot.org/uniprot/Q15185">
  PTGES3
 </a>
 (also called HSP90 cochaperone p23), but not to other known EGLN1 ligands, including FKBP8 or HSP90AB. As PTGES3-binding may facilitate HIF-alpha hydroxylation, a perturbation in this interaction would actually decrease HIF-alpha hydroxylation, hence decreased degradation and consequently increased HIF activity. The central question about the functional consequences of the Tibetan EGLN1 variant remains open...
</p>
<p>
 It is not yet clear how high-altitude populations adapted to their harsh environment, but at least we begin to grasp the amazing complexity of this phenomenon. The scientific community has studied mostly 3 populations, Tibetans, Andeans and Ethiopians settled on the Simien plateau. They all exhibit patterns of genetic adaptation largely distinct from one another and the overlap is surprisingly low. The polymorphisms identified so far may not be straightforward loss- or gain-of-function, but they may instead fine tune complex interactions in which several proteins, possibly themselves carrying adaptive variations, are involved in a tissue-specific context.
</p>
<p>
 As of this release, the UniProtKB/Swiss-Prot
 <a href="http://www.uniprot.org/uniprot/Q9GZT9">
  human EGLN1
 </a>
 has been updated with the new characterization data of the p.[Asp4Cys; Cys127Ser] polymorphism. On the new UniProt website, this information is to be found in the 'Sequences' section, 'Polymorphism' and 'Natural variant' subsections.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="New_mouse_and_zebrafish_variation_files">
 New mouse and zebrafish variation files
</h3>
<p>
 We would like to announce the release of two additional species, mouse and zebrafish, to the set of variation files available in the dedicated
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/variants/">
  variants
 </a>
 directory on the UniProt FTP sites. Both files catalogue protein altering Single Nucleotide Variants (SNVs or SNPs), stop-gained and stop-lost variants for UniProtKB/Swiss-Prot and UniProtKB/TrEMBL sequences of each species. These variants have been automatically mapped to UniProtKB sequences, including isoform sequences, through Ensembl. We very much welcome the feedback of the community on our efforts.
</p>
<h3 id="Structuring_of__cofactor__annotations">
 Structuring of 'cofactor' annotations
</h3>
<p>
 We have structured the previously free text
 <a href="http://www.uniprot.org/manual/cofactor">
  cofactor annotations
 </a>
 in UniProtKB and mapped individual cofactors to
 <a href="https://www.ebi.ac.uk/chebi/">
  ChEBI
 </a>
 identifiers. How this affects different UniProtKB distribution formats is described below.
</p>
<h4 id="Text_format">
 Text format
</h4>
<pre>CC   -!- COFACTOR:( &lt;molecule&gt;:)?
(CC       Name=&lt;cofactor&gt;; Xref=&lt;database&gt;:&lt;identifier&gt;;( Evidence={&lt;evidence&gt;};)?)*
(CC       Note=&lt;free text&gt;;)?
</pre>
<p>
 <em>
  Note: Perl-style multipliers indicate whether a pattern (as delimited by parentheses) is optional (?) or may occur 0 or more times (*).
 </em>
</p>
<p>
 A cofactor annotation consists of:
</p>
<ul>
 <li>
  An optional
  <code>
   &lt;molecule&gt;
  </code>
  value that indicates the isoform, chain or peptide to which this annotation applies.
 </li>
 <li>
  Zero or more cofactors that are each described with:
  <ul>
   <li>
    A
    <code>
     Name=
    </code>
    field that shows the cofactor name.
   </li>
   <li>
    A
    <code>
     Xref=
    </code>
    field that shows a cross-reference to the corresponding ChEBI record.
   </li>
   <li>
    An optional
    <code>
     Evidence=
    </code>
    field that provides the evidence for the cofactor (see
    <a href="http://www.uniprot.org/news/2014/10/01/release">
     Evidence in the UniProtKB flat file format
    </a>
    )
   </li>
  </ul>
 </li>
 <li>
  An optional
  <code>
   Note=
  </code>
  field that provides additional information.
 </li>
</ul>
<p>
 Each cofactor description and the optional
 <code>
  Note=
 </code>
 field start on a new line. Lines are wrapped at a line length of 75 characters and indented to increase readability.
</p>
<p>
 Examples:
</p>
<ul>
 <li>
  Protein binds alternate/several cofactors
  <pre>CC   -!- COFACTOR:
CC       Name=Mg(2+); Xref=ChEBI:CHEBI:18420;
CC         Evidence={ECO:0000255|HAMAP-Rule:MF_00086};
CC       Name=Co(2+); Xref=ChEBI:CHEBI:48828;
CC         Evidence={ECO:0000255|HAMAP-Rule:MF_00086};
CC       Note=Binds 2 divalent ions per subunit (magnesium or cobalt).
CC       {ECO:0000255|HAMAP-Rule:MF_00086};
CC   -!- COFACTOR:
CC       Name=K(+); Xref=ChEBI:CHEBI:29103;
CC         Evidence={ECO:0000255|HAMAP-Rule:MF_00086};
CC       Note=Binds 1 potassium ion per subunit. {ECO:0000255|HAMAP-
CC       Rule:MF_00086};
</pre>
 </li>
</ul>
<ul>
 <li>
  Isoforms
  <pre>CC   -!- COFACTOR: Isoform 1:
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105;
CC         Evidence={ECO:0000269|PubMed:16683188};
CC       Note=Isoform 1 binds 3 Zn(2+) ions. {ECO:0000269|PubMed:16683188};
CC   -!- COFACTOR: Isoform 2:
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105;
CC         Evidence={ECO:0000269|PubMed:16683188};
CC       Note=Isoform 2 binds 2 Zn(2+) ions. {ECO:0000269|PubMed:16683188};
</pre>
 </li>
</ul>
<ul>
 <li>
  Chains
  <pre>CC   -!- COFACTOR: Serine protease NS3:
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105;
CC         Evidence={ECO:0000269|PubMed:9060645};
CC       Note=Binds 1 zinc ion. {ECO:0000269|PubMed:9060645};
CC   -!- COFACTOR: Non-structural protein 5A:
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105; Evidence={ECO:0000250};
CC       Note=Binds 1 zinc ion in the NS5A N-terminal domain.
CC       {ECO:0000250};
</pre>
 </li>
</ul>
<ul>
 <li>
  Cofactor unknown
  <pre>CC   -!- COFACTOR:
CC       Note=Does not require a metal cofactor.
CC       {ECO:0000269|PubMed:24450804};
</pre>
 </li>
</ul>
<h4 id="XML_format">
 XML format
</h4>
<p>
 We modified the XSD type
 <code>
  commentType
 </code>
 and introduced a new XSD type
 <code>
  cofactorType
 </code>
 as shown in red. We also moved the declaration of the
 <code>
  molecule
 </code>
 element - already used in the comment type
 <code>
  "subcellular location"
 </code>
 - to a more generic context so that it can also be used by other comment types such as
 <code>
  "cofactor"
 </code>
 .
</p>
<pre><code>&lt;xs:complexType name="commentType"&gt;
        ...
        &lt;xs:sequence&gt;</code>
            <code style="color: red;">&lt;xs:element name="molecule" type="moleculeType" minOccurs="0"/&gt;</code>
            <code>&lt;xs:choice minOccurs="0"&gt;
            ...</code>
                <code style="color: red;">&lt;xs:sequence&gt;
                    &lt;xs:annotation&gt;
                        &lt;xs:documentation&gt;Used in 'cofactor' annotations.&lt;/xs:documentation&gt;
                    &lt;/xs:annotation&gt;
                    &lt;xs:element name="cofactor" type="cofactorType" maxOccurs="unbounded"/&gt;
                &lt;/xs:sequence&gt;</code>

                <code>&lt;xs:sequence&gt;
                    &lt;xs:annotation&gt;
                        &lt;xs:documentation&gt;Used in 'subcellular location' annotations.&lt;/xs:documentation&gt;
                    &lt;/xs:annotation&gt;</code>
                    <code style="color: red;">&lt;!-- &lt;xs:element name="molecule" type="moleculeType" minOccurs="0"/&gt; --&gt;</code>
                    <code>&lt;xs:element name="subcellularLocation" type="subcellularLocationType" maxOccurs="unbounded"/&gt;
                &lt;/xs:sequence&gt;
                ...
            &lt;/xs:choice&gt;
            ...
            &lt;xs:element name="text" type="evidencedStringType" minOccurs="0"&gt;
                &lt;xs:annotation&gt;
                    &lt;xs:documentation&gt;Used to store non-structured types of annotations,
                    as well as optional free-text notes of structured types of annotations.&lt;/xs:documentation&gt;
                &lt;/xs:annotation&gt;
            &lt;/xs:element&gt;
            ...
        &lt;/xs:sequence&gt;
        ...
    &lt;/xs:complexType&gt;
    ...</code>
    <code style="color: red;">&lt;xs:complexType name="cofactorType"&gt;
        &lt;xs:annotation&gt;
            &lt;xs:documentation&gt;Describes a cofactor.&lt;/xs:documentation&gt;
        &lt;/xs:annotation&gt;
        &lt;xs:sequence&gt;
            &lt;xs:element name="name" type="xs:string"/&gt;
            &lt;xs:element name="dbReference" type="dbReferenceType"/&gt;
        &lt;/xs:sequence&gt;
        &lt;xs:attribute name="evidence" type="intListType" use="optional"/&gt;
    &lt;/xs:complexType&gt;</code>
</pre>
<p>
 A cofactor annotation consists of a sequence of:
</p>
<ul>
 <li>
  An optional
  <code>
   molecule
  </code>
  element that indicates the isoform, chain or peptide to which this annotation applies.
 </li>
 <li>
  Zero or more
  <code>
   cofactor
  </code>
  elements that each describe an individual cofactor with the following child elements:
  <ul>
   <li>
    A
    <code>
     name
    </code>
    element shows the cofactor name.
   </li>
   <li>
    A
    <code>
     dbReference
    </code>
    element represents a cross-reference to the corresponding ChEBI record.
   </li>
  </ul>
 </li>
 <li>
  An optional
  <code>
   text
  </code>
  element that provides additional information.
 </li>
</ul>
<p>
 Examples:
</p>
<ul>
 <li>
  Protein binds alternate/several cofactors
  <pre>&lt;comment type="cofactor"&gt;
  &lt;cofactor evidence="1"&gt;
    &lt;name&gt;Mg(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:18420"/&gt;
  &lt;/cofactor&gt;
  &lt;cofactor evidence="1"&gt;
    &lt;name&gt;Co(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:48828"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="1"&gt;Binds 2 divalent ions per subunit (magnesium or cobalt).&lt;/text&gt;
&lt;/comment&gt;
&lt;comment type="cofactor"&gt;
  &lt;cofactor evidence="1"&gt;
    &lt;name&gt;K(+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29103"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="1"&gt;Binds 1 potassium ion per subunit.&lt;/text&gt;
&lt;/comment&gt;
...
&lt;evidence key="1" type="ECO:0000255"&gt;
  &lt;source&gt;
    &lt;dbReference type="HAMAP-Rule" id="MF_00086"/&gt;
  &lt;/source&gt;
&lt;/evidence&gt;
</pre>
 </li>
</ul>
<ul>
 <li>
  Isoforms
  <pre>&lt;comment type="cofactor"&gt;
  &lt;molecule&gt;Isoform 1&lt;/molecule&gt;
  &lt;cofactor evidence="9"&gt;
    &lt;name&gt;Zn(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29105"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="9"&gt;Isoform 1 binds 3 Zn(2+) ions.&lt;/text&gt;
&lt;/comment&gt;
&lt;comment type="cofactor"&gt;
  &lt;molecule&gt;Isoform 2&lt;/molecule&gt;
  &lt;cofactor evidence="9"&gt;
    &lt;name&gt;Zn(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29105"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="9"&gt;Isoform 2 binds 2 Zn(2+) ions.&lt;/text&gt;
&lt;/comment&gt;
...
&lt;evidence key="9" type="ECO:0000269"&gt;
  &lt;source&gt;
    &lt;dbReference type="PubMed" id="16683188"/&gt;
  &lt;/source&gt;
&lt;/evidence&gt;
</pre>
 </li>
</ul>
<ul>
 <li>
  Chains
  <pre>&lt;comment type="cofactor"&gt;
  &lt;molecule&gt;Serine protease NS3&lt;/molecule&gt;
  &lt;cofactor evidence="13"&gt;
    &lt;name&gt;Zn(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29105"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="13"&gt;Binds 1 zinc ion.&lt;/text&gt;
&lt;/comment&gt;
&lt;comment type="cofactor"&gt;
  &lt;molecule&gt;Non-structural protein 5A&lt;/molecule&gt;
  &lt;cofactor evidence="3"&gt;
    &lt;name&gt;Zn(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29105"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="3"&gt;Binds 1 zinc ion in the NS5A N-terminal domain.&lt;/text&gt;
&lt;/comment&gt;
...
&lt;evidence key="3" type="ECO:0000250"/&gt;
...
&lt;evidence key="13" type="ECO:0000269"&gt;
  &lt;source&gt;
    &lt;dbReference type="PubMed" id="9060645"/&gt;
  &lt;/source&gt;
&lt;/evidence&gt;
</pre>
 </li>
</ul>
<ul>
 <li>
  Cofactor unknown
  <pre>&lt;comment type="cofactor"&gt;
  &lt;text evidence="1"&gt;Does not require a metal cofactor.&lt;/text&gt;
&lt;/comment&gt;
...
&lt;evidence key="1" type="ECO:0000269"&gt;
  &lt;source&gt;
    &lt;dbReference type="PubMed" id="24450804"/&gt;
  &lt;/source&gt;
&lt;/evidence&gt;
</pre>
 </li>
</ul>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 We introduced a new
 <code>
  cofactor
 </code>
 property to list individual cofactors as ChEBI resource descriptions. As for other types of annotations, an optional
 <code>
  sequence
 </code>
 property may describe the molecule to which the annotation applies and an optional
 <code>
  rdfs:comment
 </code>
 property may provide additional information.
</p>
<p>
 Examples:
</p>
<p>
 <em>
  Note: Evidence tags are omitted from the examples to make it easier to read them. They are represented as for all other types of annotations by reification of the concerned statements.
 </em>
</p>
<ul>
 <li>
  Protein binds alternate/several cofactors
  <pre>uniprot:Q5M434
  up:annotation SHA:1, SHA:2 ;
  ...
SHA:1
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Binds 2 divalent ions per subunit (magnesium or cobalt)." ;
  up:cofactor &lt;http://purl.obolibrary.org/obo/CHEBI_18420&gt;,
              &lt;http://purl.obolibrary.org/obo/CHEBI_48828&gt; .
SHA:2
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Binds 1 potassium ion per subunit." ;
  up:cofactor &lt;http://purl.obolibrary.org/obo/CHEBI_29103&gt; ;
</pre>
 </li>
</ul>
<ul>
 <li>
  Isoforms
  <pre>uniprot:O15304
  up:annotation SHA:1, SHA:2 ;
  ...
SHA:1
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Isoform 1 binds 3 Zn(2+) ions." ;
  up:cofactor &lt;http://purl.obolibrary.org/obo/CHEBI_29105&gt; ;
  up:sequence isoform:O15304-1 .
SHA:2
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Isoform 2 binds 2 Zn(2+) ions." ;
  up:cofactor &lt;http://purl.obolibrary.org/obo/CHEBI_29105&gt; ;
  up:sequence isoform:O15304-2 .
</pre>
 </li>
</ul>
<ul>
 <li>
  Chains
  <pre>uniprot:P26662
  up:annotation SHA:1, SHA:2 ;
  ...
SHA:1
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Binds 1 zinc ion." ;
  up:cofactor &lt;http://purl.obolibrary.org/obo/CHEBI_29105&gt; ;
  up:sequence annotation:PRO_0000037644 .
SHA:2
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Binds 1 zinc ion in the NS5A N-terminal domain." ;
  up:cofactor &lt;http://purl.obolibrary.org/obo/CHEBI_29105&gt; ;
  up:sequence annotation:PRO_0000037647 .
</pre>
 </li>
</ul>
<ul>
 <li>
  Cofactor unknown
  <pre>uniprot:A9CEQ7
  up:annotation SHA:1 ;
  ...
SHA:1
  rdf:type up:Cofactor_Annotation ;
  rdfs:comment "Does not require a metal cofactor." ;
</pre>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03673">
   2-aminoadipic 2-oxoadipic aciduria
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04227">
   Adams-Oliver syndrome 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04235">
   Analbuminemia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04246">
   Autoinflammation with infantile enterocolitis
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04236">
   Cerebellar ataxia, areflexia, pes cavus, optic atrophy, and sensorineural hearing loss
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04247">
   Ciliary dyskinesia, primary, 30
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04243">
   Combined oxidative phosphorylation deficiency 22
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04244">
   Deafness, autosomal dominant, 65
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04231">
   Dicarboxylic aminoaciduria
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04239">
   Ectodermal dysplasia/short stature syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04230">
   Fanconi renotubular syndrome 4 with maturity-onset diabetes of the young
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04233">
   Focal segmental glomerulosclerosis 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04226">
   Glucocorticoid resistance, generalized
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04238">
   Hennekam lymphangiectasia-lymphedema syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04229">
   Hyperphosphatasia with mental retardation syndrome 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04228">
   Hypogonadotropic hypogonadism 22 with or without anosmia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04241">
   Hypoplasia or aplasia of tibia with polydactyly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04221">
   Immunodeficiency 28
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04222">
   Immunodeficiency 29
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04223">
   Immunodeficiency 30
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04224">
   Immunodeficiency 31A
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04225">
   Immunodeficiency 27B
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04234">
   Microcephaly, short stature, and impaired glucose metabolism
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04245">
   Muscular dystrophy-dystroglycanopathy limb-girdle C7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04232">
   Neutropenia, severe congenital 6, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04237">
   Polymicrogyria, bilateral temporooccipital
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04242">
   Spinocerebellar ataxia 40
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Dysalbuminemic hyperthyroxinemia -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01565">
   Hyperthyroxinemia, familial dysalbuminemic
  </a>
 </li>
 <li>
  Hennekam lymphangiectasia-lymphedema syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02804">
   Hennekam lymphangiectasia-lymphedema syndrome 1
  </a>
 </li>
 <li>
  Hyperthyroxinemia dystransthyretinemic euthyroidal -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01785">
   Hyperthyroxinemia, dystransthyretinemic
  </a>
 </li>
 <li>
  IRF8 deficiency, autosomal dominant -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03810">
   Immunodeficiency 32A
  </a>
 </li>
 <li>
  IRF8 deficiency, autosomal recessive -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03811">
   Immunodeficiency 32B
  </a>
 </li>
 <li>
  Megalencephaly-polymicrogyria-polydactyly-hydrocephalus syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03625">
   Megalencephaly-polymicrogyria-polydactyly-hydrocephalus syndrome 1
  </a>
 </li>
 <li>
  Mendelian susceptibility to mycobacterial disease -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01964">
   Immunodeficiency 27A
  </a>
 </li>
 <li>
  Mycobacteriosis atypical X-linked 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03091">
   Immunodeficiency 34
  </a>
 </li>
 <li>
  Optic atrophy 7 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02531">
   Optic atrophy 7 with or without auditory neuropathy
  </a>
 </li>
 <li>
  Protein-tyrosine kinase 2 deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02224">
   Immunodeficiency 35
  </a>
 </li>
 <li>
  STAT1 deficiency complete -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03106">
   Immunodeficiency 31B
  </a>
 </li>
 <li>
  X-linked familial atypical micobacteriosis 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02445">
   Immunodeficiency 33
  </a>
 </li>
</ul>