---
title: UniProt release 2015_03
type: releaseNotes
date: 2015-03-04
---

# Headline

## Regulation of translation initiation through folding

Many physiopathological events, such as stress or nutrient deprivation, induce rapid changes in cellular protein levels. In these cases, cells preferentially use translational control of existing mRNAs over transcriptional control, since the latter generates a slower response. Translation can be divided into 4 steps, initiation, elongation, termination, and ribosome recycling, but most regulation occurs at the initiation level.  
In eukaryotes, translation initiation involves recruitment of the 40S ribosome to mRNA by the eukaryotic initiation factor 4F (eIF4F) complex. This complex is composed of [eIF4E](http://www.uniprot.org/uniprot/?query=gene:eIF4E+and+reviewed:yes), which binds to the mRNA [5' cap structure](http://en.wikipedia.org/wiki/Five-prime%5Fcap), eIF4A, an RNA helicase and [eIF4G](http://www.uniprot.org/uniprot/?query=gene:eIF4G+and+reviewed:yes), a scaffolding protein. Availability of eIF4E is rate-limiting in this process and it is an important target for control. Under stress or starvation conditions, when translation has to be rapidly repressed, eIF4E binding proteins (4E-BPs) interact with eIF4E outcompeting eIF4G, hence preventing eIF4F assembly and cap-dependent translation initiation. 3 [4E-BPs](http://www.uniprot.org/uniprot/?query=gene:eif4ebp%2A+AND+taxonomy:40674+and+reviewed:yes) have been identified in mammals. 4E-BP2 ( [EIF4EBP2](http://www.uniprot.org/uniprot/?query=gene:eif4ebp2+and+reviewed:yes) ) is one of them. It is an intrinsically disordered protein (IDP) that contains several phosphorylation sites. In its unphosphorylated state, 4E-BP2 interacts with eIF4E via 2 domains: a YXXXXLΦ motif ( [residues 54 through 60](http://www.uniprot.org/blast/?about=Q13542%5B54-60%5D&key=Motif) ) and a [secondary dynamic motif](http://www.ncbi.nlm.nih.gov/pubmed/24207126) ( [residues 78 through 82](http://www.uniprot.org/blast/?about=Q13542%5B78-82%5D&key=Motif) ). The unphosphorylated (or minimally phosphorylated), eIF4E-binding form of EIF4EBP2 is unstable and targeted for degradation via the ubiquitin-proteasome pathway. By contrast, highly phosphorylated 4E-BP2 is very stable, but only weakly binds to eIF4E and hence can be outcompeted by eIF4G, allowing translation to occur.

How does phosphorylation regulate 4E-BP2 interaction with eIF4E and its stability? It has been [recently shown](http://www.ncbi.nlm.nih.gov/pubmed/25533957) that phosphorylation induces a widespread disorder-to-order transition occurring in 2 steps. First, phosphorylation at Thr-37 and Thr-46 by [MTOR](http://www.uniprot.org/uniprot/?query=gene:mtor+AND+taxonomy:40674+and+reviewed:yes) induces folding of residues Pro-18 to Arg-62 into a four-stranded β-domain that sequesters the helical YXXXXLΦ motif into a partially buried β-strand, blocking accessibility to eIF4E. The folding also protects Lys-57 from ubiquitination, preventing proteasomal degradation. This ordered structure is further stabilized by phosphorylation at Ser-65, Thr-70 and Ser-83. The fully phosphorylated protein has an affinity for eIF4E 4,000 fold lower than the unphosphorylated form. This observation implies that binding must be coupled to unfolding in order to free the YXXXXLΦ motif, and it is indeed what is experimentally observed. When the phosphorylated form binds eIF4E, it undergoes an order-to-disorder transition, as suggested by NMR spectra that are similar to those of the unphosphorylated form.

Although it has long been suspected that the function of IDPs may be controlled by post-translational modifications (PTMs), this is the first report experimentally showing how a PTM can fold an entire domain. This new data have been annotated into UniProtKB/Swiss-Prot and as of this release, the updated [EIF4EBP2 entry](http://www.uniprot.org/uniprot/?query=gene:eif4ebp2+and+reviewed:yes) is publicly available.

# UniProtKB news

## New proteomics mapping files

Mappings of UniProt Knowledgebase (UniProtKB) human sequences to identified human peptides from public mass spectrometry (MS) proteomics repositories can now be found in the new dedicated 'proteomics\_mapping' directory on the [UniProt FTP site](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/proteomics%5Fmapping/) together with a [description of how the mappings were generated](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/proteomics%5Fmapping/README). The mappings are based on our analysis of the content of those MS proteomics repositories that openly share with us their data and quality metrics concerning peptide identifications.

Mass spectrometry provides direct experimental evidence for the existence of proteins and these new peptide mappings greatly increase the proportion of human sequences in UniProtKB whose existence is supported by experimental proteomics data. The human reference proteome currently contains 89383 sequences and our analysis provides mass spectrometry evidence for 68229 of those sequences.

In future UniProt releases, we expect to add data from more MS proteomics repositories and additional species. We very much welcome the feedback of the community on our efforts.

## New FTP repository for reference proteomes

Based on a gene-centric perspective, UniProt Knowledgebase (UniProtKB) starts to provide [data sets for reference proteomes](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/reference%5Fproteomes/README), whose repository can be found at the new [reference\_proteomes](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/reference%5Fproteomes/) directory.

As of release 2015\_03, it encompasses 1933 species distributed in Eukaryota, Archaea and Bacteria. Viruses will be added in the next release.

## Removal of the cross-references to PhosSite

Cross-references to PhosSite have been removed.

## Removal of the cross-references to PptaseDB

Cross-references to PptaseDB have been removed.

## Changes to the [controlled vocabulary of human diseases](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease)

New diseases:

-   [Aortic aneurysm, familial thoracic 9](http://www.uniprot.org/diseases/DI-04293)
-   [Autoimmune lymphoproliferative syndrome 5](http://www.uniprot.org/diseases/DI-04302)
-   [Bleeding disorder, platelet-type 19](http://www.uniprot.org/diseases/DI-04294)
-   [Cardiac conduction disease with or without dilated cardiomyopathy](http://www.uniprot.org/diseases/DI-04282)
-   [Catel-Manzke syndrome](http://www.uniprot.org/diseases/DI-04301)
-   [Epileptic encephalopathy, early infantile, 27](http://www.uniprot.org/diseases/DI-04289)
-   [Familial cold autoinflammatory syndrome 4](http://www.uniprot.org/diseases/DI-04279)
-   [Generalized epilepsy with febrile seizures plus 9](http://www.uniprot.org/diseases/DI-04300)
-   [Immunodeficiency 38](http://www.uniprot.org/diseases/DI-04280)
-   [Leukodystrophy, hypomyelinating, 9](http://www.uniprot.org/diseases/DI-04288)
-   [Mental retardation, autosomal recessive 46](http://www.uniprot.org/diseases/DI-04283)
-   [Microcephaly and chorioretinopathy, autosomal recessive, 2](http://www.uniprot.org/diseases/DI-04299)
-   [Mitochondrial complex III deficiency, nuclear 9](http://www.uniprot.org/diseases/DI-04284)
-   [Macular degeneration, early-onset](http://www.uniprot.org/diseases/DI-04285)
-   [Macular dystrophy, vitelliform, 4](http://www.uniprot.org/diseases/DI-04286)
-   [Macular dystrophy, vitelliform, 5](http://www.uniprot.org/diseases/DI-04287)
-   [Macular dystrophy with central cone involvement](http://www.uniprot.org/diseases/DI-04295)
-   [Nemaline myopathy 10](http://www.uniprot.org/diseases/DI-04292)
-   [Ovarian dysgenesis 4](http://www.uniprot.org/diseases/DI-04296)
-   [Perrault syndrome 5](http://www.uniprot.org/diseases/DI-04281)
-   [Polyendocrine-polyneuropathy syndrome](http://www.uniprot.org/diseases/DI-04291)
-   [Retinal dystrophy and obesity](http://www.uniprot.org/diseases/DI-04298)
-   [Retinal dystrophy, juvenile cataracts, and short stature syndrome](http://www.uniprot.org/diseases/DI-04303)
-   [Spinocerebellar ataxia, autosomal recessive, 17](http://www.uniprot.org/diseases/DI-04290)
-   [Temple-Baraitser syndrome](http://www.uniprot.org/diseases/DI-04297)

Modified disease:

-   Vitelliform macular dystrophy 2 -&gt; [Macular dystrophy, vitelliform, 2](http://www.uniprot.org/diseases/DI-01124)

Deleted diseases:

-   Amelogenesis imperfecta and gingival fibromatosis syndrome
-   Glycogen storage disease 14
-   Ichthyosis, autosomal recessive, with hypotrichosis
-   Loeys-Dietz syndrome 2A
-   Loeys-Dietz syndrome 2B
-   Leigh syndrome, X-linked
-   Mental retardation, X-linked 59

## Changes to [keywords](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/keywlist)

New keyword:

-   [Proteomics identification](http://www.uniprot.org/keywords/KW-1267)

# UniParc news

## UniParc cross-references with proteome identifier and component

The UniParc XML format uses `dbReference` elements to represent cross-references to external database records that contain the same sequence as the UniParc record. Additional information about an external database record is provided with different types of `property` child elements. We have introduced two new types for cross-references to external database records from which [UniProt proteomes](http://www.uniprot.org/faq/proteome) are derived: The type `"proteome_id"` shows the identifier of the corresponding [UniProt proteome](http://www.uniprot.org/proteomes) and the type `"component"` the genomic component which encodes the protein. As a first step, we have added this information to bacterial [ENA](http://www.ebi.ac.uk/ena/) records.

Example:

    <entry dataset="uniparc">
        <accession>UPI0000131B78</accession>
        <dbReference type="EMBL" id="AAK44239" version_i="1" active="Y" version="1" created="2003-03-12" last="2014-11-23">
            <property type="NCBI_GI" value="13879058"/>
            <property type="NCBI_taxonomy_id" value="83331"/>
            <property type="protein_name" value="serine/threonine protein kinase"/>
            <property type="gene_name" value="MT0017"/>
            <property type="proteome_id" value="UP000001020"/>
            <property type="component" value="Chromosome"/>
        </dbReference>
        <dbReference type="EMBL" id="ABQ71734" version_i="1" active="Y" version="1" created="2007-07-09" last="2014-11-23">
            <property type="NCBI_GI" value="148503925"/>
            <property type="NCBI_taxonomy_id" value="419947"/>
            <property type="protein_name" value="serine/threonine protein kinase"/>
            <property type="gene_name" value="pknB"/>
            <property type="proteome_id" value="UP000001988"/>
            <property type="component" value="Chromosome"/>
        </dbReference>
        ...
        <dbReference type="EMBL_CON" id="EFD75652" version_i="1" active="Y" version="2" created="2011-12-05" last="2014-11-23">
            <property type="NCBI_taxonomy_id" value="537209"/>
            <property type="protein_name" value="transmembrane serine/threonine-protein kinase B pknB"/>
            <property type="gene_name" value="TBIG_00439"/>
            <property type="proteome_id" value="UP000004676"/>
            <property type="component" value="Unassembled WGS sequence"/>
        </dbReference>
        ...
    </entry>

This change did not affect the [UniParc XSD](https://ftp.uniprot.org/pub/databases/uniprot/current_release/uniparc/uniparc.xsd), but may nevertheless require code changes.

# UniProt RDF news

## UniProt RDF files compressed with XZ instead of gzip

The UniProt RDF distribution has been available on the [UniProt FTP site](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/rdf/) as gzip compressed RDF/XML files since 2008. We have now changed the compression algorithm from gzip to [XZ](http://tukaani.org/xz/), which has a number of features that make it a better choice for the UniProt RDF data:

-   It reduces the file size by approximately 23%, which improves FTP download time.
-   It can be decompressed in parallel, which can give faster decompression rates on current hardware with a minimum of 6-8 CPU cores.
-   It allows [random access](http://tukaani.org/xz/xz-javadoc/index.html?org/tukaani/xz/SeekableXZInputStream.html) .

## Replacement of UniProt RDF file go.rdf with go.owl

The UniProt RDF distribution that is available on the [UniProt FTP site](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/rdf/) contained a `go.rdf` file that has been replaced with a `go.owl` file that contains a subset of the official `go.owl` distribution of the Gene Ontology consortium, which is taken as a snapshot that is in sync with the GO annotations in the UniProt Knowledgebase.

In practical terms this means:

-   Scripts downloading `go.rdf` must be changed to download `go.owl` instead.
-   The URL pattern `http://purl.uniprot.org/go/` has been replaced by `http://purl.obolibrary.org/obo/GO_`
-   Querying for a GO term linked to a UniProt record using SPARQL can be done with:  
    [select ?protein ?go where {?protein up:classifiedWith ?go. ?go a owl:Class}](http://beta.sparql.uniprot.org/sparql/?format=html&query=PREFIX+owl%3A%3Chttp%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23%3E+%0D%0APREFIX+up%3A%3Chttp%3A%2F%2Fpurl.uniprot.org%2Fcore%2F%3E+%0D%0Aselect+%3Fprotein+%3Fgo+where+%7B%3Fprotein+up%3AclassifiedWith+%3Fgo+.%3Fgo+a+owl%3AClass%7D)
