---
title: UniProt release 2023_05
type: releaseNotes
date: 2023-11-08
---

# Headline

## Hush, little ribosome

Often when we think of translation, we often focus so intently on the protein we are interested in that we forget that, at the microscopic scale, the machinery involved to produce proteins is huge. Most cells contain millions of ribosomes and egg cells contain even orders of magnitude more. Translation is one of the most energy-costly processes. [It has been estimated to require ~5 ATP per peptide bond or ~2,300 ATP per typical protein synthesized](https://pubmed.ncbi.nlm.nih.gov/34983947/). That is why the whole process has to be tightly regulated not only at the level of single proteins, but also on a large scale. For instance, egg cells have to quietly await fertilization, to save energy and protect mRNAs and ribosomes, and to be ready for full-blown action in the first steps of embryogenesis. Translational repression has been extensively studied from the RNA standpoint. At the mRNA level, several mechanisms may account for translation dampening, often linked to 3'UTR sequences, including [shortening the poly(A) tail](https://pubmed.ncbi.nlm.nih.gov/36894378/), or [interfering with the formation of the eIF4F complex](https://pubmed.ncbi.nlm.nih.gov/10635326/). However, the contribution of ribosomes to the global translation repression process was not investigated [until recently](https://pubmed.ncbi.nlm.nih.gov/36653451/).

In zebrafish, as in other vertebrates, translation is repressed in eggs. It resumes about 3 hours after fertilization. At the ribosome level, this is reflected by the presence of almost exclusively monosomes before fertilization. The proportion of polysomes starts to increase from 3 to 6 hours after fertilization onwards. Four proteins have been shown to be associated with the "dormant" ribosomes (monosomes): death-associated protein-like 1 homolog ([dap1b](https://www.uniprot.org/uniprotkb/Q9I9N0)), intracellular hyaluronan-binding protein 4 ([habp4](https://www.uniprot.org/uniprotkb/Q5XJA5)), elongation factor 2b ([eef2b](https://www.uniprot.org/uniprotkb/Q6P3J5)) and eukaryotic translation initiation factor 5A-1 ([eif5a](https://www.uniprot.org/uniprotkb/Q6NX89)), forming two modules, eef2b-habp4 and eif5a-dap1b, that bind distinct sites on the same ribosome.

In active ribosomes, eef2 mediates ribosomal translocation along the mRNA and transiently interacts with the tRNA-mRNA complex at the aminoacyl (A) site of the ribosome. In dormant ribosomes, habp4 binds to the mRNA-entry channel and sequesters eef2 at the A site and blocks the interaction sites of the tRNA-mRNA complex. eif5a normally promotes translation elongation and termination and binds between the exit (E) and peptidyl (P) sites of the ribosome. In dormant ribosomes, dap1b sits in the polypeptide exit tunnel (PET), where newly formed proteins normally emerge, and forms a kind of plug. Its C-terminus occupies the same position as the C-terminal residue of a nascent peptide chain and forms a hydrogen bond with the hypusine residue of eIF5a, a residue that is essential for eif5a function.

Although habp4 binding to the mRNA-entry channel is incompatible with translation, the current hypothesis is that its main function is instead to stabilize ribosomes, and that it is dap1b that predominantly carries the translation repression function.

Can these observations be extended to other animals? Experiments carried out in _Xenopus laevis_ gave similar results. Furthermore, _in vitro_ translation assays in rabbit reticulocyte lysates showed that not only recombinant zebrafish dap1b, but also orthologous proteins in other organisms, including Xenopus, human (where the orthologous gene is called SERBP1), and even _C.elegans_, all repress translation.

If contrary to ribosomes you did not fall asleep while reading this headline, you can consult the UniProtKB/Swiss-Prot entries for [habp4](https://www.uniprot.org/uniprotkb?query=gene%3Ahabp4+AND+reviewed%3Atrue), [dap1b/SERBP1](https://www.uniprot.org/uniprotkb?query=gene%3Aserbp1+AND+reviewed%3Atrue+OR+accession%3AQ9I9N0+OR+accession%3AA3KMT2+OR+accession%3AA3KMU5+OR+accession%3AQ9I9N1), as well as [eef2b/EEF2](https://www.uniprot.org/uniprotkb?query=id%3AEF2_*+AND+reviewed%3Atrue+AND+taxonomy_id%3A7742) and [eif5a](https://www.uniprot.org/uniprotkb?query=gene%3AEIF5A+AND+reviewed%3Atrue+AND+taxonomy_id%3A7742) in our new release.

# UniProtKB news

## Cross-references to Pumba

Cross-references have been added to Pumba, a database of electrophoretic reference migration patterns.

Pumba is available at [https://pumba.dcsr.unil.ch](https://pumba.dcsr.unil.ch).

The format of the explicit links is:

| Resource abbreviation | Pumba |
| Resource identifier | UniProtKB accession number |


Example: [P11274](https://www.uniprot.org/uniprotkb/P11274)

[Show all entries having a cross-reference to Pumba.](https://www.uniprot.org/uniprotkb/?query=database:pumba)


### Text format

Example: [P11274](https://rest.uniprot.org/uniprotkb/P11274.txt)

```
DR   Pumba; P11274; -.
```

### XML format

Example: [P11274](https://rest.uniprot.org/uniprotkb/P11274.xml)

```
<dbReference type="Pumba" id="P11274"/>
```

### RDF format

Example: [P11274](https://rest.uniprot.org/uniprotkb/P11274.ttl)

```
uniprot:P11274
  rdfs:seeAlso <http://purl.uniprot.org/pumba/P11274> .
<http://purl.uniprot.org/pumba/P11274>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/Pumba> .
```


## Changes to the [controlled vocabulary of human diseases](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease)

New diseases:

* [Cerebroretinal microangiopathy with calcifications and cysts 3](https://www.uniprot.org/diseases/DI-06680)
* [Charcot-Marie-Tooth disease, axonal, 2GG](https://www.uniprot.org/diseases/DI-06681)
* [Combined low LDL and fibrinogen](https://www.uniprot.org/diseases/DI-06676)
* [Intellectual developmental disorder, autosomal recessive 79](https://www.uniprot.org/diseases/DI-06682)
* [Mitochondrial complex V deficiency, nuclear type 4A](https://www.uniprot.org/diseases/DI-06674)
* [Mitochondrial complex V deficiency, nuclear type 7](https://www.uniprot.org/diseases/DI-06675)
* [Osteopetrosis, autosomal recessive 9](https://www.uniprot.org/diseases/DI-06679)
* [Pulmonary fibrosis, and/or bone marrow failure, telomere-related, 7](https://www.uniprot.org/diseases/DI-06677)
* [Pulmonary fibrosis, and/or bone marrow failure, telomere-related, 8](https://www.uniprot.org/diseases/DI-06678)
* [RECON progeroid syndrome](https://www.uniprot.org/diseases/DI-06683)

## Changes to the [controlled vocabulary for PTMs](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/ptmlist)

New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):

* L-cysteine coenzyme A disulfide
* Sulfocysteine

## Changes in [subcellular location controlled vocabulary](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/subcell)

New subcellular locations:

* [Rhabdomere](https://www.uniprot.org/locations/SL-0551)
* [Rhabdomere membrane](https://www.uniprot.org/locations/SL-0552)

# UniProt API news

## Additional return fields for cross-references in TSV format

The format of our existing [UniProtKB return fields for cross-references](https://www.uniprot.org/help/return_fields_databases) in a result table consists of the primary identifier of the external resource and, optionally, the [UniProt isoform identifier in square brackets](https://www.uniprot.org/help/isoform_crossreferences). When an entry has several cross-references to the same external resources, these cross-references are separated by a semi-colon.

We have now introduced additional UniProtKB return fields for all cross-references that have additional identifiers or other information. The format of an individual cross-reference is identical to the format used in the UniProtKB text format (with multiple fields being separated by semi-colons). In order to continue to separate multiple cross-references to the same external resource with a semi-colon, each individual cross-reference is enclosed in double quotes.

Example: [A0JNW5 Ensembl cross-references](https://rest.uniprot.org/uniprotkb/A0JNW5.tsv?fields=xref_ensembl)

Existing return field `xref_ensembl`:

```
ENST00000279907.12 [A0JNW5-1];ENST00000356828.7 [A0JNW5-2];
```

New return field `xref_ensembl_full`:

```
"ENST00000279907.12; ENSP00000279907.7; ENSG00000111647.13. [A0JNW5-1]";"ENST00000356828.7; ENSP00000349285.3; ENSG00000111647.13. [A0JNW5-2]";
```

Example: [A0JNW5 InterPro cross-references](https://rest.uniprot.org/uniprotkb/A0JNW5.tsv?fields=xref_interpro)

Existing return field `xref_interpro`:

```
IPR026728;IPR026854;
```

New return field `xref_interpro_full`:

```
"IPR026728; BLTP3A/B.";"IPR026854; VPS13-like_N."
```
