---
title: UniProt release 2024_06
type: releaseNotes
date: 2024-11-27
---

# Headline

## What happens when ribosomes crash...

Stalled ribosomes are a serious problem for all cells, be they bacteria, archaea or eukaryote. Ribosomes can stall for a myriad of reasons; chemically damaged nucleotides, damaged or truncated mRNA, when a series of rare codons are encountered, and at certain stalling sites, where they may be used to control the translation of polycistronic mRNAs. When a translating ribosome stalls, the ribosome that follows collides with it, generating stalled disomes. If the mRNA is highly translated, more collisions can occur, yielding higher numbers of ribosomes. In all cases, the pool of active ribosomes is reduced, and potentially toxic peptides can be produced. So it is crucial for cells to detect and liberate stalled ribosomes. How is it done? This question has been answered in recent papers, which have studied "_E.coli_": https://pubmed.ncbi.nlm.nih.gov/35264790/ and
"_Bacillus subtilis_": https://pubmed.ncbi.nlm.nih.gov/35264791,38177497. Interestingly, both bacteria use proteins containing an SMR domain to detect the stalled disomes, but the proteins function quite differently.

In _E.coli_, and most other [gammaproteobacteria](https://www.uniprot.org/taxonomy/1236), the [SmrB protein](https://www.uniprot.org/uniprotkb/P0A8B2) (formerly known as YfcN) binds to stalled/collided disomes, while in _B.subtilis_ it is the "MutS2 protein": /uniprot/P94545. In bacteria, stalled disomes have a slightly different structure than actively translating or hibernating disomes, creating a unique interface, to which both SmrB (in _E.coli_) and MutS2 (in _B. subtilis_) bind via their SMR domains.

[SmrB in _E.coli_](https://www.uniprot.org/uniprotkb/P0A8B2) uses its N-terminal extension to grab onto the uS2 protein in the small subunit of the stalled disome. Once properly positioned, SmrB cuts the mRNA between the stalled and trailing ribosomes. The endonuclease catalytic activity relies upon a DxH motif within the SMR domain. After cleavage, the trailing ribosome can be directly recycled and the incomplete nascent chains tagged for degradation by a classical ribosome rescue system, while additional nuclease activity is needed for the stalled ribosome before it is taken charge of by the rescue system. SmrB's key role in this process is supported by the observation that, in its absence, bacteria become hypersensitive to antibiotics which increase stalling at specific mRNA sites (for example erythromycin), and they are also more likely to translate through strong stalling motifs, which can lead to incorrect translation products.

[_B.subtilis_ MutS2](https://www.uniprot.org/uniprotkb/P94545) is longer than SmrB, with a MutS-type ATPase domain, a KOW-like domain and the SMR domain. Both SMR and KOW domains bind to the stalled disome interface, with the SMR domain oriented differently compared to what was observed with the SmrB protein. Another difference is that there is no evidence that the SMR domain has nuclease activity. The DxH motif, crucial for _E.coli_ SmrB activity, is DxR in MutS2. While the exact catalytic activity is yet undefined, ATP hydrolysis, catalyzed by the ATPase domain, is essential to split the stalled ribosome into the 30S and 50S subunits. At this point the ribosome rescue systems can take over, releasing the 50S ribosomal subunit from its stalled peptide and liberating it to reinitiate translation. Deletion of _mutS2_ also makes the bacterium more sensitive to ribosome-targeting antibiotics.

Phylogenetically SmrB and MutS2 proteins are limited to [gammaproteobacteria](https://www.uniprot.org/taxonomy/1236) and [Bacillota](https://www.uniprot.org/taxonomy/1239), respectively; it will be interesting to see if other bacteria use the same or different domains/proteins, and how their organellar descendants, mitochondria and plastids, handle stalled ribosomes.

[_E.coli_ SmrB](https://www.uniprot.org/uniprotkb/P0A8B2) and [_B.subtilis_ MutS2](https://www.uniprot.org/uniprotkb/P94545) have been updated and are available as of this release.

# UniProtKB news



## Cross-references to FunFam

Cross-references have been added to FunFam, the CATH database of functional families.

FunFam is available at [https://www.cathdb.info](https://www.cathdb.info).

The format of the explicit links is:

| Resource abbreviation | FunFam |
| Resource identifier | Resource identifier |
| Optional information 1 | Functional family name |
| Optional information 2 | Number of hits |

Example: [Q12406](https://www.uniprot.org/uniprotkb/Q12406)

[Show all entries having a cross-reference to FunFam.](https://www.uniprot.org/uniprotkb/?query=database:funfam)


### Text format

Example: [Q12406](https://rest.uniprot.org/uniprotkb/Q12406.txt)

```
DR   FunFam; 3.30.420.40:FF:000343; Chromatin remodeling Snf/Swi complex subunit; 1.

```

### XML format

Example: [Q12406](https://rest.uniprot.org/uniprotkb/Q12406.xml)

```
<dbReference type="FunFam" id="3.30.420.40:FF:000343">
  <property type="entry name" value="Chromatin remodeling Snf/Swi complex subunit"/>
  <property type="match status" value="1"/>
</dbReference>
```

### RDF format

Example: [Q12406](https://rest.uniprot.org/uniprotkb/Q12406.ttl)

```
uniprot:Q12406
  rdfs:seeAlso <http://purl.uniprot.org/funfam/3.30.420.40:FF:000343> .
<http://purl.uniprot.org/funfam/3.30.420.40:FF:000343>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/FunFam> ;
  rdfs:comment "Chromatin remodeling Snf/Swi complex subunit" ;
  up:signatureSequenceMatch <http://purl.uniprot.org/isoforms/Q12406-1#FunFam_3.30.420.40:FF:000343_match_1> .
```

## Cross-references to AntiFam

Cross-references have been added to the AntiFam database, a resource to identify spurious protein predictions.

AntiFam is available at [https://www.ebi.ac.uk/interpro/entry/antifam](https://www.ebi.ac.uk/interpro/entry/antifam).

The format of the explicit links is:

| Resource abbreviation | AntiFam |
| Resource identifier | Resource identifier |
| Optional information 1 | Entry name |

Example: [A0A0N7KKX8](https://www.uniprot.org/uniprotkb/A0A0N7KKX8)

[Show all entries having a cross-reference to AntiFam.](https://www.uniprot.org/uniprotkb/?query=database:antifam)


### Text format

Example: [A0A0N7KKX8](https://rest.uniprot.org/uniprotkb/A0A0N7KKX8.txt)

```
DR   AntiFam; ANF00226; Spurious_ORF_226.
```

### XML format

Example: [A0A0N7KKX8](https://rest.uniprot.org/uniprotkb/A0A0N7KKX8.xml)

```
<dbReference type="AntiFam" id="ANF00226">
  <property type="entry name" value="Spurious_ORF_226"/>
</dbReference>
```

### RDF format

Example: [A0A0N7KKX8](https://rest.uniprot.org/uniprotkb/A0A0N7KKX8.ttl)

```
uniprot:A0A0N7KKX8
  rdfs:seeAlso <http://purl.uniprot.org/antifam/ANF00226> .
<http://purl.uniprot.org/antifam/ANF00226>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/AntiFam> ;
  rdfs:comment "Spurious_ORF_226" .
```

## Changes to the [controlled vocabulary of human diseases](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease)

New diseases:

* [Auroneurodental syndrome](https://www.uniprot.org/diseases/DI-06913)
* [Autoinflammation with arthritis and vasculitis](https://www.uniprot.org/diseases/DI-06922)
* [Cardiomyopathy, dilated, 2K](https://www.uniprot.org/diseases/DI-06924)
* [Deafness, autosomal recessive, 125](https://www.uniprot.org/diseases/DI-06921)
* [Ehlers-Danlos syndrome, classic-like, 3](https://www.uniprot.org/diseases/DI-06915)
* [Immunodeficiency 121 with autoinflammation](https://www.uniprot.org/diseases/DI-06912)
* [Immunodeficiency 122](https://www.uniprot.org/diseases/DI-06920)
* [Immunodeficiency 123 with HPV-related verrucosis](https://www.uniprot.org/diseases/DI-06931)
* [Immunodeficiency 125](https://www.uniprot.org/diseases/DI-06932)
* [Multiple mitochondrial dysfunctions syndrome 9B](https://www.uniprot.org/diseases/DI-06923)
* [Muscular dystrophy, limb-girdle, autosomal recessive 29](https://www.uniprot.org/diseases/DI-06918)
* [Neurodevelopmental disorder with characteristic facial and ectodermal features and tetraparesis 1](https://www.uniprot.org/diseases/DI-06930)
* [Neurodevelopmental disorder with hypotonia, feeding difficulties, facial dysmorphism, and brain abnormalities](https://www.uniprot.org/diseases/DI-06914)
* [Neuronopathy, distal hereditary motor, autosomal recessive 11, with spasticity](https://www.uniprot.org/diseases/DI-06919)
* [Ovarian dysgenesis 11](https://www.uniprot.org/diseases/DI-06925)
* [Parkinson disease 26, autosomal dominant](https://www.uniprot.org/diseases/DI-06928)
* [Polycystic kidney disease 8](https://www.uniprot.org/diseases/DI-06926)
* [Prostate cancer, hereditary, X-linked 3.](https://www.uniprot.org/diseases/DI-06916)
* [Spastic paraplegia 30B, autosomal recessive](https://www.uniprot.org/diseases/DI-06917)
* [Spastic paraplegia 92, autosomal recessive](https://www.uniprot.org/diseases/DI-06927)
* [Spermatogenic failure 95](https://www.uniprot.org/diseases/DI-06929)

Modified diseases:

* Anemia, sideroblastic, spinocerebellar ataxia -> [Spinocerebellar ataxia, X-linked 6, with or without sideroblastic anemia](https://www.uniprot.org/diseases/DI-02459)
* Bone fragility with contractures, arterial rupture, and deafness -> [BCARD syndrome](https://www.uniprot.org/diseases/DI-01923)
* Dyskinesia, familial, with facial myokymia -> [Dyskinesia with orofacial involvement, autosomal dominant](https://www.uniprot.org/diseases/DI-03514)
* Ehlers-Danlos syndrome, classic-like -> [Ehlers-Danlos syndrome, classic-like, 1](https://www.uniprot.org/diseases/DI-01097)
* Glycogen storage disease 2 -> [Pompe disease](https://www.uniprot.org/diseases/DI-00522)
* Lissencephaly 4 -> [Lissencephaly 4 with microcephaly](https://www.uniprot.org/diseases/DI-03160)
* Phenylketonuria -> [Phenylalanine hydroxylase deficiency](https://www.uniprot.org/diseases/DI-02159)
* Renal tubular acidosis, proximal, with ocular abnormalities and impaired intellectual development -> [Proximal renal tubular acidosis-ocular anomaly syndrome](https://www.uniprot.org/diseases/DI-02226)
* Spastic paraplegia 30 -> [Spastic paraplegia 30A, autosomal dominant](https://www.uniprot.org/diseases/DI-03243)

Deleted diseases:

* Hyperphenylalaninemia.
* Juvenile myoclonic epilepsy 7.
* Non-phenylketonuria hyperphenylalaninemia

## Changes to the [controlled vocabulary for PTMs](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/ptmlist)

New term for the feature key 'Cross-link' ('CROSSLNK' in the flat file):

* 3-(L-phenylalan-2'-yl)-L-valine (Phe-Val)

## Changes in UniProt website REST API for UniParc

UniParc entries may contain ten thousands of cross-references. This information is not relevant for all users and can make data downloads slow. For these reasons we have made the following changes to our API:

- **New endpoint for UniParc core data** (sequence and InterPro matches):
    - `/uniparc/{upi}/light`

- **New endpoints for UniParc cross-references**:
    - `/uniparc/{upi}/databases`
    - `/uniparc/{upi}/databases/stream`

- **Changes to existing endpoints**:  
  The responses of the following endpoints will no longer include cross-references:
    - `/uniparc/search`
    - `/uniparc/stream`
    - `/uniparc/upis`

- **Backward compatibility**:
    - `/uniparc/{upi}` will continue to return the full UniParc entry, including all cross-references.

All current search fields will still be supported for querying.

We have updated [uniparc.xsd](https://www.uniprot.org/docs/uniparc.xsd) to support the new endpoint for UniParc core data by making the `entry` element's child element `dbReference` optional. The UniParc XML files on our [FTP site](https://ftp.uniprot.org/pub/databases/uniprot/current_release/uniparc/xml/all/) will remain unchanged.
