---
title: UniProt release 2025_03
type: releaseNotes
date: 2025-06-18
---

# Headline

## Complement-ary information on granzyme K

The first observation of the complement system dates back to 1888, when [George Nuttall found that sheep blood serum had mild killing activity against the bacterium that causes anthrax](https://babel.hathitrust.org/cgi/pt?id=uc1.b3063944&view=1up&seq=361). The killing activity was thermolabile. Eleven years and many experiments later, [Paul Ehrlich and Julius Morgenroth name the heat-sensitive serum component 'complement'](https://babel.hathitrust.org/cgi/pt?id=uc1.c2892587&view=1up&seq=515), as it was supporting (complementing) the antimicrobial activity of antiserum. Nowadays we know that the function of the complement system [extends far beyond the elimination of microbes](https://pubmed.ncbi.nlm.nih.gov/20720586/). It acts as a rapid and efficient immune surveillance system that has distinct effects on healthy and altered host cells and foreign intruders. By eliminating cellular debris and infectious microbes, orchestrating immune responses and sending 'dangerâ€™ signals, complement contributes substantially to homeostasis, but it can also take action against healthy cells if not properly controlled.

The complement system can be activated by three main mechanisms, known as the classical, lectin and alternative pathways. In the classical pathway, activation is triggered by immunoglobulins IgM or IgG complexed with antigens. In the alternative pathway, it is the hydrolysis of the C3 component that causes complement activation. In the lectin pathway, it is the binding of a mannose-binding lectin to pathogen surfaces. Although the triggers are different, all three converge to proteolytic cascades that generate the same effector molecules: the anaphylatoxins C3a and C5a, which trigger a range of pro-inflammatory and chemotactic responses, the opsonins C4b and C3b, which label targets for clearance by phagocytosis and enhance B cell activation, and the C5b-9 membrane attack complex, which lyses target cells.

After over a century of extensive research, one could have thought that it is textbook knowledge and that no big surprise awaits us, but this year [two publications](https://pubmed.ncbi.nlm.nih.gov/39814882,39914456) revealed yet a fourth complement activation pathway, catalyzed by granzyme K ([GZMK](https://www.uniprot.org/uniprotkb/?query=gene:gzmk<ins>taxonomy_name:mammalia</ins>reviewed:true)).

Granzymes are a family of tryptase-like enzymes that are [predominantly produced by cytotoxic lymphocytes and natural killer cells](https://pubmed.ncbi.nlm.nih.gov/38689140/). They have both cytotoxic and noncytotoxic functions. During cytotoxic responses, they eliminate malignant or virus-infected cells by initiating apoptotic cell death. They can also modulate inflammation and tissue remodeling by targeting both intracellular and extracellular substrates. GZMK was, however, the least studied member of this family.

While studying rheumatoid arthritis and osteoarthritis, two autoimmune inflammatory diseases, [Donado et al.](https://pubmed.ncbi.nlm.nih.gov/39914456) found that most tissue-infiltrating CD8^+^ T cells constitutively synthesize and secrete GZMK. GZMK directly cleaves C2 and C4, similarly to the initiator proteases of the classical pathway ([C1S](https://www.uniprot.org/uniprotkb/?query=gene:c1s+AND+reviewed:true)) and lectin pathway ([MASP-1](https://www.uniprot.org/uniprotkb/?query=name:MASP-1+AND+reviewed:true)). C2b and C4b combine with each other to form C3 convertase, which, as its name indicates, catalyzes the proteolytic cleavage of C3 into C3a and C3b, a key step in the complement cascade. Of note, in the same experimental setting, another member of the family, GZMA, the closest GZMK homolog, did not exhibit this activity. GZMK substrates are at hand, as synovium fibroblasts appear to be large producers of complement C2, C3 and C4A/B transcripts in rheumatoid arthritis. In these cells, low expression is constitutive, but it is strongly increased by interferon-gamma (IFNG) and tumor necrosis factor (TNF), both of which are abundantly produced by GZMK^+^ CD8^+^ T cells. Another amazing feature of GMZK is its ability to directly bind to plasma membranes via heparan sulfate glycosaminoglycans, hence it orchestrates the assembly of membrane-bound C3 convertase, eliciting efficient opsonization of target surfaces.

C3 convertase generates bioactive products C3a and C3b. C3b molecules can then associate with membrane-localized C3 convertase and shifts its substrate specificity, enabling cleavage of C5 into two products: C5a, which is a potent anaphylatoxin, and C5b. The C5b fragment subsequently recruits C6, C7, C8 and C9 molecules to form the membrane attack complex (also known as the terminal complement complex) on the plasma membrane of the target cell. All these steps were activated by GMZK.

In the context of inflammatory diseases, such as those studied in this publication, GMZK-triggered activation of the complement cascade could lead to exacerbation of inflammation. This proved to be the case. Using mouse models of rheumatoid arthritis and psoriasis, Donado et al. were able to show that GZMK knockout animals exhibited attenuated complement activation and less severe disease phenotype.

These observations were, by and large, confirmed by [Lan et al.](https://pubmed.ncbi.nlm.nih.gov/39814882), although these latter were investigating different diseases, namely chronic rhinosinusitis, and associated nasal polyp, and asthma. However, Lan et al. found GZMK directly cleaved not only C2 and C4, but also C3 and C5, while Donado et al. found that C3 and C5 activation was indirect. Although further studies will be necessary to elucidate these points, the fact that GZMK drives the activation of the entire complement cascade remains.

These findings may have clinical impacts, such as the development of GZMK-targeted therapies to control autoimmune and inflammatory diseases, which would spare the antimicrobial functions of complement, specifically inhibiting this lymphocyte-enabled pathway in chronically inflamed tissues.

[UniProtKB/Swiss-Prot GZMK entries](https://www.uniprot.org/uniprotkb/?query=gene:gzmk+reviewed:true) have been complemented with these new findings and are publicly available as of this release.

# UniProtKB news

## Cross-references to CARD

Cross-references have been added to the CARD database, The Comprehensive Antibiotic Resistance Database.

CARD is available at [https://card.mcmaster.ca](https://card.mcmaster.ca).

The format of the explicit links is:

| Resource abbreviation | CARD |
| Resource identifier | Resource identifier |
| Optional information 1 | CARD short name |
| Optional information 2 | Resistance mechanism identifier |
| Optional information 3 | Resistance mechanism name |

Example: [P28585](https://www.uniprot.org/uniprotkb/P28585)

[Show all entries having a cross-reference to CARD.](https://www.uniprot.org/uniprotkb/?query=database:card)


### Text format

Example: [P28585](https://rest.uniprot.org/uniprotkb/P28585.txt)

```
DR   CARD; ARO:3001864; CTX-M-1; ARO:0001004; antibiotic inactivation.
```

### XML format

Example: [P28585](https://rest.uniprot.org/uniprotkb/P28585.xml)

```
<dbReference type="CARD" id="ARO:3001864">
  <property type="molecule name" value="CTX-M-1"/>
  <property type="mechanism identifier" value="ARO:0001004"/>
  <property type="mechanism name" value="antibiotic inactivation"/>
</dbReference>
```

### RDF format

Example: [P28585](https://rest.uniprot.org/uniprotkb/P28585.ttl)

```
uniprot:P28585
  up:classifiedWith <http://purl.obolibrary.org/obo/ARO_3001864> .
<http://purl.obolibrary.org/obo/ARO_3001864>
   rdfs:label "CTX-M-1" ;
   rdfs:subClassOf [ owl:onProperty <http://purl.obolibrary.org/obo/RO_0000056> ;
      owl:someValuesFrom <http://purl.obolibrary.org/obo/ARO_0001004>
      ] .
<http://purl.obolibrary.org/obo/ARO_0001004> rdfs:label "antibiotic inactivation" .
```

## Cross-references to FunCoup

Cross-references have been added to the FunCoup database, FunCoup: the functional gene/protein association networks database.

FunCoup is available at [https://funcoup.org/](https://funcoup.org/).

The format of the explicit links is:

| Resource abbreviation | FunCoup |
| Resource identifier | UniProtKB accession number |
| Optional information 1 | Number of interactors |

Example: [P35222](https://www.uniprot.org/uniprotkb/P35222)

[Show all entries having a cross-reference to FunCoup.](https://www.uniprot.org/uniprotkb/?query=database:funcoup)


### Text format

Example: [P35222](https://rest.uniprot.org/uniprotkb/P35222.txt)

```
DR   FunCoup; P35222; 2315.
```

### XML format

Example: [P35222](https://rest.uniprot.org/uniprotkb/P35222.xml)

```
<dbReference type="FunCoup" id="P35222">
  <property type="interactions" value="2315"/>
</dbReference>
```

### RDF format

Example: [P35222](https://rest.uniprot.org/uniprotkb/P35222.ttl)

```
uniprot:P35222
  rdfs:seeAlso <http://purl.uniprot.org/funcoup/P35222> .
<http://purl.uniprot.org/funcoup/P35222>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/FunCoup> ;
  rdfs:comment "2315" .
```

## Cross-references to PAN-GO

Cross-references have been added to the PAN-GO database, The PAN-GO gene functionome.

PAN-GO is available at [https://functionome.geneontology.org/](https://functionome.geneontology.org/).

The format of the explicit links is:

| Resource abbreviation | PAN-GO |
| Resource identifier | UniProtKB accession number |
| Optional information 1 | Number of GO annotations based on evolutionary models |

Example: [P28288](https://www.uniprot.org/uniprotkb/P28288)

[Show all entries having a cross-reference to PAN-GO.](https://www.uniprot.org/uniprotkb/?query=database:pan-go)


### Text format

Example: [P28288](https://rest.uniprot.org/uniprotkb/P28288.txt)

```
DR   PAN-GO; P28288; 3 GO annotations based on evolutionary models.
```

### XML format

Example: [P28288](https://rest.uniprot.org/uniprotkb/P28288.xml)

```
<dbReference type="PAN-GO" id="P28288">
  <property type="GO annotations" value="3 GO annotations based on evolutionary models"/>
</dbReference>
```

### RDF format

Example: [P28288](https://rest.uniprot.org/uniprotkb/P28288.ttl)

```
uniprot:P28288
  rdfs:seeAlso <http://purl.uniprot.org/pan-go/P28288> .
<http://purl.uniprot.org/pan-go/P28288>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/PAN-GO> ;
  rdfs:comment "3 GO annotations based on evolutionary models" .
```

## Changes to the [controlled vocabulary of human diseases](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease)

New diseases:

* [Epilepsy, idiopathic generalized 19](https://www.uniprot.org/diseases/DI-06987)
* [Heterotaxy, visceral, 13, autosomal](https://www.uniprot.org/diseases/DI-06990)
* [Heterotaxy, visceral, 14, autosomal](https://www.uniprot.org/diseases/DI-06992)
* [Immunodeficiency 131](https://www.uniprot.org/diseases/DI-07001)
* [Immunodeficiency 132B](https://www.uniprot.org/diseases/DI-07000)
* [Intellectual developmental disorder with polymicrogyria and seizures](https://www.uniprot.org/diseases/DI-06982)
* [Intellectual developmental disorder, autosomal recessive 83](https://www.uniprot.org/diseases/DI-07004)
* [Kyphomelic dysplasia](https://www.uniprot.org/diseases/DI-07003)
* [Leukodystrophy, demyelinating, adult-onset, autosomal dominant, atypical](https://www.uniprot.org/diseases/DI-06985)
* [Mitochondrial DNA depletion syndrome 21](https://www.uniprot.org/diseases/DI-06989)
* [Muggenthaler-Chowdhury-Chioza syndrome](https://www.uniprot.org/diseases/DI-06986)
* [Myopathy, myofibrillar, 13, with rimmed vacuoles](https://www.uniprot.org/diseases/DI-06995)
* [Neurodevelopmental disorder with hypotonia, poor growth, dysmorphic facies, and agammaglobulinemia](https://www.uniprot.org/diseases/DI-06994)
* [Neurodevelopmental disorder with microcephaly, absent speech, and hypotonia](https://www.uniprot.org/diseases/DI-06984)
* [Neurodevelopmental disorder with poor growth, absent speech, progressive ataxia, and dysmorphic facies](https://www.uniprot.org/diseases/DI-06993)
* [Neurodevelopmental disorder with speech or visual impairment and brain hypomyelination](https://www.uniprot.org/diseases/DI-06983)
* [Ocular pterygium-digital keloid dysplasia syndrome](https://www.uniprot.org/diseases/DI-06996)
* [Oocyte/zygote/embryo maturation arrest 22](https://www.uniprot.org/diseases/DI-06997)
* [Premature ovarian failure 26](https://www.uniprot.org/diseases/DI-06988)
* [Spermatogenic failure, X-linked, 9](https://www.uniprot.org/diseases/DI-06981)
* [Spondylometaphyseal dysplasia, Algerian type](https://www.uniprot.org/diseases/DI-06991)
* [Neuronopathy, distal hereditary motor, autosomal dominant 15](https://www.uniprot.org/diseases/DI-06998)
* [Charcot-Marie-Tooth disease, axonal, type 2JJ](https://www.uniprot.org/diseases/DI-06999)
* [Spondyloepimetaphyseal dysplasia, Li-Shao-Li type](https://www.uniprot.org/diseases/DI-07002)
* [Perrault syndrome 7](https://www.uniprot.org/diseases/DI-07005)
* [Neurodevelopmental disorder with progressive spasticity and brain abnormalities](https://www.uniprot.org/diseases/DI-07006)
* [Tubulointerstitial kidney disease, autosomal dominant 6](https://www.uniprot.org/diseases/DI-07007)
* [Neurodegenerative disorder, X-linked, female-restricted, with parkinsonism and cognitive impairement](https://www.uniprot.org/diseases/DI-07008)

Modified diseases:

* Charcot-Marie-Tooth disease, demyelinating, 1A -> [Charcot-Marie-Tooth disease, demyelinating, type 1A](https://www.uniprot.org/diseases/DI-00268)
* Charcot-Marie-Tooth disease, demyelinating, 1B -> [Charcot-Marie-Tooth disease, demyelinating, type 1B](https://www.uniprot.org/diseases/DI-00269)
* Charcot-Marie-Tooth disease, demyelinating, 1C -> [Charcot-Marie-Tooth disease, demyelinating, type 1C](https://www.uniprot.org/diseases/DI-00270)
* Charcot-Marie-Tooth disease, demyelinating, 1D -> [Charcot-Marie-Tooth disease, demyelinating, type 1D](https://www.uniprot.org/diseases/DI-00271)
* Charcot-Marie-Tooth disease, demyelinating, 1E -> [Charcot-Marie-Tooth disease, demyelinating, type 1E](https://www.uniprot.org/diseases/DI-00272)
* Charcot-Marie-Tooth disease, demyelinating, 1F -> [Charcot-Marie-Tooth disease, demyelinating, type 1F](https://www.uniprot.org/diseases/DI-00273)
* Charcot-Marie-Tooth disease, demyelinating, 1H -> [Charcot-Marie-Tooth disease, demyelinating, type 1H](https://www.uniprot.org/diseases/DI-06336)
* Charcot-Marie-Tooth disease, demyelinating, 1I -> [Charcot-Marie-Tooth disease, demyelinating, type 1I](https://www.uniprot.org/diseases/DI-06335)
* Charcot-Marie-Tooth disease, demyelinating, 1J -> [Charcot-Marie-Tooth disease, demyelinating, type 1J](https://www.uniprot.org/diseases/DI-06544)
* Charcot-Marie-Tooth disease, axonal, 2A1 -> [Charcot-Marie-Tooth disease, axonal, type 2A1](https://www.uniprot.org/diseases/DI-00274)
* Charcot-Marie-Tooth disease, axonal, 2A2A -> [Charcot-Marie-Tooth disease, axonal, type 2A2A](https://www.uniprot.org/diseases/DI-00275)
* Charcot-Marie-Tooth disease, axonal, 2A2B -> [Charcot-Marie-Tooth disease, axonal, type 2A2B](https://www.uniprot.org/diseases/DI-04811)
* Charcot-Marie-Tooth disease, axonal, 2B -> [Charcot-Marie-Tooth disease, axonal, type 2B](https://www.uniprot.org/diseases/DI-00276)
* Charcot-Marie-Tooth disease, axonal, 2B1 -> [Charcot-Marie-Tooth disease, axonal, type 2B1](https://www.uniprot.org/diseases/DI-00277)
* Charcot-Marie-Tooth disease, axonal, 2B2 -> [Charcot-Marie-Tooth disease, axonal, type 2B2](https://www.uniprot.org/diseases/DI-02671)
* Charcot-Marie-Tooth disease, axonal, 2C -> [Charcot-Marie-Tooth disease, axonal, autosomal dominant, type 2C](https://www.uniprot.org/diseases/DI-02687)
* Charcot-Marie-Tooth disease, axonal, 2CC -> [Charcot-Marie-Tooth disease, axonal, type 2CC](https://www.uniprot.org/diseases/DI-04709)
* Charcot-Marie-Tooth disease, axonal, 2D -> [Charcot-Marie-Tooth disease, axonal, type 2D](https://www.uniprot.org/diseases/DI-00278)
* Charcot-Marie-Tooth disease, axonal, 2DD -> [Charcot-Marie-Tooth disease, axonal, type 2DD](https://www.uniprot.org/diseases/DI-05276)
* Charcot-Marie-Tooth disease, axonal, 2E -> [Charcot-Marie-Tooth disease, axonal, type 2E](https://www.uniprot.org/diseases/DI-00279)
* Charcot-Marie-Tooth disease, axonal, 2EE -> [Charcot-Marie-Tooth disease, axonal, type 2EE](https://www.uniprot.org/diseases/DI-05543)
* Charcot-Marie-Tooth disease, axonal, 2F -> [Charcot-Marie-Tooth disease, axonal, type 2F](https://www.uniprot.org/diseases/DI-00280)
* Charcot-Marie-Tooth disease, axonal, 2FF -> [Charcot-Marie-Tooth disease, axonal, type 2FF](https://www.uniprot.org/diseases/DI-06222)
* Charcot-Marie-Tooth disease, axonal, 2GG -> [Charcot-Marie-Tooth disease, axonal, type 2GG](https://www.uniprot.org/diseases/DI-06681)
* Charcot-Marie-Tooth disease, axonal, 2HH -> [Charcot-Marie-Tooth disease, axonal, type 2HH](https://www.uniprot.org/diseases/DI-06245)
* Charcot-Marie-Tooth disease, axonal, 2I -> [Charcot-Marie-Tooth disease, axonal, type 2I](https://www.uniprot.org/diseases/DI-00281)
* Charcot-Marie-Tooth disease, axonal, 2II -> [Charcot-Marie-Tooth disease, axonal, type 2II](https://www.uniprot.org/diseases/DI-06520)
* Charcot-Marie-Tooth disease, axonal, 2J -> [Charcot-Marie-Tooth disease, axonal, type 2J](https://www.uniprot.org/diseases/DI-00282)
* Charcot-Marie-Tooth disease, axonal, 2K -> [Charcot-Marie-Tooth disease, axonal, type 2K](https://www.uniprot.org/diseases/DI-00283)
* Charcot-Marie-Tooth disease, axonal, 2L -> [Charcot-Marie-Tooth disease, axonal, type 2L](https://www.uniprot.org/diseases/DI-00284)
* Charcot-Marie-Tooth disease, axonal, 2M -> [Charcot-Marie-Tooth disease, axonal, type 2M](https://www.uniprot.org/diseases/DI-03481)
* Charcot-Marie-Tooth disease, axonal, 2N -> [Charcot-Marie-Tooth disease, axonal, type 2N](https://www.uniprot.org/diseases/DI-02678)
* Charcot-Marie-Tooth disease, axonal, 2O -> [Charcot-Marie-Tooth disease, axonal, type 2O](https://www.uniprot.org/diseases/DI-03264)
* Charcot-Marie-Tooth disease, axonal, 2P -> [Charcot-Marie-Tooth disease, axonal, type 2P](https://www.uniprot.org/diseases/DI-03339)
* Charcot-Marie-Tooth disease, axonal, 2Q -> [Charcot-Marie-Tooth disease, axonal, type 2Q](https://www.uniprot.org/diseases/DI-03672)
* Charcot-Marie-Tooth disease, axonal, 2R -> [Charcot-Marie-Tooth disease, axonal, type 2R](https://www.uniprot.org/diseases/DI-03924)
* Charcot-Marie-Tooth disease, axonal, 2S -> [Charcot-Marie-Tooth disease, axonal, type 2S](https://www.uniprot.org/diseases/DI-04308)
* Charcot-Marie-Tooth disease, axonal, 2T -> [Charcot-Marie-Tooth disease, axonal, type 2T](https://www.uniprot.org/diseases/DI-04343)
* Charcot-Marie-Tooth disease, axonal, 2U -> [Charcot-Marie-Tooth disease, axonal, type 2U](https://www.uniprot.org/diseases/DI-04362)
* Charcot-Marie-Tooth disease, axonal, 2V -> [Charcot-Marie-Tooth disease, axonal, type 2V](https://www.uniprot.org/diseases/DI-04496)
* Charcot-Marie-Tooth disease, axonal, 2W -> [Charcot-Marie-Tooth disease, axonal, type 2W](https://www.uniprot.org/diseases/DI-04574)
* Charcot-Marie-Tooth disease, axonal, 2X -> [Charcot-Marie-Tooth disease, axonal, type 2X](https://www.uniprot.org/diseases/DI-04588)
* Charcot-Marie-Tooth disease, axonal, 2Y -> [Charcot-Marie-Tooth disease, axonal, type 2Y](https://www.uniprot.org/diseases/DI-04589)
* Charcot-Marie-Tooth disease, axonal, 2Z -> [Charcot-Marie-Tooth disease, axonal, type 2Z](https://www.uniprot.org/diseases/DI-04590)
* Charcot-Marie-Tooth disease, demyelinating, 1G -> [Charcot-Marie-Tooth disease, demyelinating, type 1G](https://www.uniprot.org/diseases/DI-05460)
* Charcot-Marie-Tooth disease, demyelinating, 4A -> [Charcot-Marie-Tooth disease, demyelinating, type 4A](https://www.uniprot.org/diseases/DI-00285)
* Charcot-Marie-Tooth disease, demyelinating, 4B1 -> [Charcot-Marie-Tooth disease, demyelinating, type 4B1](https://www.uniprot.org/diseases/DI-00286)
* Charcot-Marie-Tooth disease, demyelinating, 4B2 -> [Charcot-Marie-Tooth disease, demyelinating, type 4B2](https://www.uniprot.org/diseases/DI-00287)
* Charcot-Marie-Tooth disease, demyelinating, 4B3 -> [Charcot-Marie-Tooth disease, demyelinating, type 4B3](https://www.uniprot.org/diseases/DI-03784)
* Charcot-Marie-Tooth disease, demyelinating, 4C -> [Charcot-Marie-Tooth disease, demyelinating, type 4C](https://www.uniprot.org/diseases/DI-00288)
* Charcot-Marie-Tooth disease, demyelinating, 4D -> [Charcot-Marie-Tooth disease, demyelinating, type 4D](https://www.uniprot.org/diseases/DI-00289)
* Charcot-Marie-Tooth disease, demyelinating, 4F -> [Charcot-Marie-Tooth disease, demyelinating, type 4F](https://www.uniprot.org/diseases/DI-03559)
* Charcot-Marie-Tooth disease, demyelinating, 4H -> [Charcot-Marie-Tooth disease, demyelinating, type 4H](https://www.uniprot.org/diseases/DI-00290)
* Charcot-Marie-Tooth disease, demyelinating, 4J -> [Charcot-Marie-Tooth disease, demyelinating, type 4J](https://www.uniprot.org/diseases/DI-00291)
* Charcot-Marie-Tooth disease, demyelinating, 4K -> [Charcot-Marie-Tooth disease, demyelinating, type 4K](https://www.uniprot.org/diseases/DI-04591)
* Encephalopathy, acute, infection-induced, 5, herpes-specific -> [Immunodeficiency 132A](https://www.uniprot.org/diseases/DI-03543)
* Infantile cataract, skin abnormalities, glutamate excess, and impaired intellectual development -> [CASGID syndrome](https://www.uniprot.org/diseases/DI-05490)
* Kariminejad-Reversade neurodevelopmental syndrome -> [Kariminejad neurodevelopmental syndrome](https://www.uniprot.org/diseases/DI-06937)
* Leukodystrophy, demyelinating, autosomal dominant, adult-onset -> [Leukodystrophy, demyelinating, adult-onset, autosomal dominant, typical](https://www.uniprot.org/diseases/DI-00646)
* Posterior column ataxia with retinitis pigmentosa -> [Retinopathy-sensory neuropathy syndrome](https://www.uniprot.org/diseases/DI-03015)
* Tubulointerstitial kidney disease, autosomal dominant, 1 -> [Tubulointerstitial kidney disease, autosomal dominant 1](https://www.uniprot.org/diseases/DI-00493)
* Tubulointerstitial kidney disease, autosomal dominant, 2 -> [Tubulointerstitial kidney disease, autosomal dominant 2](https://www.uniprot.org/diseases/DI-03826)
* Tubulointerstitial kidney disease, autosomal dominant, 4 -> [Tubulointerstitial kidney disease, autosomal dominant 4](https://www.uniprot.org/diseases/DI-02783)
* Tubulointerstitial kidney disease, autosomal dominant, 5 -> [Tubulointerstitial kidney disease, autosomal dominant 5](https://www.uniprot.org/diseases/DI-04780)

## Changes to the [controlled vocabulary for PTMs](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/ptmlist)

Modified term for the feature key 'Modified residue' ('MOD_RES' in the flat file):

* O-UMP-histidine -> N(tele)-UMP-histidine

# RDF news

## Change of URIs for the PROSITE database in RDF

For historic reasons, UniProt had to generate URIs to cross-reference databases that did not have an RDF representation or did not provide stable URIs such as [PURLs](https://en.wikipedia.org/wiki/Persistent_uniform_resource_locator) (permanent uniform resource locators). The SIB Swiss Institute of Bioinformatics has now committed to provide stable URIs for the resources that it supports. [PROSITE](https://prosite.expasy.org/) is now also using this service, therefore we have changed the PROSITE URIs accordingly:

The links to PROSITE signatures (used in UniProtKB cross-references and in UniParc) change from:

```
http://purl.uniprot.org/prosite/PS50835
```
to:
```
https://purl.expasy.org/prosite/signature/PS50835
```

The links to PROSITE annotation rules (aka: ProRule as used in UniProtKB attributions) change from:
```
http://purl.uniprot.org/prosite-prorule/PRU00114
```
to:
```
https://purl.expasy.org/prosite/rule/PRU00114
```

## Change of URIs for taxonomic ranks in RDF

The NCBI [replaced the taxonomic rank 'Superkingdom' by the new ranks 'Domain' and 'Realm'](https://ncbiinsights.ncbi.nlm.nih.gov/2025/02/27/new-ranks-ncbi-taxonomy/). We use this opportunity to change the URIs of all ranks by adding the prefix `Taxonomic_Rank_` to add context to terms like 'Domain' that can have several meanings in the context of UniProt. For instance,

```
<owl:Thing rdf:about="http://purl.uniprot.org/core/Species">
```

changes to:
```
<owl:Thing rdf:about="http://purl.uniprot.org/core/Taxonomic_Rank_Species">
```
