---
title: UniProt release 2026_01
type: releaseNotes
date: 2026-01-28
---

# Headline

## Complement-ary information on granzyme K

The first observation of the complement system dates back to 1888, when [George Nuttall found that sheep blood serum had mild killing activity against the bacterium that causes anthrax](https://babel.hathitrust.org/cgi/pt?id=uc1.b3063944&view=1up&seq=361). The killing activity was thermolabile. Eleven years and many experiments later, [Paul Ehrlich and Julius Morgenroth name the heat-sensitive serum component 'complement'](https://babel.hathitrust.org/cgi/pt?id=uc1.c2892587&view=1up&seq=515), as it was supporting (complementing) the antimicrobial activity of antiserum. Nowadays we know that the function of the complement system [extends far beyond the elimination of microbes](https://pubmed.ncbi.nlm.nih.gov/20720586/). It acts as a rapid and efficient immune surveillance system that has distinct effects on healthy and altered host cells and foreign intruders. By eliminating cellular debris and infectious microbes, orchestrating immune responses and sending 'dangerâ€™ signals, complement contributes substantially to homeostasis, but it can also take action against healthy cells if not properly controlled.

The complement system can be activated by three main mechanisms, known as the classical, lectin and alternative pathways. In the classical pathway, activation is triggered by immunoglobulins IgM or IgG complexed with antigens. In the alternative pathway, it is the hydrolysis of the C3 component that causes complement activation. In the lectin pathway, it is the binding of a mannose-binding lectin to pathogen surfaces. Although the triggers are different, all three converge to proteolytic cascades that generate the same effector molecules: the anaphylatoxins C3a and C5a, which trigger a range of pro-inflammatory and chemotactic responses, the opsonins C4b and C3b, which label targets for clearance by phagocytosis and enhance B cell activation, and the C5b-9 membrane attack complex, which lyses target cells.

After over a century of extensive research, one could have thought that it is textbook knowledge and that no big surprise awaits us, but this year [two publications](https://pubmed.ncbi.nlm.nih.gov/39814882,39914456) revealed yet a fourth complement activation pathway, catalyzed by granzyme K ([GZMK](https://www.uniprot.org/uniprotkb/?query=gene:gzmk+AND+taxonomy_name:mammalia+AND+reviewed:true)).

Granzymes are a family of tryptase-like enzymes that are [predominantly produced by cytotoxic lymphocytes and natural killer cells](https://pubmed.ncbi.nlm.nih.gov/38689140/). They have both cytotoxic and noncytotoxic functions. During cytotoxic responses, they eliminate malignant or virus-infected cells by initiating apoptotic cell death. They can also modulate inflammation and tissue remodeling by targeting both intracellular and extracellular substrates. GZMK was, however, the least studied member of this family.

While studying rheumatoid arthritis and osteoarthritis, two autoimmune inflammatory diseases, [Donado et al.](https://pubmed.ncbi.nlm.nih.gov/39914456) found that most tissue-infiltrating CD8^+^ T cells constitutively synthesize and secrete GZMK. GZMK directly cleaves C2 and C4, similarly to the initiator proteases of the classical pathway ([C1S](https://www.uniprot.org/uniprotkb/?query=gene:c1s+AND+reviewed:true)) and lectin pathway ([MASP-1](https://www.uniprot.org/uniprotkb/?query=name:MASP-1+AND+reviewed:true)). C2b and C4b combine with each other to form C3 convertase, which, as its name indicates, catalyzes the proteolytic cleavage of C3 into C3a and C3b, a key step in the complement cascade. Of note, in the same experimental setting, another member of the family, GZMA, the closest GZMK homolog, did not exhibit this activity. GZMK substrates are at hand, as synovium fibroblasts appear to be large producers of complement C2, C3 and C4A/B transcripts in rheumatoid arthritis. In these cells, low expression is constitutive, but it is strongly increased by interferon-gamma (IFNG) and tumor necrosis factor (TNF), both of which are abundantly produced by GZMK^+^ CD8^+^ T cells. Another amazing feature of GMZK is its ability to directly bind to plasma membranes via heparan sulfate glycosaminoglycans, hence it orchestrates the assembly of membrane-bound C3 convertase, eliciting efficient opsonization of target surfaces.

C3 convertase generates bioactive products C3a and C3b. C3b molecules can then associate with membrane-localized C3 convertase and shifts its substrate specificity, enabling cleavage of C5 into two products: C5a, which is a potent anaphylatoxin, and C5b. The C5b fragment subsequently recruits C6, C7, C8 and C9 molecules to form the membrane attack complex (also known as the terminal complement complex) on the plasma membrane of the target cell. All these steps were activated by GMZK.

In the context of inflammatory diseases, such as those studied in this publication, GMZK-triggered activation of the complement cascade could lead to exacerbation of inflammation. This proved to be the case. Using mouse models of rheumatoid arthritis and psoriasis, Donado et al. were able to show that GZMK knockout animals exhibited attenuated complement activation and less severe disease phenotype.

These observations were, by and large, confirmed by [Lan et al.](https://pubmed.ncbi.nlm.nih.gov/39814882), although these latter were investigating different diseases, namely chronic rhinosinusitis, and associated nasal polyp, and asthma. However, Lan et al. found GZMK directly cleaved not only C2 and C4, but also C3 and C5, while Donado et al. found that C3 and C5 activation was indirect. Although further studies will be necessary to elucidate these points, the fact that GZMK drives the activation of the entire complement cascade remains.

These findings may have clinical impacts, such as the development of GZMK-targeted therapies to control autoimmune and inflammatory diseases, which would spare the antimicrobial functions of complement, specifically inhibiting this lymphocyte-enabled pathway in chronically inflamed tissues.

[UniProtKB/Swiss-Prot GZMK entries](https://www.uniprot.org/uniprotkb/?query=gene:gzmk+reviewed:true) have been complemented with these new findings and are publicly available as of this release.

# UniProtKB news

## Continuing to reorganize the protein space in UniProtKB

Starting from release 2025_04 (15th October 2025), we have been refactoring our [Reference Proteome](https://www.uniprot.org/help/reference_proteome) selection pipeline to ensure an improved representation of species biodiversity in the UniProt Knowledgebase (UniProtKB). From release 2026_02 onwards (currently planned for the first half of 2026), we will restrict the protein space in UniProtKB to only those sequences which are part of a Reference Proteome in addition to the expert reviewed UniProtKB/Swiss-Prot section, and also unreviewed entries associated with experimental [Gene Ontology](https://www.uniprot.org/help/gene-ontology) annotations or additional biologically important data such as a 3D structure.

This reorganization, between release 2025_04 and 2026_02, will result in a change in the content and size of the database. The number of Reference Proteomes will increase by 36% (reflecting a 34% increase on species covered), while the number of proteins in UniProtKB can be decreased by 45%.

All deprecated proteomes ([download list](https://ftp.ebi.ac.uk/pub/contrib/UniProt/proteomes/proteomes_losing_UniProtKB_entries.tsv)), plus all proteomes which do not qualify for consideration in the new pipeline, will be available through [UniParc](https://www.uniprot.org/uniparc/). When you search in the [Proteomes](https://www.uniprot.org/proteomes/) portal for these proteomes, you will be directed to UniParc to access your protein set. To download your proteome, we recommend to use the new [UniParc proteomes FASTA format](https://www.uniprot.org/help/fasta-headers#uniparc-proteomes) which contains biological information from the UniParc source database entries that are associated with the requested proteome.

For more details on our novel workflow and how reference proteomes are changing, check: [Inside UniProt: Capturing the Diversity of Life - Reorganizing the Protein Space in UniProtKB](https://insideuniprot.blogspot.com/2025/06/capturing-diversity-of-life.html)

For a brief summary of the changes to proteomes and subsequent UniProtKB changes please visit our [summary of proteomes](https://www.uniprot.org/help/refprot_only_changes) changes help page.

## Removal of the cross-references to neXtProt

Cross-references to neXtProt have been removed.

## Removal of the cross-references to TreeFam

Cross-references to TreeFam have been removed.

## Cross-references to Agora

Cross-references have been added to Agora database, the Alzheimer's Disease Explorer database.

Agora is available at [https://agora.adknowledgeportal.org](https://agora.adknowledgeportal.org).

The format of the explicit links is:

| Field | Description |
| --- | --- |
| Resource abbreviation | Agora |
| Resource identifier | Resource identifier |
| Optional information 1 | Alzheimer target status |

The Alzheimer target status can take the following values:
* "Nominated Target for Alzheimer's Disease" if Agora has identified the gene to be a good target for new Alzheimer's Disease treatment or prevention
* "-" if not

Example: [Q15149](https://www.uniprot.org/uniprotkb/Q15149)

[Show all entries having a cross-reference to Agora.](https://www.uniprot.org/uniprotkb/?query=database:agora)

### Text format

Example: [Q15149](https://rest.uniprot.org/uniprotkb/Q15149.txt)

```
DR   Agora; ENSG00000178209; Nominated Target for Alzheimer's Disease.
```

### XML format

Example: [Q15149](https://rest.uniprot.org/uniprotkb/Q15149.xml)

```
<dbReference type="Agora" id="ENSG00000178209">
  <property type="nominated target" value="Nominated Target for Alzheimer's Disease"/>
</dbReference>
```

### RDF format

Example: [Q15149](https://rest.uniprot.org/uniprotkb/Q15149.ttl)

```
uniprot:Q15149
  rdfs:seeAlso <http://purl.uniprot.org/agora/ENSG00000178209> .
<http://purl.uniprot.org/agora/ENSG00000178209>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/Agora> ;
  rdfs:comment "Nominated Target for Alzheimer's Disease" .
```

## Cross-references to CIViC

Cross-references have been added to the CIViC database, a database for Clinical Interpretation of Variants in Cancer.

CIViC is available at [https://civicdb.org](https://civicdb.org).

The format of the explicit links is:

| Field | Description |
| --- | --- |
| Resource abbreviation | CIViC |
| Resource identifier | Resource identifier |
| Optional information 1 | Number of evidence items |

Example: [P31749](https://www.uniprot.org/uniprotkb/P31749)

[Show all entries having a cross-reference to CIViC.](https://www.uniprot.org/uniprotkb/?query=database:civic)


### Text format

Example: [P31749](https://rest.uniprot.org/uniprotkb/P31749.txt)

```
DR   CIViC; 207; 18 evidence items across 16 molecular profiles.
```

### XML format

Example: [P31749](https://rest.uniprot.org/uniprotkb/P31749.xml)

```
<dbReference type="CIViC" id="207">
  <property type="evidences" value="18 evidence items across 16 molecular profiles"/>
</dbReference>
```

### RDF format

Example: [P31749](https://rest.uniprot.org/uniprotkb/P31749.ttl)

```
uniprot:P31749
  rdfs:seeAlso <http://purl.uniprot.org/civic/207> .
<http://purl.uniprot.org/civic/207>
  rdf:type up:Resource ;
  up:database <http://purl.uniprot.org/database/CIViC> ;
  rdfs:comment "18 evidence items across 16 molecular profiles" .
```

## Change of the cross-references to PharmGKB

The PharmGKB resource has changed its name to ClinPGx and we have updated our cross-references to reflect this name change.

## Changes to the [controlled vocabulary of human diseases](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease)

New diseases:

* [Achalasia-progeroid syndrome](https://www.uniprot.org/diseases/DI-07010)
* [Alsahan-Harris syndrome](https://www.uniprot.org/diseases/DI-07071)
* [Autoimmune disease, multisystem, infantile-onset, 5](https://www.uniprot.org/diseases/DI-07050)
* [Brain small vessel disease 4](https://www.uniprot.org/diseases/DI-07081)
* [Brain small vessel disease 5 with osteoporosis](https://www.uniprot.org/diseases/DI-07084)
* [Camurati-Engelmann disease 2](https://www.uniprot.org/diseases/DI-07091)
* [Cardiomyopathy, dilated, 1QQ](https://www.uniprot.org/diseases/DI-07054)
* [Cardiomyopathy, dilated, 2L](https://www.uniprot.org/diseases/DI-07052)
* [Cardiomyopathy, dilated, 2M](https://www.uniprot.org/diseases/DI-07061)
* [Cardiomyopathy, familial hypertrophic, 31](https://www.uniprot.org/diseases/DI-07067)
* [Cataract, alopecia, oral mucosal disorder, and psoriasis-like syndrome](https://www.uniprot.org/diseases/DI-07056)
* [Cerebral arteriopathy, autosomal recessive, with subcortical infarcts and leukoencephalopathy 1](https://www.uniprot.org/diseases/DI-07076)
* [Ciliary dyskinesia, primary, 54](https://www.uniprot.org/diseases/DI-07012)
* [Ciliary dyskinesia, primary, 55](https://www.uniprot.org/diseases/DI-07070)
* [Combined oxidative phosphorylation deficiency 60](https://www.uniprot.org/diseases/DI-07036)
* [Congenital disorder of glycosylation 1EE with or without immunodeficiency](https://www.uniprot.org/diseases/DI-07015)
* [Congenital myopathy 26](https://www.uniprot.org/diseases/DI-07043)
* [Congenital myopathy 27](https://www.uniprot.org/diseases/DI-07088)
* [Cranioectodermal dysplasia 5](https://www.uniprot.org/diseases/DI-07023)
* [Cranioectodermal dysplasia 6](https://www.uniprot.org/diseases/DI-07087)
* [Craniofaciocardiohepatic syndrome](https://www.uniprot.org/diseases/DI-07032)
* [Developmental and epileptic encephalopathy 118](https://www.uniprot.org/diseases/DI-07055)
* [Developmental dysplasia of the hip 4](https://www.uniprot.org/diseases/DI-07074)
* [Diamond-Blackfan anemia 22](https://www.uniprot.org/diseases/DI-07062)
* [Diarrhea 14, congenital](https://www.uniprot.org/diseases/DI-07017)
* [Diarrhea 15, congenital](https://www.uniprot.org/diseases/DI-07018)
* [Dursun-Ozgul neurodevelopmental syndrome](https://www.uniprot.org/diseases/DI-07089)
* [Ectodermal dysplasia 17 with or without limb malformations](https://www.uniprot.org/diseases/DI-07042)
* [Fanconi anemia, complementation group X](https://www.uniprot.org/diseases/DI-07058)
* [Fetomaternal alloimmune thrombocytopenia 1](https://www.uniprot.org/diseases/DI-07077)
* [Fetomaternal alloimmune thrombocytopenia 2](https://www.uniprot.org/diseases/DI-07078)
* [Fetomaternal alloimmune thrombocytopenia 3](https://www.uniprot.org/diseases/DI-07079)
* [FICUS syndrome](https://www.uniprot.org/diseases/DI-07033)
* [Fischer-Zirnsak progeroid syndrome](https://www.uniprot.org/diseases/DI-07014)
* [Guillouet-Gordon syndrome](https://www.uniprot.org/diseases/DI-07041)
* [Holoprosencephaly 10](https://www.uniprot.org/diseases/DI-07016)
* [Houge-Janssens syndrome 4](https://www.uniprot.org/diseases/DI-07030)
* [Infection-induced acute-onset axonal neuropathy](https://www.uniprot.org/diseases/DI-07086)
* [Leukodystrophy, adult-onset, autosomal dominant, without amyloid angiopathy](https://www.uniprot.org/diseases/DI-07044)
* [Leukodystrophy and cerebellar atrophy](https://www.uniprot.org/diseases/DI-07031)
* [Li-Takada-Miyake syndrome](https://www.uniprot.org/diseases/DI-07040)
* [ICHAD syndrome](https://www.uniprot.org/diseases/DI-07049)
* [Immunodeficiency 133 with ectodermal dysplasia with or without peripheral neuropathy](https://www.uniprot.org/diseases/DI-07057)
* [Immunodysregulation with variable immunodeficiency and autoimmunity](https://www.uniprot.org/diseases/DI-07048)
* [Intellectual developmental disorder, autosomal dominant 76](https://www.uniprot.org/diseases/DI-07069)
* [Lipodystrophy, familial partial, 1](https://www.uniprot.org/diseases/DI-07075)
* [Maturity-onset diabetes of the young 12](https://www.uniprot.org/diseases/DI-07037)
* [Monilethrix 2](https://www.uniprot.org/diseases/DI-07024)
* [Monilethrix 3](https://www.uniprot.org/diseases/DI-07025)
* [Nanophthalmos 1](https://www.uniprot.org/diseases/DI-07027)
* [Neurodegeneration, early-childhood-onset, with retinitis pigmentosa, sensorineural hearing loss, and demyelinating peripheral neuropathy](https://www.uniprot.org/diseases/DI-07013)
* [Neurodevelopmental disorder with achalasia, polyneuropathy, and alacrima](https://www.uniprot.org/diseases/DI-07083)
* [Neurodevelopmental disorder with ataxia and brain abnormalities](https://www.uniprot.org/diseases/DI-07038)
* [Neurodevelopmental disorder with behavioral, ear, and skeletal abnormalities](https://www.uniprot.org/diseases/DI-07063)
* [Neurodevelopmental disorder with dysmorphic facies, brain anomalies, and seizures](https://www.uniprot.org/diseases/DI-07039)
* [Neurodevelopmental disorder with poor growth, seizures, and brain abnormalities](https://www.uniprot.org/diseases/DI-07021)
* [Neurodevelopmental disorder with poor or absent speech, dysmorphic facies, and behavioral abnormalities](https://www.uniprot.org/diseases/DI-07029)
* [Neurodevelopmental disorder with thin corpus callosum, hypotonia, and absent language](https://www.uniprot.org/diseases/DI-07019)
* [Neurodevelopmental disorder with white matter abnormalities and gait disturbance](https://www.uniprot.org/diseases/DI-07020)
* [Nil-Deshwar neurodevelopmental syndrome](https://www.uniprot.org/diseases/DI-07064)
* [Oculovertebral syndrome](https://www.uniprot.org/diseases/DI-07073)
* [Oocyte/zygote/embryo maturation arrest 23](https://www.uniprot.org/diseases/DI-07046)
* [Oocyte/zygote/embryo maturation arrest 24](https://www.uniprot.org/diseases/DI-07047)
* [Orofaciodigital syndrome 9](https://www.uniprot.org/diseases/DI-07072)
* [Paul-Chao neurodevelopmental syndrome](https://www.uniprot.org/diseases/DI-07009)
* [Polycystic kidney disease 9](https://www.uniprot.org/diseases/DI-07022)
* [Pompe disease, late-onset](https://www.uniprot.org/diseases/DI-07080)
* [Popov-Chang syndrome](https://www.uniprot.org/diseases/DI-05565)
* [Pulmonary hypertension, primary, 7](https://www.uniprot.org/diseases/DI-07053)
* [Rhabdomyolysis 2](https://www.uniprot.org/diseases/DI-07051)
* [Retinitis pigmentosa 99](https://www.uniprot.org/diseases/DI-07028)
* [Retinitis pigmentosa 100](https://www.uniprot.org/diseases/DI-07068)
* [Short-rib thoracic dysplasia 22 without polydactyly](https://www.uniprot.org/diseases/DI-07060)
* [Spastic ataxia 11, autosomal dominant](https://www.uniprot.org/diseases/DI-07045)
* [Spermatogenic failure 98](https://www.uniprot.org/diseases/DI-07011)
* [Spermatogenic failure 99](https://www.uniprot.org/diseases/DI-07034)
* [Spermatogenic failure 100](https://www.uniprot.org/diseases/DI-07035)
* [Spermatogenic failure 101](https://www.uniprot.org/diseases/DI-07066)
* [Spondyloepiphyseal dysplasia, Holling type](https://www.uniprot.org/diseases/DI-07090)
* [Stargardt disease 5](https://www.uniprot.org/diseases/DI-07059)
* [Tayoun-Maawali syndrome](https://www.uniprot.org/diseases/DI-07026)
* [Ververi-Brady syndrome 2](https://www.uniprot.org/diseases/DI-07082)
* [Vitreoretinopathy, exudative 8](https://www.uniprot.org/diseases/DI-07065)
* [Wilms tumor 7](https://www.uniprot.org/diseases/DI-07085)

Modified diseases:

* Amyotrophic lateral sclerosis 22, with or without frontotemporal dementia -> [Frontotemporal dementia and/or amyotrophic lateral sclerosis 9](https://www.uniprot.org/diseases/DI-04318)
* Autoimmune disease with susceptibility to Mycobacterium tuberculosis -> [Autoimmune disease, multisystem, infantile-onset, 4](https://www.uniprot.org/diseases/DI-06972)
* Cerebral arteriopathy, autosomal recessive, with subcortical infarcts and leukoencephalopathy -> [Cerebral arteriopathy, autosomal recessive, with subcortical infarcts and leukoencephalopathy 2](https://www.uniprot.org/diseases/DI-02549)
* Cholestasis, neonatal intrahepatic, caused by citrin deficiency -> [Citrin deficiency, neonatal or infantile onset](https://www.uniprot.org/diseases/DI-00799)
* Citrullinemia 2 -> [Citrin deficiency, adolescent or adult onset](https://www.uniprot.org/diseases/DI-00310)
* Cone dystrophy, retinal 3A -> [Achromatopsia 6](https://www.uniprot.org/diseases/DI-00315)
* Cone dystrophy retinal 3B -> [Cone dystrophy with supernormal rod responses](https://www.uniprot.org/diseases/DI-00316)
* Diarrhea 4, malabsorptive, congenital -> [Diarrhea 4, malabsorptive, congenital, with diabetes mellitus and combined pituitary hormone deficiency](https://www.uniprot.org/diseases/DI-01408)
* Dominant optic atrophy plus syndrome -> [Optic atrophy plus syndrome](https://www.uniprot.org/diseases/DI-02096)
* Ectodermal dysplasia 14, hair/tooth type with or without hypohidrosis -> [Ectodermal dysplasia 14, hypohidrotic/hair/tooth/nail type](https://www.uniprot.org/diseases/DI-05382)
* Global developmental delay with or without impaired intellectual development -> [Neurodevelopmental disorder with developmental delay and with or without motor or speech delay](https://www.uniprot.org/diseases/DI-05485)
* Immunodeficiency 39 -> [Immunodeficiency 39, susceptibility to viral infections](https://www.uniprot.org/diseases/DI-04423)
* Lymphedema-yellow nails -> [Yellow nail syndrome](https://www.uniprot.org/diseases/DI-00691)
* Monilethrix -> [Monilethrix 1](https://www.uniprot.org/diseases/DI-01992)
* Myasthenic syndrome, congenital, 18 -> [Developmental and epileptic encephalopathy 117](https://www.uniprot.org/diseases/DI-04403)
* Myopathy, myofibrillar, 2 -> [Myopathy, myofibrillar, 2A, adult-onset](https://www.uniprot.org/diseases/DI-01179)
* Myopathy, myofibrillar, fatal infantile hypertonic, alpha-B crystallin-related -> [Myopathy, myofibrillar, 2B, infantile-onset](https://www.uniprot.org/diseases/DI-03083)
* Pompe disease -> [Pompe disease, infantile-onset](https://www.uniprot.org/diseases/DI-00522)
* Spondyloepiphyseal dysplasia Maroteaux type -> [Spondyloepiphyseal dysplasia, Maroteaux type](https://www.uniprot.org/diseases/DI-02969)
* Ververi-Brady syndrome -> [Ververi-Brady syndrome 1](https://www.uniprot.org/diseases/DI-05250)

Deleted diseases:

* GNAS hyperfunction
* Spinocerebellar ataxia 47

## Changes to the [controlled vocabulary for PTMs](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/ptmlist)

New term for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
* Lysine 5-hydroxy-oxazole-4-carbothionic acid (Lys-Cys)

New terms for the feature key 'Glycosylation' ('CARBOHYD' in the flat file):

* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAcl-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked ([beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked ([alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine
* N-linked (alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->3)-[beta-D-GlcNAc-(1->4)]-[alpha-Neu5Ac-(2->6)-beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->2)-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-[alpha-L-Fuc-(1->6)]-beta-D-GlcNAc) (complex) asparagine

Deleted term for the feature key 'Glycosylation' ('CARBOHYD' in the flat file):

* O-linked (Xyl...) (keratan sulfate) threonine

New term for the feature key 'Modified residue' ('MOD_RES' in the flat file):

* N6-isobutyryllysine

## Changes in [subcellular location controlled vocabulary](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/subcell)

New subcellular locations:

* [Apicoplast membrane](https://www.uniprot.org/locations/SL-0554)
* [Host cell apoplast](https://www.uniprot.org/locations/SL-0555)
